{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/usr/bin/env python\n",
    "# -*- coding: utf-8 -*-\n",
    "import pandas as pd\n",
    "from Bio import SeqIO\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "import shlex\n",
    "from Bio.Seq import Seq\n",
    "import subprocess\n",
    "import os\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "#helper functions\n",
    "def read_gff(path):\n",
    "    df = pd.read_csv(path, index_col=False, sep='\\t', header=None, comment=\"#\")\n",
    "    df.columns = ['seqname', 'source', 'feature', 'start', 'end', 'score', 'strand', 'frame', 'attribute']\n",
    "    return df\n",
    "\n",
    "def sys(command):\n",
    "    \"\"\"\n",
    "    \"\"\"\n",
    "    print(\"-\" * 10)\n",
    "    print(\"Executing: %s\" % command)\n",
    "    process = subprocess.Popen(shlex.split(command),stdout=subprocess.PIPE,stderr=subprocess.PIPE, cwd=\"/home/juan/Desktop/juan/bio/gliadinas/\")\n",
    "    stdout, stderr = process.communicate()\n",
    "    #print(\"Result: stdout: %s - stderr: %s\" % (stdout, stderr))\n",
    "    #print(\"-\" * 10)\n",
    "    if stderr:\n",
    "        print(stderr)\n",
    "    return stdout, stderr\n",
    "\n",
    "\n",
    "\n",
    "def gff2FA(df_gff, path_sequence, windows, output):\n",
    "    \"\"\"Extract fasta files from annotations\n",
    "    \"\"\"\n",
    "    fasta_seq = SeqIO.parse(path_genome, 'fasta')\n",
    "    buffer_seqs = []\n",
    "    cont = 0\n",
    "    for record in fasta_seq:\n",
    "        print(record.id)\n",
    "        dff_extract = df_gff[df_gff.seqname == record.id]\n",
    "        for key,val in dff_extract.iterrows():\n",
    "            clean_seq = ''.join(str(record.seq).splitlines())\n",
    "            if int(val.start) - windows < 0:\n",
    "                start = 0\n",
    "            else:\n",
    "                start = int(val.start) - windows\n",
    "            if int(val.end) + windows > len(clean_seq):\n",
    "                end = len(clean_seq)\n",
    "            else:\n",
    "                end = int(val.end) + windows\n",
    "            new_seq = clean_seq[start:end]\n",
    "            att = val.attribute\n",
    "            id = record.id + '_' + str(start) + '_' + str(end)\n",
    "            desc = \"seq_id:\" + str(record.id)\n",
    "            desc += \" feature_start:\" + str(val.start)\n",
    "            desc += \" feature_end:\" + str(val.end)\n",
    "            desc += \" genome_start:\" + str(start)\n",
    "            desc += \" genome_end:\" + str(end)\n",
    "            desc += \" feature:\" + str(val.feature)\n",
    "            desc += \" attributes:\" + val.attribute\n",
    "            seq = SeqRecord(Seq(new_seq), id=id, description=desc)\n",
    "            buffer_seqs.append(seq)\n",
    "            cont += 1\n",
    "    SeqIO.write(buffer_seqs, output, \"fasta\")\n",
    "    return buffer_seqs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "adapters = \"data/adapters_new_rc.fa\"\n",
    "path_output_dir = \"data/clean/\"\n",
    "path_spades = \"sw/SPAdes-3.13.0-Linux/bin/spades\"\n",
    "path_cutadapt = \"sw/cutadapt-2.7/\"\n",
    "path_fastqc = \"sw/FastQC/\"\n",
    "adapter_f = \"AATGATACGGCGACCACCGAGATCTACAC\"\n",
    "adapter_r = \"CAAGCAGAAGACGGCATACGAGAT\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Executing: pwd\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(b'/home/juan/Desktop/juan/bio/gliadinas\\n', b'')"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sys(\"pwd\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rm: cannot remove 'data/clean/*': No such file or directory\n"
     ]
    }
   ],
   "source": [
    "!rm -r data/clean/*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar434-2019_S86_L001_R1_001.fastq.gz data/reads/sar434-2019_S86_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar434-2019_S86_L001_R1_001.fastq.gz data/reads/sar434-2019_S86_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar434-2019_S86_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar434-2019_S86_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar434-2019_S86_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar434-2019_S86_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar434-2019_S86_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (33 us/read; 1.80 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     736\\nReads with adapters:                       594 (80.7%)\\nReads written (passing filters):           736 (100.0%)\\n\\nTotal basepairs processed:       184,736 bp\\nQuality-trimmed:                  16,646 bp (9.0%)\\nTotal written (filtered):        167,205 bp (90.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 594 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 34.5%\\n  C: 20.5%\\n  G: 41.1%\\n  T: 3.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t313\\t184.0\\t0\\t313\\n2\\t272\\t46.0\\t0\\t272\\n3\\t8\\t11.5\\t0\\t8\\n4\\t1\\t2.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar434-2019_S86_L001_R1_001.fastq.gz\\n=============================================\\n736 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar434-2019_S86_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar434-2019_S86_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar434-2019_S86_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar434-2019_S86_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar434-2019_S86_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (30 us/read; 1.99 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     736\\nReads with adapters:                        40 (5.4%)\\nReads written (passing filters):           736 (100.0%)\\n\\nTotal basepairs processed:       184,736 bp\\nQuality-trimmed:                  27,408 bp (14.8%)\\nTotal written (filtered):        157,283 bp (85.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 40 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.5%\\n  C: 27.5%\\n  G: 32.5%\\n  T: 17.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t36\\t184.0\\t0\\t36\\n2\\t3\\t46.0\\t0\\t3\\n3\\t1\\t11.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar434-2019_S86_L001_R2_001.fastq.gz\\n=============================================\\n736 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar434-2019_S86_L001_R1_001_trimmed.fq.gz and sar434-2019_S86_L001_R2_001_trimmed.fq.gz\\nfile_1: sar434-2019_S86_L001_R1_001_trimmed.fq.gz, file_2: sar434-2019_S86_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar434-2019_S86_L001_R1_001_trimmed.fq.gz and sar434-2019_S86_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar434-2019_S86_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar434-2019_S86_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 736\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 45 (6.11%)\\n\\n\\n  >>> Now running FastQC on the validated data sar434-2019_S86_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar434-2019_S86_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar434-2019_S86_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar434-2019_S86_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar434-2019_S86_L001_R1_001_trimmed.fq.gz and sar434-2019_S86_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar435-2019_S87_L001_R1_001.fastq.gz data/reads/sar435-2019_S87_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar435-2019_S87_L001_R1_001.fastq.gz data/reads/sar435-2019_S87_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar435-2019_S87_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar435-2019_S87_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar435-2019_S87_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar435-2019_S87_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar435-2019_S87_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.85 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     906\\nReads with adapters:                       670 (74.0%)\\nReads written (passing filters):           906 (100.0%)\\n\\nTotal basepairs processed:       227,406 bp\\nQuality-trimmed:                  24,507 bp (10.8%)\\nTotal written (filtered):        201,903 bp (88.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 670 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.4%\\n  C: 16.9%\\n  G: 35.2%\\n  T: 5.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t351\\t226.5\\t0\\t351\\n2\\t312\\t56.6\\t0\\t312\\n3\\t7\\t14.2\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar435-2019_S87_L001_R1_001.fastq.gz\\n=============================================\\n906 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar435-2019_S87_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar435-2019_S87_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar435-2019_S87_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar435-2019_S87_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar435-2019_S87_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (41 us/read; 1.47 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     906\\nReads with adapters:                        57 (6.3%)\\nReads written (passing filters):           906 (100.0%)\\n\\nTotal basepairs processed:       227,406 bp\\nQuality-trimmed:                  48,157 bp (21.2%)\\nTotal written (filtered):        179,183 bp (78.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 57 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 14.0%\\n  C: 24.6%\\n  G: 35.1%\\n  T: 24.6%\\n  none/other: 1.8%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t49\\t226.5\\t0\\t49\\n2\\t7\\t56.6\\t0\\t7\\n3\\t1\\t14.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar435-2019_S87_L001_R2_001.fastq.gz\\n=============================================\\n906 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar435-2019_S87_L001_R1_001_trimmed.fq.gz and sar435-2019_S87_L001_R2_001_trimmed.fq.gz\\nfile_1: sar435-2019_S87_L001_R1_001_trimmed.fq.gz, file_2: sar435-2019_S87_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar435-2019_S87_L001_R1_001_trimmed.fq.gz and sar435-2019_S87_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar435-2019_S87_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar435-2019_S87_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 906\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 92 (10.15%)\\n\\n\\n  >>> Now running FastQC on the validated data sar435-2019_S87_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar435-2019_S87_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar435-2019_S87_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar435-2019_S87_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar435-2019_S87_L001_R1_001_trimmed.fq.gz and sar435-2019_S87_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar436-2019_S88_L001_R1_001.fastq.gz data/reads/sar436-2019_S88_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar436-2019_S88_L001_R1_001.fastq.gz data/reads/sar436-2019_S88_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar436-2019_S88_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar436-2019_S88_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar436-2019_S88_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar436-2019_S88_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar436-2019_S88_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (40 us/read; 1.51 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     724\\nReads with adapters:                       507 (70.0%)\\nReads written (passing filters):           724 (100.0%)\\n\\nTotal basepairs processed:       181,724 bp\\nQuality-trimmed:                  17,047 bp (9.4%)\\nTotal written (filtered):        163,890 bp (90.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 507 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.6%\\n  C: 17.9%\\n  G: 36.5%\\n  T: 4.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t240\\t181.0\\t0\\t240\\n2\\t256\\t45.2\\t0\\t256\\n3\\t10\\t11.3\\t0\\t10\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar436-2019_S88_L001_R1_001.fastq.gz\\n=============================================\\n724 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar436-2019_S88_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar436-2019_S88_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar436-2019_S88_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar436-2019_S88_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar436-2019_S88_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (41 us/read; 1.48 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     724\\nReads with adapters:                        59 (8.1%)\\nReads written (passing filters):           724 (100.0%)\\n\\nTotal basepairs processed:       181,724 bp\\nQuality-trimmed:                  30,716 bp (16.9%)\\nTotal written (filtered):        150,932 bp (83.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 59 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.0%\\n  C: 40.7%\\n  G: 18.6%\\n  T: 18.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t48\\t181.0\\t0\\t48\\n2\\t7\\t45.2\\t0\\t7\\n3\\t2\\t11.3\\t0\\t2\\n4\\t2\\t2.8\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar436-2019_S88_L001_R2_001.fastq.gz\\n=============================================\\n724 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar436-2019_S88_L001_R1_001_trimmed.fq.gz and sar436-2019_S88_L001_R2_001_trimmed.fq.gz\\nfile_1: sar436-2019_S88_L001_R1_001_trimmed.fq.gz, file_2: sar436-2019_S88_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar436-2019_S88_L001_R1_001_trimmed.fq.gz and sar436-2019_S88_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar436-2019_S88_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar436-2019_S88_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 724\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 43 (5.94%)\\n\\n\\n  >>> Now running FastQC on the validated data sar436-2019_S88_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar436-2019_S88_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar436-2019_S88_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar436-2019_S88_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar436-2019_S88_L001_R1_001_trimmed.fq.gz and sar436-2019_S88_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar437-2019_S89_L001_R1_001.fastq.gz data/reads/sar437-2019_S89_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar437-2019_S89_L001_R1_001.fastq.gz data/reads/sar437-2019_S89_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar437-2019_S89_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar437-2019_S89_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar437-2019_S89_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar437-2019_S89_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar437-2019_S89_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (33 us/read; 1.81 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     503\\nReads with adapters:                       340 (67.6%)\\nReads written (passing filters):           503 (100.0%)\\n\\nTotal basepairs processed:       126,253 bp\\nQuality-trimmed:                  22,682 bp (18.0%)\\nTotal written (filtered):        103,060 bp (81.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 340 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.6%\\n  C: 15.0%\\n  G: 39.7%\\n  T: 4.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t175\\t125.8\\t0\\t175\\n2\\t159\\t31.4\\t0\\t159\\n3\\t6\\t7.9\\t0\\t6\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar437-2019_S89_L001_R1_001.fastq.gz\\n=============================================\\n503 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar437-2019_S89_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar437-2019_S89_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar437-2019_S89_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar437-2019_S89_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar437-2019_S89_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.01 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     503\\nReads with adapters:                        41 (8.2%)\\nReads written (passing filters):           503 (100.0%)\\n\\nTotal basepairs processed:       126,253 bp\\nQuality-trimmed:                  33,037 bp (26.2%)\\nTotal written (filtered):         93,167 bp (73.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 41 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 12.2%\\n  C: 31.7%\\n  G: 22.0%\\n  T: 34.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t33\\t125.8\\t0\\t33\\n2\\t8\\t31.4\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar437-2019_S89_L001_R2_001.fastq.gz\\n=============================================\\n503 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar437-2019_S89_L001_R1_001_trimmed.fq.gz and sar437-2019_S89_L001_R2_001_trimmed.fq.gz\\nfile_1: sar437-2019_S89_L001_R1_001_trimmed.fq.gz, file_2: sar437-2019_S89_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar437-2019_S89_L001_R1_001_trimmed.fq.gz and sar437-2019_S89_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar437-2019_S89_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar437-2019_S89_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 503\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 44 (8.75%)\\n\\n\\n  >>> Now running FastQC on the validated data sar437-2019_S89_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar437-2019_S89_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar437-2019_S89_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar437-2019_S89_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar437-2019_S89_L001_R1_001_trimmed.fq.gz and sar437-2019_S89_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar438-2019_S90_L001_R1_001.fastq.gz data/reads/sar438-2019_S90_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar438-2019_S90_L001_R1_001.fastq.gz data/reads/sar438-2019_S90_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar438-2019_S90_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar438-2019_S90_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar438-2019_S90_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar438-2019_S90_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar438-2019_S90_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (50 us/read; 1.20 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     641\\nReads with adapters:                       423 (66.0%)\\nReads written (passing filters):           641 (100.0%)\\n\\nTotal basepairs processed:       160,891 bp\\nQuality-trimmed:                  29,672 bp (18.4%)\\nTotal written (filtered):        130,586 bp (81.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 423 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.5%\\n  C: 18.2%\\n  G: 39.0%\\n  T: 7.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t224\\t160.2\\t0\\t224\\n2\\t188\\t40.1\\t0\\t188\\n3\\t11\\t10.0\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar438-2019_S90_L001_R1_001.fastq.gz\\n=============================================\\n641 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar438-2019_S90_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar438-2019_S90_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar438-2019_S90_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar438-2019_S90_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar438-2019_S90_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (32 us/read; 1.89 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     641\\nReads with adapters:                        57 (8.9%)\\nReads written (passing filters):           641 (100.0%)\\n\\nTotal basepairs processed:       160,891 bp\\nQuality-trimmed:                  40,873 bp (25.4%)\\nTotal written (filtered):        119,950 bp (74.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 57 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 31.6%\\n  C: 26.3%\\n  G: 17.5%\\n  T: 24.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t47\\t160.2\\t0\\t47\\n2\\t9\\t40.1\\t0\\t9\\n3\\t1\\t10.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar438-2019_S90_L001_R2_001.fastq.gz\\n=============================================\\n641 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar438-2019_S90_L001_R1_001_trimmed.fq.gz and sar438-2019_S90_L001_R2_001_trimmed.fq.gz\\nfile_1: sar438-2019_S90_L001_R1_001_trimmed.fq.gz, file_2: sar438-2019_S90_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar438-2019_S90_L001_R1_001_trimmed.fq.gz and sar438-2019_S90_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar438-2019_S90_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar438-2019_S90_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 641\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 61 (9.52%)\\n\\n\\n  >>> Now running FastQC on the validated data sar438-2019_S90_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar438-2019_S90_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar438-2019_S90_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar438-2019_S90_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar438-2019_S90_L001_R1_001_trimmed.fq.gz and sar438-2019_S90_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar439-2019_S91_L001_R1_001.fastq.gz data/reads/sar439-2019_S91_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar439-2019_S91_L001_R1_001.fastq.gz data/reads/sar439-2019_S91_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar439-2019_S91_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar439-2019_S91_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar439-2019_S91_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar439-2019_S91_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar439-2019_S91_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 2.01 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,275\\nReads with adapters:                       933 (73.2%)\\nReads written (passing filters):         1,275 (100.0%)\\n\\nTotal basepairs processed:       320,025 bp\\nQuality-trimmed:                  35,594 bp (11.1%)\\nTotal written (filtered):        282,989 bp (88.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 933 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 30.2%\\n  C: 15.0%\\n  G: 49.5%\\n  T: 5.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t444\\t318.8\\t0\\t444\\n2\\t470\\t79.7\\t0\\t470\\n3\\t18\\t19.9\\t0\\t18\\n4\\t1\\t5.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar439-2019_S91_L001_R1_001.fastq.gz\\n=============================================\\n1275 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar439-2019_S91_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar439-2019_S91_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar439-2019_S91_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar439-2019_S91_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar439-2019_S91_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (29 us/read; 2.10 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,275\\nReads with adapters:                        89 (7.0%)\\nReads written (passing filters):         1,275 (100.0%)\\n\\nTotal basepairs processed:       320,025 bp\\nQuality-trimmed:                  57,544 bp (18.0%)\\nTotal written (filtered):        262,378 bp (82.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 89 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.6%\\n  C: 22.5%\\n  G: 31.5%\\n  T: 22.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t79\\t318.8\\t0\\t79\\n2\\t7\\t79.7\\t0\\t7\\n3\\t2\\t19.9\\t0\\t2\\n4\\t1\\t5.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar439-2019_S91_L001_R2_001.fastq.gz\\n=============================================\\n1275 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar439-2019_S91_L001_R1_001_trimmed.fq.gz and sar439-2019_S91_L001_R2_001_trimmed.fq.gz\\nfile_1: sar439-2019_S91_L001_R1_001_trimmed.fq.gz, file_2: sar439-2019_S91_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar439-2019_S91_L001_R1_001_trimmed.fq.gz and sar439-2019_S91_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar439-2019_S91_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar439-2019_S91_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1275\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 89 (6.98%)\\n\\n\\n  >>> Now running FastQC on the validated data sar439-2019_S91_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar439-2019_S91_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar439-2019_S91_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar439-2019_S91_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar439-2019_S91_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar439-2019_S91_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar439-2019_S91_L001_R1_001_trimmed.fq.gz and sar439-2019_S91_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar440-2019_S92_L001_R1_001.fastq.gz data/reads/sar440-2019_S92_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar440-2019_S92_L001_R1_001.fastq.gz data/reads/sar440-2019_S92_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nInput file 'data/reads/sar440-2019_S92_L001_R1_001.fastq.gz' seems to be completely empty. Consider respecifying!\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar441-2019_S93_L001_R1_001.fastq.gz data/reads/sar441-2019_S93_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar441-2019_S93_L001_R1_001.fastq.gz data/reads/sar441-2019_S93_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar441-2019_S93_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar441-2019_S93_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar441-2019_S93_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar441-2019_S93_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar441-2019_S93_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (41 us/read; 1.47 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     661\\nReads with adapters:                       437 (66.1%)\\nReads written (passing filters):           661 (100.0%)\\n\\nTotal basepairs processed:       165,911 bp\\nQuality-trimmed:                  27,978 bp (16.9%)\\nTotal written (filtered):        137,297 bp (82.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 437 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.2%\\n  C: 20.6%\\n  G: 38.9%\\n  T: 5.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t246\\t165.2\\t0\\t246\\n2\\t183\\t41.3\\t0\\t183\\n3\\t8\\t10.3\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar441-2019_S93_L001_R1_001.fastq.gz\\n=============================================\\n661 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar441-2019_S93_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar441-2019_S93_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar441-2019_S93_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar441-2019_S93_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar441-2019_S93_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (28 us/read; 2.13 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     661\\nReads with adapters:                        61 (9.2%)\\nReads written (passing filters):           661 (100.0%)\\n\\nTotal basepairs processed:       165,911 bp\\nQuality-trimmed:                  45,445 bp (27.4%)\\nTotal written (filtered):        120,383 bp (72.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 61 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.7%\\n  C: 18.0%\\n  G: 31.1%\\n  T: 31.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t44\\t165.2\\t0\\t44\\n2\\t13\\t41.3\\t0\\t13\\n3\\t3\\t10.3\\t0\\t3\\n4\\t1\\t2.6\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar441-2019_S93_L001_R2_001.fastq.gz\\n=============================================\\n661 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar441-2019_S93_L001_R1_001_trimmed.fq.gz and sar441-2019_S93_L001_R2_001_trimmed.fq.gz\\nfile_1: sar441-2019_S93_L001_R1_001_trimmed.fq.gz, file_2: sar441-2019_S93_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar441-2019_S93_L001_R1_001_trimmed.fq.gz and sar441-2019_S93_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar441-2019_S93_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar441-2019_S93_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 661\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 68 (10.29%)\\n\\n\\n  >>> Now running FastQC on the validated data sar441-2019_S93_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar441-2019_S93_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar441-2019_S93_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar441-2019_S93_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar441-2019_S93_L001_R1_001_trimmed.fq.gz and sar441-2019_S93_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar442-2019_S94_L001_R1_001.fastq.gz data/reads/sar442-2019_S94_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar442-2019_S94_L001_R1_001.fastq.gz data/reads/sar442-2019_S94_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar442-2019_S94_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar442-2019_S94_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar442-2019_S94_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar442-2019_S94_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar442-2019_S94_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (41 us/read; 1.47 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,093\\nReads with adapters:                       906 (82.9%)\\nReads written (passing filters):         1,093 (100.0%)\\n\\nTotal basepairs processed:       274,343 bp\\nQuality-trimmed:                  20,516 bp (7.5%)\\nTotal written (filtered):        252,454 bp (92.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 906 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.6%\\n  C: 16.7%\\n  G: 40.3%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t448\\t273.2\\t0\\t448\\n2\\t449\\t68.3\\t0\\t449\\n3\\t9\\t17.1\\t0\\t9\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar442-2019_S94_L001_R1_001.fastq.gz\\n=============================================\\n1093 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar442-2019_S94_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar442-2019_S94_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar442-2019_S94_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar442-2019_S94_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar442-2019_S94_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.42 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,093\\nReads with adapters:                        68 (6.2%)\\nReads written (passing filters):         1,093 (100.0%)\\n\\nTotal basepairs processed:       274,343 bp\\nQuality-trimmed:                  37,934 bp (13.8%)\\nTotal written (filtered):        236,324 bp (86.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 68 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.5%\\n  C: 20.6%\\n  G: 36.8%\\n  T: 17.6%\\n  none/other: 1.5%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t56\\t273.2\\t0\\t56\\n2\\t8\\t68.3\\t0\\t8\\n3\\t3\\t17.1\\t0\\t3\\n4\\t1\\t4.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar442-2019_S94_L001_R2_001.fastq.gz\\n=============================================\\n1093 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar442-2019_S94_L001_R1_001_trimmed.fq.gz and sar442-2019_S94_L001_R2_001_trimmed.fq.gz\\nfile_1: sar442-2019_S94_L001_R1_001_trimmed.fq.gz, file_2: sar442-2019_S94_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar442-2019_S94_L001_R1_001_trimmed.fq.gz and sar442-2019_S94_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar442-2019_S94_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar442-2019_S94_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1093\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 57 (5.22%)\\n\\n\\n  >>> Now running FastQC on the validated data sar442-2019_S94_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar442-2019_S94_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar442-2019_S94_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar442-2019_S94_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar442-2019_S94_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar442-2019_S94_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar442-2019_S94_L001_R1_001_trimmed.fq.gz and sar442-2019_S94_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar443-2019_S95_L001_R1_001.fastq.gz data/reads/sar443-2019_S95_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar443-2019_S95_L001_R1_001.fastq.gz data/reads/sar443-2019_S95_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar443-2019_S95_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar443-2019_S95_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar443-2019_S95_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar443-2019_S95_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar443-2019_S95_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.92 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     885\\nReads with adapters:                       619 (69.9%)\\nReads written (passing filters):           885 (100.0%)\\n\\nTotal basepairs processed:       222,135 bp\\nQuality-trimmed:                  30,489 bp (13.7%)\\nTotal written (filtered):        190,724 bp (85.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 619 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 36.0%\\n  C: 20.7%\\n  G: 37.5%\\n  T: 5.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t327\\t221.2\\t0\\t327\\n2\\t281\\t55.3\\t0\\t281\\n3\\t11\\t13.8\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar443-2019_S95_L001_R1_001.fastq.gz\\n=============================================\\n885 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar443-2019_S95_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar443-2019_S95_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar443-2019_S95_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar443-2019_S95_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar443-2019_S95_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (52 us/read; 1.15 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     885\\nReads with adapters:                        69 (7.8%)\\nReads written (passing filters):           885 (100.0%)\\n\\nTotal basepairs processed:       222,135 bp\\nQuality-trimmed:                  49,407 bp (22.2%)\\nTotal written (filtered):        172,645 bp (77.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 69 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.5%\\n  C: 26.1%\\n  G: 21.7%\\n  T: 21.7%\\n  none/other: 2.9%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t59\\t221.2\\t0\\t59\\n2\\t7\\t55.3\\t0\\t7\\n3\\t2\\t13.8\\t0\\t2\\n4\\t1\\t3.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar443-2019_S95_L001_R2_001.fastq.gz\\n=============================================\\n885 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar443-2019_S95_L001_R1_001_trimmed.fq.gz and sar443-2019_S95_L001_R2_001_trimmed.fq.gz\\nfile_1: sar443-2019_S95_L001_R1_001_trimmed.fq.gz, file_2: sar443-2019_S95_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar443-2019_S95_L001_R1_001_trimmed.fq.gz and sar443-2019_S95_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar443-2019_S95_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar443-2019_S95_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 885\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 75 (8.47%)\\n\\n\\n  >>> Now running FastQC on the validated data sar443-2019_S95_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar443-2019_S95_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar443-2019_S95_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar443-2019_S95_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar443-2019_S95_L001_R1_001_trimmed.fq.gz and sar443-2019_S95_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar444-2019_S96_L001_R1_001.fastq.gz data/reads/sar444-2019_S96_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar444-2019_S96_L001_R1_001.fastq.gz data/reads/sar444-2019_S96_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar444-2019_S96_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar444-2019_S96_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar444-2019_S96_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar444-2019_S96_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar444-2019_S96_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,363\\nReads with adapters:                     1,007 (73.9%)\\nReads written (passing filters):         1,363 (100.0%)\\n\\nTotal basepairs processed:       342,113 bp\\nQuality-trimmed:                  37,745 bp (11.0%)\\nTotal written (filtered):        302,773 bp (88.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1007 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.1%\\n  C: 11.4%\\n  G: 54.1%\\n  T: 5.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t434\\t340.8\\t0\\t434\\n2\\t559\\t85.2\\t0\\t559\\n3\\t13\\t21.3\\t0\\t13\\n4\\t1\\t5.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar444-2019_S96_L001_R1_001.fastq.gz\\n=============================================\\n1363 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar444-2019_S96_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar444-2019_S96_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar444-2019_S96_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar444-2019_S96_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar444-2019_S96_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.40 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,363\\nReads with adapters:                       118 (8.7%)\\nReads written (passing filters):         1,363 (100.0%)\\n\\nTotal basepairs processed:       342,113 bp\\nQuality-trimmed:                  67,392 bp (19.7%)\\nTotal written (filtered):        274,572 bp (80.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 118 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.6%\\n  C: 20.3%\\n  G: 20.3%\\n  T: 33.9%\\n  none/other: 0.8%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t94\\t340.8\\t0\\t94\\n2\\t18\\t85.2\\t0\\t18\\n3\\t5\\t21.3\\t0\\t5\\n4\\t1\\t5.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar444-2019_S96_L001_R2_001.fastq.gz\\n=============================================\\n1363 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar444-2019_S96_L001_R1_001_trimmed.fq.gz and sar444-2019_S96_L001_R2_001_trimmed.fq.gz\\nfile_1: sar444-2019_S96_L001_R1_001_trimmed.fq.gz, file_2: sar444-2019_S96_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar444-2019_S96_L001_R1_001_trimmed.fq.gz and sar444-2019_S96_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar444-2019_S96_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar444-2019_S96_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1363\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 113 (8.29%)\\n\\n\\n  >>> Now running FastQC on the validated data sar444-2019_S96_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar444-2019_S96_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar444-2019_S96_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar444-2019_S96_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar444-2019_S96_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar444-2019_S96_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar444-2019_S96_L001_R1_001_trimmed.fq.gz and sar444-2019_S96_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar445-2019_S97_L001_R1_001.fastq.gz data/reads/sar445-2019_S97_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar445-2019_S97_L001_R1_001.fastq.gz data/reads/sar445-2019_S97_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar445-2019_S97_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar445-2019_S97_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar445-2019_S97_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar445-2019_S97_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar445-2019_S97_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (37 us/read; 1.64 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,219\\nReads with adapters:                       909 (74.6%)\\nReads written (passing filters):         1,219 (100.0%)\\n\\nTotal basepairs processed:       305,969 bp\\nQuality-trimmed:                  31,577 bp (10.3%)\\nTotal written (filtered):        272,932 bp (89.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 909 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.7%\\n  C: 11.2%\\n  G: 56.5%\\n  T: 5.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t375\\t304.8\\t0\\t375\\n2\\t520\\t76.2\\t0\\t520\\n3\\t11\\t19.0\\t0\\t11\\n4\\t3\\t4.8\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar445-2019_S97_L001_R1_001.fastq.gz\\n=============================================\\n1219 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar445-2019_S97_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar445-2019_S97_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar445-2019_S97_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar445-2019_S97_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar445-2019_S97_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.36 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,219\\nReads with adapters:                        98 (8.0%)\\nReads written (passing filters):         1,219 (100.0%)\\n\\nTotal basepairs processed:       305,969 bp\\nQuality-trimmed:                  48,307 bp (15.8%)\\nTotal written (filtered):        257,555 bp (84.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 98 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.5%\\n  C: 28.6%\\n  G: 23.5%\\n  T: 24.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t91\\t304.8\\t0\\t91\\n2\\t6\\t76.2\\t0\\t6\\n4\\t1\\t4.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar445-2019_S97_L001_R2_001.fastq.gz\\n=============================================\\n1219 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar445-2019_S97_L001_R1_001_trimmed.fq.gz and sar445-2019_S97_L001_R2_001_trimmed.fq.gz\\nfile_1: sar445-2019_S97_L001_R1_001_trimmed.fq.gz, file_2: sar445-2019_S97_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar445-2019_S97_L001_R1_001_trimmed.fq.gz and sar445-2019_S97_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar445-2019_S97_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar445-2019_S97_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1219\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 41 (3.36%)\\n\\n\\n  >>> Now running FastQC on the validated data sar445-2019_S97_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar445-2019_S97_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar445-2019_S97_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar445-2019_S97_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar445-2019_S97_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar445-2019_S97_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar445-2019_S97_L001_R1_001_trimmed.fq.gz and sar445-2019_S97_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar446-2019_S98_L001_R1_001.fastq.gz data/reads/sar446-2019_S98_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar446-2019_S98_L001_R1_001.fastq.gz data/reads/sar446-2019_S98_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar446-2019_S98_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar446-2019_S98_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar446-2019_S98_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar446-2019_S98_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar446-2019_S98_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (43 us/read; 1.39 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,099\\nReads with adapters:                       833 (75.8%)\\nReads written (passing filters):         1,099 (100.0%)\\n\\nTotal basepairs processed:       275,849 bp\\nQuality-trimmed:                  25,630 bp (9.3%)\\nTotal written (filtered):        248,903 bp (90.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 833 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.3%\\n  C: 13.3%\\n  G: 53.7%\\n  T: 3.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t365\\t274.8\\t0\\t365\\n2\\t453\\t68.7\\t0\\t453\\n3\\t15\\t17.2\\t0\\t15\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar446-2019_S98_L001_R1_001.fastq.gz\\n=============================================\\n1099 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar446-2019_S98_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar446-2019_S98_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar446-2019_S98_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar446-2019_S98_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar446-2019_S98_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (33 us/read; 1.84 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,099\\nReads with adapters:                        77 (7.0%)\\nReads written (passing filters):         1,099 (100.0%)\\n\\nTotal basepairs processed:       275,849 bp\\nQuality-trimmed:                  46,477 bp (16.8%)\\nTotal written (filtered):        229,285 bp (83.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 77 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 11.7%\\n  C: 36.4%\\n  G: 31.2%\\n  T: 20.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t71\\t274.8\\t0\\t71\\n2\\t3\\t68.7\\t0\\t3\\n3\\t2\\t17.2\\t0\\t2\\n4\\t1\\t4.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar446-2019_S98_L001_R2_001.fastq.gz\\n=============================================\\n1099 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar446-2019_S98_L001_R1_001_trimmed.fq.gz and sar446-2019_S98_L001_R2_001_trimmed.fq.gz\\nfile_1: sar446-2019_S98_L001_R1_001_trimmed.fq.gz, file_2: sar446-2019_S98_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar446-2019_S98_L001_R1_001_trimmed.fq.gz and sar446-2019_S98_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar446-2019_S98_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar446-2019_S98_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1099\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 76 (6.92%)\\n\\n\\n  >>> Now running FastQC on the validated data sar446-2019_S98_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar446-2019_S98_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar446-2019_S98_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar446-2019_S98_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar446-2019_S98_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar446-2019_S98_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar446-2019_S98_L001_R1_001_trimmed.fq.gz and sar446-2019_S98_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar447-2019_S99_L001_R1_001.fastq.gz data/reads/sar447-2019_S99_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar447-2019_S99_L001_R1_001.fastq.gz data/reads/sar447-2019_S99_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar447-2019_S99_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar447-2019_S99_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar447-2019_S99_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar447-2019_S99_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar447-2019_S99_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,228\\nReads with adapters:                       791 (64.4%)\\nReads written (passing filters):         1,228 (100.0%)\\n\\nTotal basepairs processed:       308,228 bp\\nQuality-trimmed:                  47,241 bp (15.3%)\\nTotal written (filtered):        259,745 bp (84.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 791 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 30.0%\\n  C: 15.3%\\n  G: 49.2%\\n  T: 5.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t359\\t307.0\\t0\\t359\\n2\\t414\\t76.8\\t0\\t414\\n3\\t17\\t19.2\\t0\\t17\\n4\\t1\\t4.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar447-2019_S99_L001_R1_001.fastq.gz\\n=============================================\\n1228 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar447-2019_S99_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar447-2019_S99_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar447-2019_S99_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar447-2019_S99_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar447-2019_S99_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (29 us/read; 2.05 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,228\\nReads with adapters:                       112 (9.1%)\\nReads written (passing filters):         1,228 (100.0%)\\n\\nTotal basepairs processed:       308,228 bp\\nQuality-trimmed:                  76,998 bp (25.0%)\\nTotal written (filtered):        231,086 bp (75.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 112 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.7%\\n  C: 23.2%\\n  G: 17.9%\\n  T: 31.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t88\\t307.0\\t0\\t88\\n2\\t18\\t76.8\\t0\\t18\\n3\\t4\\t19.2\\t0\\t4\\n4\\t2\\t4.8\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar447-2019_S99_L001_R2_001.fastq.gz\\n=============================================\\n1228 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar447-2019_S99_L001_R1_001_trimmed.fq.gz and sar447-2019_S99_L001_R2_001_trimmed.fq.gz\\nfile_1: sar447-2019_S99_L001_R1_001_trimmed.fq.gz, file_2: sar447-2019_S99_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar447-2019_S99_L001_R1_001_trimmed.fq.gz and sar447-2019_S99_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar447-2019_S99_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar447-2019_S99_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1228\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 130 (10.59%)\\n\\n\\n  >>> Now running FastQC on the validated data sar447-2019_S99_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar447-2019_S99_L001_R1_001_val_1.fq.gz\\nApprox 90% complete for sar447-2019_S99_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar447-2019_S99_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar447-2019_S99_L001_R2_001_val_2.fq.gz\\nApprox 90% complete for sar447-2019_S99_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar447-2019_S99_L001_R1_001_trimmed.fq.gz and sar447-2019_S99_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar448-2019_S100_L001_R1_001.fastq.gz data/reads/sar448-2019_S100_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar448-2019_S100_L001_R1_001.fastq.gz data/reads/sar448-2019_S100_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar448-2019_S100_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar448-2019_S100_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar448-2019_S100_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar448-2019_S100_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar448-2019_S100_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.89 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     800\\nReads with adapters:                       551 (68.9%)\\nReads written (passing filters):           800 (100.0%)\\n\\nTotal basepairs processed:       200,800 bp\\nQuality-trimmed:                  27,988 bp (13.9%)\\nTotal written (filtered):        171,978 bp (85.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 551 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.1%\\n  C: 16.0%\\n  G: 35.0%\\n  T: 6.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t280\\t200.0\\t0\\t280\\n2\\t260\\t50.0\\t0\\t260\\n3\\t10\\t12.5\\t0\\t10\\n4\\t1\\t3.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar448-2019_S100_L001_R1_001.fastq.gz\\n=============================================\\n800 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar448-2019_S100_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar448-2019_S100_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar448-2019_S100_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar448-2019_S100_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar448-2019_S100_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (27 us/read; 2.24 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     800\\nReads with adapters:                        73 (9.1%)\\nReads written (passing filters):           800 (100.0%)\\n\\nTotal basepairs processed:       200,800 bp\\nQuality-trimmed:                  47,323 bp (23.6%)\\nTotal written (filtered):        153,389 bp (76.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 73 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 20.5%\\n  C: 26.0%\\n  G: 37.0%\\n  T: 16.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t62\\t200.0\\t0\\t62\\n2\\t10\\t50.0\\t0\\t10\\n6\\t1\\t0.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar448-2019_S100_L001_R2_001.fastq.gz\\n=============================================\\n800 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar448-2019_S100_L001_R1_001_trimmed.fq.gz and sar448-2019_S100_L001_R2_001_trimmed.fq.gz\\nfile_1: sar448-2019_S100_L001_R1_001_trimmed.fq.gz, file_2: sar448-2019_S100_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar448-2019_S100_L001_R1_001_trimmed.fq.gz and sar448-2019_S100_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar448-2019_S100_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar448-2019_S100_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 800\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 83 (10.38%)\\n\\n\\n  >>> Now running FastQC on the validated data sar448-2019_S100_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar448-2019_S100_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar448-2019_S100_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar448-2019_S100_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar448-2019_S100_L001_R1_001_trimmed.fq.gz and sar448-2019_S100_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar449-2019_S101_L001_R1_001.fastq.gz data/reads/sar449-2019_S101_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar449-2019_S101_L001_R1_001.fastq.gz data/reads/sar449-2019_S101_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar449-2019_S101_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar449-2019_S101_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar449-2019_S101_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar449-2019_S101_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar449-2019_S101_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,221\\nReads with adapters:                       834 (68.3%)\\nReads written (passing filters):         1,221 (100.0%)\\n\\nTotal basepairs processed:       306,471 bp\\nQuality-trimmed:                  35,031 bp (11.4%)\\nTotal written (filtered):        270,146 bp (88.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 834 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.6%\\n  C: 15.1%\\n  G: 52.6%\\n  T: 5.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t395\\t305.2\\t0\\t395\\n2\\t420\\t76.3\\t0\\t420\\n3\\t17\\t19.1\\t0\\t17\\n4\\t2\\t4.8\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar449-2019_S101_L001_R1_001.fastq.gz\\n=============================================\\n1221 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar449-2019_S101_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar449-2019_S101_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar449-2019_S101_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar449-2019_S101_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar449-2019_S101_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (27 us/read; 2.22 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,221\\nReads with adapters:                       110 (9.0%)\\nReads written (passing filters):         1,221 (100.0%)\\n\\nTotal basepairs processed:       306,471 bp\\nQuality-trimmed:                  60,685 bp (19.8%)\\nTotal written (filtered):        245,652 bp (80.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 110 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 17.3%\\n  C: 29.1%\\n  G: 26.4%\\n  T: 27.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t88\\t305.2\\t0\\t88\\n2\\t20\\t76.3\\t0\\t20\\n3\\t2\\t19.1\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar449-2019_S101_L001_R2_001.fastq.gz\\n=============================================\\n1221 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar449-2019_S101_L001_R1_001_trimmed.fq.gz and sar449-2019_S101_L001_R2_001_trimmed.fq.gz\\nfile_1: sar449-2019_S101_L001_R1_001_trimmed.fq.gz, file_2: sar449-2019_S101_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar449-2019_S101_L001_R1_001_trimmed.fq.gz and sar449-2019_S101_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar449-2019_S101_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar449-2019_S101_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1221\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 91 (7.45%)\\n\\n\\n  >>> Now running FastQC on the validated data sar449-2019_S101_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar449-2019_S101_L001_R1_001_val_1.fq.gz\\nApprox 85% complete for sar449-2019_S101_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar449-2019_S101_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar449-2019_S101_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar449-2019_S101_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar449-2019_S101_L001_R1_001_trimmed.fq.gz and sar449-2019_S101_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar450-2019_S102_L001_R1_001.fastq.gz data/reads/sar450-2019_S102_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar450-2019_S102_L001_R1_001.fastq.gz data/reads/sar450-2019_S102_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar450-2019_S102_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar450-2019_S102_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar450-2019_S102_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar450-2019_S102_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar450-2019_S102_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (33 us/read; 1.81 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     948\\nReads with adapters:                       680 (71.7%)\\nReads written (passing filters):           948 (100.0%)\\n\\nTotal basepairs processed:       237,948 bp\\nQuality-trimmed:                  29,146 bp (12.2%)\\nTotal written (filtered):        207,721 bp (87.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 680 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.7%\\n  C: 13.5%\\n  G: 50.9%\\n  T: 5.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t295\\t237.0\\t0\\t295\\n2\\t369\\t59.2\\t0\\t369\\n3\\t16\\t14.8\\t0\\t16\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar450-2019_S102_L001_R1_001.fastq.gz\\n=============================================\\n948 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar450-2019_S102_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar450-2019_S102_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar450-2019_S102_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar450-2019_S102_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar450-2019_S102_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (25 us/read; 2.38 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     948\\nReads with adapters:                        61 (6.4%)\\nReads written (passing filters):           948 (100.0%)\\n\\nTotal basepairs processed:       237,948 bp\\nQuality-trimmed:                  48,275 bp (20.3%)\\nTotal written (filtered):        189,601 bp (79.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 61 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.7%\\n  C: 26.2%\\n  G: 32.8%\\n  T: 21.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t53\\t237.0\\t0\\t53\\n2\\t6\\t59.2\\t0\\t6\\n3\\t1\\t14.8\\t0\\t1\\n4\\t1\\t3.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar450-2019_S102_L001_R2_001.fastq.gz\\n=============================================\\n948 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar450-2019_S102_L001_R1_001_trimmed.fq.gz and sar450-2019_S102_L001_R2_001_trimmed.fq.gz\\nfile_1: sar450-2019_S102_L001_R1_001_trimmed.fq.gz, file_2: sar450-2019_S102_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar450-2019_S102_L001_R1_001_trimmed.fq.gz and sar450-2019_S102_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar450-2019_S102_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar450-2019_S102_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 948\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 84 (8.86%)\\n\\n\\n  >>> Now running FastQC on the validated data sar450-2019_S102_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar450-2019_S102_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar450-2019_S102_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar450-2019_S102_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar450-2019_S102_L001_R1_001_trimmed.fq.gz and sar450-2019_S102_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar451-2019_S103_L001_R1_001.fastq.gz data/reads/sar451-2019_S103_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar451-2019_S103_L001_R1_001.fastq.gz data/reads/sar451-2019_S103_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar451-2019_S103_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar451-2019_S103_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar451-2019_S103_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar451-2019_S103_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar451-2019_S103_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (33 us/read; 1.79 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,408\\nReads with adapters:                     1,012 (71.9%)\\nReads written (passing filters):         1,408 (100.0%)\\n\\nTotal basepairs processed:       353,408 bp\\nQuality-trimmed:                  44,263 bp (12.5%)\\nTotal written (filtered):        307,516 bp (87.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1012 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.0%\\n  C: 12.2%\\n  G: 54.0%\\n  T: 4.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t414\\t352.0\\t0\\t414\\n2\\t581\\t88.0\\t0\\t581\\n3\\t15\\t22.0\\t0\\t15\\n4\\t2\\t5.5\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar451-2019_S103_L001_R1_001.fastq.gz\\n=============================================\\n1408 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar451-2019_S103_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar451-2019_S103_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar451-2019_S103_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar451-2019_S103_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar451-2019_S103_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (25 us/read; 2.41 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,408\\nReads with adapters:                        99 (7.0%)\\nReads written (passing filters):         1,408 (100.0%)\\n\\nTotal basepairs processed:       353,408 bp\\nQuality-trimmed:                  68,124 bp (19.3%)\\nTotal written (filtered):        285,166 bp (80.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 99 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.2%\\n  C: 34.3%\\n  G: 31.3%\\n  T: 15.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t83\\t352.0\\t0\\t83\\n2\\t14\\t88.0\\t0\\t14\\n3\\t1\\t22.0\\t0\\t1\\n4\\t1\\t5.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar451-2019_S103_L001_R2_001.fastq.gz\\n=============================================\\n1408 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar451-2019_S103_L001_R1_001_trimmed.fq.gz and sar451-2019_S103_L001_R2_001_trimmed.fq.gz\\nfile_1: sar451-2019_S103_L001_R1_001_trimmed.fq.gz, file_2: sar451-2019_S103_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar451-2019_S103_L001_R1_001_trimmed.fq.gz and sar451-2019_S103_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar451-2019_S103_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar451-2019_S103_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1408\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 90 (6.39%)\\n\\n\\n  >>> Now running FastQC on the validated data sar451-2019_S103_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar451-2019_S103_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar451-2019_S103_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar451-2019_S103_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar451-2019_S103_L001_R2_001_val_2.fq.gz\\nApprox 70% complete for sar451-2019_S103_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar451-2019_S103_L001_R1_001_trimmed.fq.gz and sar451-2019_S103_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar452-2019_S104_L001_R1_001.fastq.gz data/reads/sar452-2019_S104_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar452-2019_S104_L001_R1_001.fastq.gz data/reads/sar452-2019_S104_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar452-2019_S104_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar452-2019_S104_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar452-2019_S104_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar452-2019_S104_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar452-2019_S104_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     883\\nReads with adapters:                       586 (66.4%)\\nReads written (passing filters):           883 (100.0%)\\n\\nTotal basepairs processed:       221,633 bp\\nQuality-trimmed:                  33,636 bp (15.2%)\\nTotal written (filtered):        187,074 bp (84.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 586 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.9%\\n  C: 11.9%\\n  G: 54.3%\\n  T: 7.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t256\\t220.8\\t0\\t256\\n2\\t323\\t55.2\\t0\\t323\\n3\\t7\\t13.8\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar452-2019_S104_L001_R1_001.fastq.gz\\n=============================================\\n883 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar452-2019_S104_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar452-2019_S104_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar452-2019_S104_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar452-2019_S104_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar452-2019_S104_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.30 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     883\\nReads with adapters:                        68 (7.7%)\\nReads written (passing filters):           883 (100.0%)\\n\\nTotal basepairs processed:       221,633 bp\\nQuality-trimmed:                  54,730 bp (24.7%)\\nTotal written (filtered):        166,824 bp (75.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 68 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.5%\\n  C: 22.1%\\n  G: 33.8%\\n  T: 17.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t60\\t220.8\\t0\\t60\\n2\\t5\\t55.2\\t0\\t5\\n3\\t3\\t13.8\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar452-2019_S104_L001_R2_001.fastq.gz\\n=============================================\\n883 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar452-2019_S104_L001_R1_001_trimmed.fq.gz and sar452-2019_S104_L001_R2_001_trimmed.fq.gz\\nfile_1: sar452-2019_S104_L001_R1_001_trimmed.fq.gz, file_2: sar452-2019_S104_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar452-2019_S104_L001_R1_001_trimmed.fq.gz and sar452-2019_S104_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar452-2019_S104_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar452-2019_S104_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 883\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 73 (8.27%)\\n\\n\\n  >>> Now running FastQC on the validated data sar452-2019_S104_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar452-2019_S104_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar452-2019_S104_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar452-2019_S104_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar452-2019_S104_L001_R1_001_trimmed.fq.gz and sar452-2019_S104_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar453-2019_S105_L001_R1_001.fastq.gz data/reads/sar453-2019_S105_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar453-2019_S105_L001_R1_001.fastq.gz data/reads/sar453-2019_S105_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar453-2019_S105_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar453-2019_S105_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar453-2019_S105_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar453-2019_S105_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar453-2019_S105_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (29 us/read; 2.03 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,537\\nReads with adapters:                     1,073 (69.8%)\\nReads written (passing filters):         1,537 (100.0%)\\n\\nTotal basepairs processed:       385,787 bp\\nQuality-trimmed:                  43,192 bp (11.2%)\\nTotal written (filtered):        340,880 bp (88.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1073 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.3%\\n  C: 13.8%\\n  G: 54.1%\\n  T: 6.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t457\\t384.2\\t0\\t457\\n2\\t594\\t96.1\\t0\\t594\\n3\\t20\\t24.0\\t0\\t20\\n4\\t1\\t6.0\\t0\\t1\\n6\\t1\\t0.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar453-2019_S105_L001_R1_001.fastq.gz\\n=============================================\\n1537 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar453-2019_S105_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar453-2019_S105_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar453-2019_S105_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar453-2019_S105_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar453-2019_S105_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (37 us/read; 1.60 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,537\\nReads with adapters:                       142 (9.2%)\\nReads written (passing filters):         1,537 (100.0%)\\n\\nTotal basepairs processed:       385,787 bp\\nQuality-trimmed:                  73,883 bp (19.2%)\\nTotal written (filtered):        311,733 bp (80.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 142 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.1%\\n  C: 21.8%\\n  G: 19.0%\\n  T: 38.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t121\\t384.2\\t0\\t121\\n2\\t15\\t96.1\\t0\\t15\\n3\\t4\\t24.0\\t0\\t4\\n4\\t2\\t6.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar453-2019_S105_L001_R2_001.fastq.gz\\n=============================================\\n1537 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar453-2019_S105_L001_R1_001_trimmed.fq.gz and sar453-2019_S105_L001_R2_001_trimmed.fq.gz\\nfile_1: sar453-2019_S105_L001_R1_001_trimmed.fq.gz, file_2: sar453-2019_S105_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar453-2019_S105_L001_R1_001_trimmed.fq.gz and sar453-2019_S105_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar453-2019_S105_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar453-2019_S105_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1537\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 114 (7.42%)\\n\\n\\n  >>> Now running FastQC on the validated data sar453-2019_S105_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar453-2019_S105_L001_R1_001_val_1.fq.gz\\nApprox 70% complete for sar453-2019_S105_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar453-2019_S105_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar453-2019_S105_L001_R2_001_val_2.fq.gz\\nApprox 70% complete for sar453-2019_S105_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar453-2019_S105_L001_R1_001_trimmed.fq.gz and sar453-2019_S105_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar454-2019_S106_L001_R1_001.fastq.gz data/reads/sar454-2019_S106_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar454-2019_S106_L001_R1_001.fastq.gz data/reads/sar454-2019_S106_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar454-2019_S106_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar454-2019_S106_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar454-2019_S106_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar454-2019_S106_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar454-2019_S106_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (33 us/read; 1.84 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,297\\nReads with adapters:                       993 (76.6%)\\nReads written (passing filters):         1,297 (100.0%)\\n\\nTotal basepairs processed:       325,547 bp\\nQuality-trimmed:                  29,052 bp (8.9%)\\nTotal written (filtered):        294,893 bp (90.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 993 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.7%\\n  C: 13.6%\\n  G: 53.8%\\n  T: 4.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t404\\t324.2\\t0\\t404\\n2\\t571\\t81.1\\t0\\t571\\n3\\t17\\t20.3\\t0\\t17\\n5\\t1\\t1.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar454-2019_S106_L001_R1_001.fastq.gz\\n=============================================\\n1297 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar454-2019_S106_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar454-2019_S106_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar454-2019_S106_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar454-2019_S106_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar454-2019_S106_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.38 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,297\\nReads with adapters:                        66 (5.1%)\\nReads written (passing filters):         1,297 (100.0%)\\n\\nTotal basepairs processed:       325,547 bp\\nQuality-trimmed:                  53,669 bp (16.5%)\\nTotal written (filtered):        271,803 bp (83.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 66 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.7%\\n  C: 25.8%\\n  G: 22.7%\\n  T: 31.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t58\\t324.2\\t0\\t58\\n2\\t7\\t81.1\\t0\\t7\\n3\\t1\\t20.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar454-2019_S106_L001_R2_001.fastq.gz\\n=============================================\\n1297 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar454-2019_S106_L001_R1_001_trimmed.fq.gz and sar454-2019_S106_L001_R2_001_trimmed.fq.gz\\nfile_1: sar454-2019_S106_L001_R1_001_trimmed.fq.gz, file_2: sar454-2019_S106_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar454-2019_S106_L001_R1_001_trimmed.fq.gz and sar454-2019_S106_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar454-2019_S106_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar454-2019_S106_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1297\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 104 (8.02%)\\n\\n\\n  >>> Now running FastQC on the validated data sar454-2019_S106_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar454-2019_S106_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar454-2019_S106_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar454-2019_S106_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar454-2019_S106_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar454-2019_S106_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar454-2019_S106_L001_R1_001_trimmed.fq.gz and sar454-2019_S106_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar455-2019_S107_L001_R1_001.fastq.gz data/reads/sar455-2019_S107_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar455-2019_S107_L001_R1_001.fastq.gz data/reads/sar455-2019_S107_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar455-2019_S107_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar455-2019_S107_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar455-2019_S107_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar455-2019_S107_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar455-2019_S107_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (31 us/read; 1.93 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     625\\nReads with adapters:                       403 (64.5%)\\nReads written (passing filters):           625 (100.0%)\\n\\nTotal basepairs processed:       156,875 bp\\nQuality-trimmed:                  25,269 bp (16.1%)\\nTotal written (filtered):        131,005 bp (83.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 403 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 36.2%\\n  C: 18.9%\\n  G: 36.7%\\n  T: 8.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t219\\t156.2\\t0\\t219\\n2\\t172\\t39.1\\t0\\t172\\n3\\t10\\t9.8\\t0\\t10\\n4\\t2\\t2.4\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar455-2019_S107_L001_R1_001.fastq.gz\\n=============================================\\n625 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar455-2019_S107_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar455-2019_S107_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar455-2019_S107_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar455-2019_S107_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar455-2019_S107_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (28 us/read; 2.16 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     625\\nReads with adapters:                        71 (11.4%)\\nReads written (passing filters):           625 (100.0%)\\n\\nTotal basepairs processed:       156,875 bp\\nQuality-trimmed:                  37,678 bp (24.0%)\\nTotal written (filtered):        119,104 bp (75.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 71 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.9%\\n  C: 18.3%\\n  G: 19.7%\\n  T: 38.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t54\\t156.2\\t0\\t54\\n2\\t12\\t39.1\\t0\\t12\\n3\\t5\\t9.8\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar455-2019_S107_L001_R2_001.fastq.gz\\n=============================================\\n625 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar455-2019_S107_L001_R1_001_trimmed.fq.gz and sar455-2019_S107_L001_R2_001_trimmed.fq.gz\\nfile_1: sar455-2019_S107_L001_R1_001_trimmed.fq.gz, file_2: sar455-2019_S107_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar455-2019_S107_L001_R1_001_trimmed.fq.gz and sar455-2019_S107_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar455-2019_S107_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar455-2019_S107_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 625\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 47 (7.52%)\\n\\n\\n  >>> Now running FastQC on the validated data sar455-2019_S107_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar455-2019_S107_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar455-2019_S107_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar455-2019_S107_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar455-2019_S107_L001_R1_001_trimmed.fq.gz and sar455-2019_S107_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar456-2019_S108_L001_R1_001.fastq.gz data/reads/sar456-2019_S108_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar456-2019_S108_L001_R1_001.fastq.gz data/reads/sar456-2019_S108_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar456-2019_S108_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar456-2019_S108_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar456-2019_S108_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar456-2019_S108_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar456-2019_S108_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (37 us/read; 1.62 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     573\\nReads with adapters:                       426 (74.3%)\\nReads written (passing filters):           573 (100.0%)\\n\\nTotal basepairs processed:       143,823 bp\\nQuality-trimmed:                  16,761 bp (11.7%)\\nTotal written (filtered):        126,385 bp (87.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 426 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 33.6%\\n  C: 15.0%\\n  G: 48.1%\\n  T: 3.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t186\\t143.2\\t0\\t186\\n2\\t229\\t35.8\\t0\\t229\\n3\\t11\\t9.0\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar456-2019_S108_L001_R1_001.fastq.gz\\n=============================================\\n573 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar456-2019_S108_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar456-2019_S108_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar456-2019_S108_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar456-2019_S108_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar456-2019_S108_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (32 us/read; 1.88 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     573\\nReads with adapters:                        41 (7.2%)\\nReads written (passing filters):           573 (100.0%)\\n\\nTotal basepairs processed:       143,823 bp\\nQuality-trimmed:                  24,267 bp (16.9%)\\nTotal written (filtered):        119,509 bp (83.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 41 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.5%\\n  C: 12.2%\\n  G: 46.3%\\n  T: 22.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t35\\t143.2\\t0\\t35\\n2\\t6\\t35.8\\t0\\t6\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar456-2019_S108_L001_R2_001.fastq.gz\\n=============================================\\n573 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar456-2019_S108_L001_R1_001_trimmed.fq.gz and sar456-2019_S108_L001_R2_001_trimmed.fq.gz\\nfile_1: sar456-2019_S108_L001_R1_001_trimmed.fq.gz, file_2: sar456-2019_S108_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar456-2019_S108_L001_R1_001_trimmed.fq.gz and sar456-2019_S108_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar456-2019_S108_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar456-2019_S108_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 573\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 31 (5.41%)\\n\\n\\n  >>> Now running FastQC on the validated data sar456-2019_S108_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar456-2019_S108_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar456-2019_S108_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar456-2019_S108_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar456-2019_S108_L001_R1_001_trimmed.fq.gz and sar456-2019_S108_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar457-2019_S109_L001_R1_001.fastq.gz data/reads/sar457-2019_S109_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar457-2019_S109_L001_R1_001.fastq.gz data/reads/sar457-2019_S109_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar457-2019_S109_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar457-2019_S109_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar457-2019_S109_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar457-2019_S109_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar457-2019_S109_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,672\\nReads with adapters:                     1,281 (76.6%)\\nReads written (passing filters):         1,672 (100.0%)\\n\\nTotal basepairs processed:       419,672 bp\\nQuality-trimmed:                  48,009 bp (11.4%)\\nTotal written (filtered):        369,712 bp (88.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1281 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.7%\\n  C: 15.9%\\n  G: 38.7%\\n  T: 4.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t633\\t418.0\\t0\\t633\\n2\\t626\\t104.5\\t0\\t626\\n3\\t22\\t26.1\\t0\\t22\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar457-2019_S109_L001_R1_001.fastq.gz\\n=============================================\\n1672 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar457-2019_S109_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar457-2019_S109_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar457-2019_S109_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar457-2019_S109_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar457-2019_S109_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (24 us/read; 2.45 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,672\\nReads with adapters:                        95 (5.7%)\\nReads written (passing filters):         1,672 (100.0%)\\n\\nTotal basepairs processed:       419,672 bp\\nQuality-trimmed:                  78,797 bp (18.8%)\\nTotal written (filtered):        340,761 bp (81.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 95 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 14.7%\\n  C: 21.1%\\n  G: 30.5%\\n  T: 33.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t80\\t418.0\\t0\\t80\\n2\\t12\\t104.5\\t0\\t12\\n3\\t2\\t26.1\\t0\\t2\\n4\\t1\\t6.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar457-2019_S109_L001_R2_001.fastq.gz\\n=============================================\\n1672 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar457-2019_S109_L001_R1_001_trimmed.fq.gz and sar457-2019_S109_L001_R2_001_trimmed.fq.gz\\nfile_1: sar457-2019_S109_L001_R1_001_trimmed.fq.gz, file_2: sar457-2019_S109_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar457-2019_S109_L001_R1_001_trimmed.fq.gz and sar457-2019_S109_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar457-2019_S109_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar457-2019_S109_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1672\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 111 (6.64%)\\n\\n\\n  >>> Now running FastQC on the validated data sar457-2019_S109_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar457-2019_S109_L001_R1_001_val_1.fq.gz\\nApprox 60% complete for sar457-2019_S109_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar457-2019_S109_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar457-2019_S109_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar457-2019_S109_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar457-2019_S109_L001_R1_001_trimmed.fq.gz and sar457-2019_S109_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar458-2019_S110_L001_R1_001.fastq.gz data/reads/sar458-2019_S110_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar458-2019_S110_L001_R1_001.fastq.gz data/reads/sar458-2019_S110_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar458-2019_S110_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar458-2019_S110_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar458-2019_S110_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar458-2019_S110_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar458-2019_S110_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,142\\nReads with adapters:                       883 (77.3%)\\nReads written (passing filters):         1,142 (100.0%)\\n\\nTotal basepairs processed:       286,642 bp\\nQuality-trimmed:                  28,538 bp (10.0%)\\nTotal written (filtered):        256,764 bp (89.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 883 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.1%\\n  C: 17.3%\\n  G: 37.6%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t445\\t285.5\\t0\\t445\\n2\\t420\\t71.4\\t0\\t420\\n3\\t17\\t17.8\\t0\\t17\\n4\\t1\\t4.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar458-2019_S110_L001_R1_001.fastq.gz\\n=============================================\\n1142 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar458-2019_S110_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar458-2019_S110_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar458-2019_S110_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar458-2019_S110_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar458-2019_S110_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (27 us/read; 2.23 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,142\\nReads with adapters:                        78 (6.8%)\\nReads written (passing filters):         1,142 (100.0%)\\n\\nTotal basepairs processed:       286,642 bp\\nQuality-trimmed:                  46,179 bp (16.1%)\\nTotal written (filtered):        240,373 bp (83.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 78 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 16.7%\\n  C: 29.5%\\n  G: 23.1%\\n  T: 29.5%\\n  none/other: 1.3%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t68\\t285.5\\t0\\t68\\n2\\t8\\t71.4\\t0\\t8\\n3\\t2\\t17.8\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar458-2019_S110_L001_R2_001.fastq.gz\\n=============================================\\n1142 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar458-2019_S110_L001_R1_001_trimmed.fq.gz and sar458-2019_S110_L001_R2_001_trimmed.fq.gz\\nfile_1: sar458-2019_S110_L001_R1_001_trimmed.fq.gz, file_2: sar458-2019_S110_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar458-2019_S110_L001_R1_001_trimmed.fq.gz and sar458-2019_S110_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar458-2019_S110_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar458-2019_S110_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1142\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 83 (7.27%)\\n\\n\\n  >>> Now running FastQC on the validated data sar458-2019_S110_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar458-2019_S110_L001_R1_001_val_1.fq.gz\\nApprox 90% complete for sar458-2019_S110_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar458-2019_S110_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar458-2019_S110_L001_R2_001_val_2.fq.gz\\nApprox 90% complete for sar458-2019_S110_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar458-2019_S110_L001_R1_001_trimmed.fq.gz and sar458-2019_S110_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar459-2019_S111_L001_R1_001.fastq.gz data/reads/sar459-2019_S111_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar459-2019_S111_L001_R1_001.fastq.gz data/reads/sar459-2019_S111_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar459-2019_S111_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar459-2019_S111_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar459-2019_S111_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar459-2019_S111_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar459-2019_S111_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (41 us/read; 1.46 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,921\\nReads with adapters:                     1,375 (71.6%)\\nReads written (passing filters):         1,921 (100.0%)\\n\\nTotal basepairs processed:       482,171 bp\\nQuality-trimmed:                  52,726 bp (10.9%)\\nTotal written (filtered):        427,286 bp (88.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1375 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.1%\\n  C: 14.8%\\n  G: 54.9%\\n  T: 3.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t611\\t480.2\\t0\\t611\\n2\\t744\\t120.1\\t0\\t744\\n3\\t20\\t30.0\\t0\\t20\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar459-2019_S111_L001_R1_001.fastq.gz\\n=============================================\\n1921 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar459-2019_S111_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar459-2019_S111_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar459-2019_S111_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar459-2019_S111_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar459-2019_S111_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (24 us/read; 2.51 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,921\\nReads with adapters:                       141 (7.3%)\\nReads written (passing filters):         1,921 (100.0%)\\n\\nTotal basepairs processed:       482,171 bp\\nQuality-trimmed:                  96,320 bp (20.0%)\\nTotal written (filtered):        385,687 bp (80.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 141 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.6%\\n  C: 31.2%\\n  G: 22.0%\\n  T: 31.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t120\\t480.2\\t0\\t120\\n2\\t19\\t120.1\\t0\\t19\\n3\\t2\\t30.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar459-2019_S111_L001_R2_001.fastq.gz\\n=============================================\\n1921 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar459-2019_S111_L001_R1_001_trimmed.fq.gz and sar459-2019_S111_L001_R2_001_trimmed.fq.gz\\nfile_1: sar459-2019_S111_L001_R1_001_trimmed.fq.gz, file_2: sar459-2019_S111_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar459-2019_S111_L001_R1_001_trimmed.fq.gz and sar459-2019_S111_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar459-2019_S111_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar459-2019_S111_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1921\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 169 (8.80%)\\n\\n\\n  >>> Now running FastQC on the validated data sar459-2019_S111_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar459-2019_S111_L001_R1_001_val_1.fq.gz\\nApprox 55% complete for sar459-2019_S111_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar459-2019_S111_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar459-2019_S111_L001_R2_001_val_2.fq.gz\\nApprox 55% complete for sar459-2019_S111_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar459-2019_S111_L001_R1_001_trimmed.fq.gz and sar459-2019_S111_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar460-2019_S112_L001_R1_001.fastq.gz data/reads/sar460-2019_S112_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar460-2019_S112_L001_R1_001.fastq.gz data/reads/sar460-2019_S112_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar460-2019_S112_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar460-2019_S112_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar460-2019_S112_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar460-2019_S112_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar460-2019_S112_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (33 us/read; 1.83 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     891\\nReads with adapters:                       579 (65.0%)\\nReads written (passing filters):           891 (100.0%)\\n\\nTotal basepairs processed:       223,641 bp\\nQuality-trimmed:                  31,405 bp (14.0%)\\nTotal written (filtered):        191,331 bp (85.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 579 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.4%\\n  C: 20.2%\\n  G: 48.0%\\n  T: 7.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t273\\t222.8\\t0\\t273\\n2\\t286\\t55.7\\t0\\t286\\n3\\t20\\t13.9\\t0\\t20\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar460-2019_S112_L001_R1_001.fastq.gz\\n=============================================\\n891 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar460-2019_S112_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar460-2019_S112_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar460-2019_S112_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar460-2019_S112_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar460-2019_S112_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (56 us/read; 1.08 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     891\\nReads with adapters:                        85 (9.5%)\\nReads written (passing filters):           891 (100.0%)\\n\\nTotal basepairs processed:       223,641 bp\\nQuality-trimmed:                  55,266 bp (24.7%)\\nTotal written (filtered):        168,266 bp (75.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 85 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.1%\\n  C: 21.2%\\n  G: 22.4%\\n  T: 29.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t70\\t222.8\\t0\\t70\\n2\\t10\\t55.7\\t0\\t10\\n3\\t3\\t13.9\\t0\\t3\\n5\\t2\\t0.9\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar460-2019_S112_L001_R2_001.fastq.gz\\n=============================================\\n891 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar460-2019_S112_L001_R1_001_trimmed.fq.gz and sar460-2019_S112_L001_R2_001_trimmed.fq.gz\\nfile_1: sar460-2019_S112_L001_R1_001_trimmed.fq.gz, file_2: sar460-2019_S112_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar460-2019_S112_L001_R1_001_trimmed.fq.gz and sar460-2019_S112_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar460-2019_S112_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar460-2019_S112_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 891\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 89 (9.99%)\\n\\n\\n  >>> Now running FastQC on the validated data sar460-2019_S112_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar460-2019_S112_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar460-2019_S112_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar460-2019_S112_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar460-2019_S112_L001_R1_001_trimmed.fq.gz and sar460-2019_S112_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar461-2019_S113_L001_R1_001.fastq.gz data/reads/sar461-2019_S113_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar461-2019_S113_L001_R1_001.fastq.gz data/reads/sar461-2019_S113_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar461-2019_S113_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar461-2019_S113_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar461-2019_S113_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar461-2019_S113_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar461-2019_S113_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,270\\nReads with adapters:                       966 (76.1%)\\nReads written (passing filters):         1,270 (100.0%)\\n\\nTotal basepairs processed:       318,770 bp\\nQuality-trimmed:                  36,867 bp (11.6%)\\nTotal written (filtered):        280,414 bp (88.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 966 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.1%\\n  C: 17.2%\\n  G: 37.4%\\n  T: 4.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t460\\t317.5\\t0\\t460\\n2\\t492\\t79.4\\t0\\t492\\n3\\t12\\t19.8\\t0\\t12\\n4\\t1\\t5.0\\t0\\t1\\n5\\t1\\t1.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar461-2019_S113_L001_R1_001.fastq.gz\\n=============================================\\n1270 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar461-2019_S113_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar461-2019_S113_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar461-2019_S113_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar461-2019_S113_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar461-2019_S113_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (43 us/read; 1.40 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,270\\nReads with adapters:                        84 (6.6%)\\nReads written (passing filters):         1,270 (100.0%)\\n\\nTotal basepairs processed:       318,770 bp\\nQuality-trimmed:                  51,914 bp (16.3%)\\nTotal written (filtered):        266,759 bp (83.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 84 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.0%\\n  C: 22.6%\\n  G: 27.4%\\n  T: 31.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t75\\t317.5\\t0\\t75\\n2\\t7\\t79.4\\t0\\t7\\n3\\t1\\t19.8\\t0\\t1\\n5\\t1\\t1.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar461-2019_S113_L001_R2_001.fastq.gz\\n=============================================\\n1270 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar461-2019_S113_L001_R1_001_trimmed.fq.gz and sar461-2019_S113_L001_R2_001_trimmed.fq.gz\\nfile_1: sar461-2019_S113_L001_R1_001_trimmed.fq.gz, file_2: sar461-2019_S113_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar461-2019_S113_L001_R1_001_trimmed.fq.gz and sar461-2019_S113_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar461-2019_S113_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar461-2019_S113_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1270\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 75 (5.91%)\\n\\n\\n  >>> Now running FastQC on the validated data sar461-2019_S113_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar461-2019_S113_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar461-2019_S113_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar461-2019_S113_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar461-2019_S113_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar461-2019_S113_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar461-2019_S113_L001_R1_001_trimmed.fq.gz and sar461-2019_S113_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar462-2019_S114_L001_R1_001.fastq.gz data/reads/sar462-2019_S114_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar462-2019_S114_L001_R1_001.fastq.gz data/reads/sar462-2019_S114_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar462-2019_S114_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar462-2019_S114_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar462-2019_S114_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar462-2019_S114_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar462-2019_S114_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.03 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,712\\nReads with adapters:                     1,270 (74.2%)\\nReads written (passing filters):         1,712 (100.0%)\\n\\nTotal basepairs processed:       429,712 bp\\nQuality-trimmed:                  43,086 bp (10.0%)\\nTotal written (filtered):        384,664 bp (89.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1270 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.4%\\n  C: 14.2%\\n  G: 53.5%\\n  T: 6.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t606\\t428.0\\t0\\t606\\n2\\t637\\t107.0\\t0\\t637\\n3\\t26\\t26.8\\t0\\t26\\n4\\t1\\t6.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar462-2019_S114_L001_R1_001.fastq.gz\\n=============================================\\n1712 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar462-2019_S114_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar462-2019_S114_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar462-2019_S114_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar462-2019_S114_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar462-2019_S114_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (25 us/read; 2.40 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,712\\nReads with adapters:                       146 (8.5%)\\nReads written (passing filters):         1,712 (100.0%)\\n\\nTotal basepairs processed:       429,712 bp\\nQuality-trimmed:                  73,698 bp (17.2%)\\nTotal written (filtered):        355,844 bp (82.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 146 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.2%\\n  C: 21.9%\\n  G: 31.5%\\n  T: 26.7%\\n  none/other: 0.7%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t126\\t428.0\\t0\\t126\\n2\\t16\\t107.0\\t0\\t16\\n3\\t4\\t26.8\\t0\\t4\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar462-2019_S114_L001_R2_001.fastq.gz\\n=============================================\\n1712 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar462-2019_S114_L001_R1_001_trimmed.fq.gz and sar462-2019_S114_L001_R2_001_trimmed.fq.gz\\nfile_1: sar462-2019_S114_L001_R1_001_trimmed.fq.gz, file_2: sar462-2019_S114_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar462-2019_S114_L001_R1_001_trimmed.fq.gz and sar462-2019_S114_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar462-2019_S114_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar462-2019_S114_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1712\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 78 (4.56%)\\n\\n\\n  >>> Now running FastQC on the validated data sar462-2019_S114_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar462-2019_S114_L001_R1_001_val_1.fq.gz\\nApprox 60% complete for sar462-2019_S114_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar462-2019_S114_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar462-2019_S114_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar462-2019_S114_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar462-2019_S114_L001_R1_001_trimmed.fq.gz and sar462-2019_S114_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar463-2019_S115_L001_R1_001.fastq.gz data/reads/sar463-2019_S115_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar463-2019_S115_L001_R1_001.fastq.gz data/reads/sar463-2019_S115_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar463-2019_S115_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar463-2019_S115_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar463-2019_S115_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar463-2019_S115_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar463-2019_S115_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.91 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,010\\nReads with adapters:                       718 (71.1%)\\nReads written (passing filters):         1,010 (100.0%)\\n\\nTotal basepairs processed:       253,510 bp\\nQuality-trimmed:                  32,455 bp (12.8%)\\nTotal written (filtered):        219,936 bp (86.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 718 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.5%\\n  C: 13.9%\\n  G: 51.5%\\n  T: 5.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t330\\t252.5\\t0\\t330\\n2\\t375\\t63.1\\t0\\t375\\n3\\t13\\t15.8\\t0\\t13\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar463-2019_S115_L001_R1_001.fastq.gz\\n=============================================\\n1010 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar463-2019_S115_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar463-2019_S115_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar463-2019_S115_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar463-2019_S115_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar463-2019_S115_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.34 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,010\\nReads with adapters:                        80 (7.9%)\\nReads written (passing filters):         1,010 (100.0%)\\n\\nTotal basepairs processed:       253,510 bp\\nQuality-trimmed:                  49,410 bp (19.5%)\\nTotal written (filtered):        204,004 bp (80.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 80 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.2%\\n  C: 20.0%\\n  G: 33.8%\\n  T: 25.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t69\\t252.5\\t0\\t69\\n2\\t6\\t63.1\\t0\\t6\\n3\\t5\\t15.8\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar463-2019_S115_L001_R2_001.fastq.gz\\n=============================================\\n1010 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar463-2019_S115_L001_R1_001_trimmed.fq.gz and sar463-2019_S115_L001_R2_001_trimmed.fq.gz\\nfile_1: sar463-2019_S115_L001_R1_001_trimmed.fq.gz, file_2: sar463-2019_S115_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar463-2019_S115_L001_R1_001_trimmed.fq.gz and sar463-2019_S115_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar463-2019_S115_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar463-2019_S115_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1010\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 64 (6.34%)\\n\\n\\n  >>> Now running FastQC on the validated data sar463-2019_S115_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar463-2019_S115_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar463-2019_S115_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar463-2019_S115_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar463-2019_S115_L001_R1_001_trimmed.fq.gz and sar463-2019_S115_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar464-2019_S116_L001_R1_001.fastq.gz data/reads/sar464-2019_S116_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar464-2019_S116_L001_R1_001.fastq.gz data/reads/sar464-2019_S116_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar464-2019_S116_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar464-2019_S116_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar464-2019_S116_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar464-2019_S116_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar464-2019_S116_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (69 us/read; 0.86 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,113\\nReads with adapters:                       853 (76.6%)\\nReads written (passing filters):         1,113 (100.0%)\\n\\nTotal basepairs processed:       279,363 bp\\nQuality-trimmed:                  28,778 bp (10.3%)\\nTotal written (filtered):        249,300 bp (89.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 853 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.5%\\n  C: 19.1%\\n  G: 40.2%\\n  T: 5.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t438\\t278.2\\t0\\t438\\n2\\t398\\t69.6\\t0\\t398\\n3\\t17\\t17.4\\t0\\t17\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar464-2019_S116_L001_R1_001.fastq.gz\\n=============================================\\n1113 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar464-2019_S116_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar464-2019_S116_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar464-2019_S116_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar464-2019_S116_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar464-2019_S116_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.36 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,113\\nReads with adapters:                        97 (8.7%)\\nReads written (passing filters):         1,113 (100.0%)\\n\\nTotal basepairs processed:       279,363 bp\\nQuality-trimmed:                  55,346 bp (19.8%)\\nTotal written (filtered):        223,910 bp (80.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 97 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.6%\\n  C: 19.6%\\n  G: 40.2%\\n  T: 16.5%\\n  none/other: 2.1%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t88\\t278.2\\t0\\t88\\n2\\t8\\t69.6\\t0\\t8\\n3\\t1\\t17.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar464-2019_S116_L001_R2_001.fastq.gz\\n=============================================\\n1113 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar464-2019_S116_L001_R1_001_trimmed.fq.gz and sar464-2019_S116_L001_R2_001_trimmed.fq.gz\\nfile_1: sar464-2019_S116_L001_R1_001_trimmed.fq.gz, file_2: sar464-2019_S116_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar464-2019_S116_L001_R1_001_trimmed.fq.gz and sar464-2019_S116_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar464-2019_S116_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar464-2019_S116_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1113\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 95 (8.54%)\\n\\n\\n  >>> Now running FastQC on the validated data sar464-2019_S116_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar464-2019_S116_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar464-2019_S116_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar464-2019_S116_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar464-2019_S116_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar464-2019_S116_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar464-2019_S116_L001_R1_001_trimmed.fq.gz and sar464-2019_S116_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar465-2019_S117_L001_R1_001.fastq.gz data/reads/sar465-2019_S117_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar465-2019_S117_L001_R1_001.fastq.gz data/reads/sar465-2019_S117_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar465-2019_S117_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar465-2019_S117_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar465-2019_S117_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar465-2019_S117_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar465-2019_S117_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,151\\nReads with adapters:                       733 (63.7%)\\nReads written (passing filters):         1,151 (100.0%)\\n\\nTotal basepairs processed:       288,901 bp\\nQuality-trimmed:                  45,222 bp (15.7%)\\nTotal written (filtered):        242,575 bp (84.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 733 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.3%\\n  C: 19.6%\\n  G: 35.1%\\n  T: 7.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t388\\t287.8\\t0\\t388\\n2\\t320\\t71.9\\t0\\t320\\n3\\t24\\t18.0\\t0\\t24\\n4\\t1\\t4.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar465-2019_S117_L001_R1_001.fastq.gz\\n=============================================\\n1151 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar465-2019_S117_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar465-2019_S117_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar465-2019_S117_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar465-2019_S117_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar465-2019_S117_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.37 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,151\\nReads with adapters:                       115 (10.0%)\\nReads written (passing filters):         1,151 (100.0%)\\n\\nTotal basepairs processed:       288,901 bp\\nQuality-trimmed:                  84,969 bp (29.4%)\\nTotal written (filtered):        203,800 bp (70.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 115 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 20.9%\\n  C: 23.5%\\n  G: 32.2%\\n  T: 23.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t103\\t287.8\\t0\\t103\\n2\\t8\\t71.9\\t0\\t8\\n3\\t3\\t18.0\\t0\\t3\\n4\\t1\\t4.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar465-2019_S117_L001_R2_001.fastq.gz\\n=============================================\\n1151 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar465-2019_S117_L001_R1_001_trimmed.fq.gz and sar465-2019_S117_L001_R2_001_trimmed.fq.gz\\nfile_1: sar465-2019_S117_L001_R1_001_trimmed.fq.gz, file_2: sar465-2019_S117_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar465-2019_S117_L001_R1_001_trimmed.fq.gz and sar465-2019_S117_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar465-2019_S117_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar465-2019_S117_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1151\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 161 (13.99%)\\n\\n\\n  >>> Now running FastQC on the validated data sar465-2019_S117_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar465-2019_S117_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar465-2019_S117_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar465-2019_S117_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar465-2019_S117_L001_R1_001_trimmed.fq.gz and sar465-2019_S117_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar466-2019_S118_L001_R1_001.fastq.gz data/reads/sar466-2019_S118_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar466-2019_S118_L001_R1_001.fastq.gz data/reads/sar466-2019_S118_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar466-2019_S118_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar466-2019_S118_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar466-2019_S118_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar466-2019_S118_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar466-2019_S118_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,774\\nReads with adapters:                     1,368 (77.1%)\\nReads written (passing filters):         1,774 (100.0%)\\n\\nTotal basepairs processed:       445,274 bp\\nQuality-trimmed:                  40,830 bp (9.2%)\\nTotal written (filtered):        402,249 bp (90.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1368 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.1%\\n  C: 12.8%\\n  G: 54.1%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t561\\t443.5\\t0\\t561\\n2\\t787\\t110.9\\t0\\t787\\n3\\t20\\t27.7\\t0\\t20\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar466-2019_S118_L001_R1_001.fastq.gz\\n=============================================\\n1774 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar466-2019_S118_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar466-2019_S118_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar466-2019_S118_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar466-2019_S118_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar466-2019_S118_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (34 us/read; 1.75 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,774\\nReads with adapters:                       143 (8.1%)\\nReads written (passing filters):         1,774 (100.0%)\\n\\nTotal basepairs processed:       445,274 bp\\nQuality-trimmed:                  86,336 bp (19.4%)\\nTotal written (filtered):        358,779 bp (80.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 143 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.4%\\n  C: 14.0%\\n  G: 37.1%\\n  T: 25.2%\\n  none/other: 1.4%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t130\\t443.5\\t0\\t130\\n2\\t10\\t110.9\\t0\\t10\\n3\\t3\\t27.7\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar466-2019_S118_L001_R2_001.fastq.gz\\n=============================================\\n1774 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar466-2019_S118_L001_R1_001_trimmed.fq.gz and sar466-2019_S118_L001_R2_001_trimmed.fq.gz\\nfile_1: sar466-2019_S118_L001_R1_001_trimmed.fq.gz, file_2: sar466-2019_S118_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar466-2019_S118_L001_R1_001_trimmed.fq.gz and sar466-2019_S118_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar466-2019_S118_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar466-2019_S118_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1774\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 148 (8.34%)\\n\\n\\n  >>> Now running FastQC on the validated data sar466-2019_S118_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar466-2019_S118_L001_R1_001_val_1.fq.gz\\nApprox 60% complete for sar466-2019_S118_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar466-2019_S118_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar466-2019_S118_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar466-2019_S118_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar466-2019_S118_L001_R1_001_trimmed.fq.gz and sar466-2019_S118_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar467-2019_S119_L001_R1_001.fastq.gz data/reads/sar467-2019_S119_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar467-2019_S119_L001_R1_001.fastq.gz data/reads/sar467-2019_S119_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar467-2019_S119_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar467-2019_S119_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar467-2019_S119_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar467-2019_S119_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar467-2019_S119_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.99 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,192\\nReads with adapters:                       869 (72.9%)\\nReads written (passing filters):         1,192 (100.0%)\\n\\nTotal basepairs processed:       299,192 bp\\nQuality-trimmed:                  36,532 bp (12.2%)\\nTotal written (filtered):        261,268 bp (87.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 869 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.0%\\n  C: 13.0%\\n  G: 53.0%\\n  T: 4.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t364\\t298.0\\t0\\t364\\n2\\t487\\t74.5\\t0\\t487\\n3\\t18\\t18.6\\t0\\t18\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar467-2019_S119_L001_R1_001.fastq.gz\\n=============================================\\n1192 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar467-2019_S119_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar467-2019_S119_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar467-2019_S119_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar467-2019_S119_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar467-2019_S119_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (38 us/read; 1.59 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,192\\nReads with adapters:                       111 (9.3%)\\nReads written (passing filters):         1,192 (100.0%)\\n\\nTotal basepairs processed:       299,192 bp\\nQuality-trimmed:                  61,098 bp (20.4%)\\nTotal written (filtered):        237,971 bp (79.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 111 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.6%\\n  C: 20.7%\\n  G: 33.3%\\n  T: 24.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t100\\t298.0\\t0\\t100\\n2\\t10\\t74.5\\t0\\t10\\n3\\t1\\t18.6\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar467-2019_S119_L001_R2_001.fastq.gz\\n=============================================\\n1192 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar467-2019_S119_L001_R1_001_trimmed.fq.gz and sar467-2019_S119_L001_R2_001_trimmed.fq.gz\\nfile_1: sar467-2019_S119_L001_R1_001_trimmed.fq.gz, file_2: sar467-2019_S119_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar467-2019_S119_L001_R1_001_trimmed.fq.gz and sar467-2019_S119_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar467-2019_S119_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar467-2019_S119_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1192\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 112 (9.40%)\\n\\n\\n  >>> Now running FastQC on the validated data sar467-2019_S119_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar467-2019_S119_L001_R1_001_val_1.fq.gz\\nApprox 90% complete for sar467-2019_S119_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar467-2019_S119_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar467-2019_S119_L001_R2_001_val_2.fq.gz\\nApprox 90% complete for sar467-2019_S119_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar467-2019_S119_L001_R1_001_trimmed.fq.gz and sar467-2019_S119_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar468-2019_S120_L001_R1_001.fastq.gz data/reads/sar468-2019_S120_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar468-2019_S120_L001_R1_001.fastq.gz data/reads/sar468-2019_S120_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar468-2019_S120_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar468-2019_S120_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar468-2019_S120_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar468-2019_S120_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar468-2019_S120_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,039\\nReads with adapters:                       768 (73.9%)\\nReads written (passing filters):         1,039 (100.0%)\\n\\nTotal basepairs processed:       260,789 bp\\nQuality-trimmed:                  29,753 bp (11.4%)\\nTotal written (filtered):        229,814 bp (88.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 768 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.1%\\n  C: 12.8%\\n  G: 54.6%\\n  T: 4.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t328\\t259.8\\t0\\t328\\n2\\t426\\t64.9\\t0\\t426\\n3\\t14\\t16.2\\t0\\t14\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar468-2019_S120_L001_R1_001.fastq.gz\\n=============================================\\n1039 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar468-2019_S120_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar468-2019_S120_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar468-2019_S120_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar468-2019_S120_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar468-2019_S120_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (29 us/read; 2.09 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,039\\nReads with adapters:                        80 (7.7%)\\nReads written (passing filters):         1,039 (100.0%)\\n\\nTotal basepairs processed:       260,789 bp\\nQuality-trimmed:                  44,201 bp (16.9%)\\nTotal written (filtered):        216,492 bp (83.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 80 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.2%\\n  C: 23.8%\\n  G: 27.5%\\n  T: 22.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t67\\t259.8\\t0\\t67\\n2\\t10\\t64.9\\t0\\t10\\n3\\t3\\t16.2\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar468-2019_S120_L001_R2_001.fastq.gz\\n=============================================\\n1039 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar468-2019_S120_L001_R1_001_trimmed.fq.gz and sar468-2019_S120_L001_R2_001_trimmed.fq.gz\\nfile_1: sar468-2019_S120_L001_R1_001_trimmed.fq.gz, file_2: sar468-2019_S120_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar468-2019_S120_L001_R1_001_trimmed.fq.gz and sar468-2019_S120_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar468-2019_S120_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar468-2019_S120_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1039\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 67 (6.45%)\\n\\n\\n  >>> Now running FastQC on the validated data sar468-2019_S120_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar468-2019_S120_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar468-2019_S120_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar468-2019_S120_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar468-2019_S120_L001_R1_001_trimmed.fq.gz and sar468-2019_S120_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar469-2019_S121_L001_R1_001.fastq.gz data/reads/sar469-2019_S121_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar469-2019_S121_L001_R1_001.fastq.gz data/reads/sar469-2019_S121_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar469-2019_S121_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar469-2019_S121_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar469-2019_S121_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar469-2019_S121_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar469-2019_S121_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (33 us/read; 1.80 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,176\\nReads with adapters:                       926 (78.7%)\\nReads written (passing filters):         1,176 (100.0%)\\n\\nTotal basepairs processed:       295,176 bp\\nQuality-trimmed:                  25,333 bp (8.6%)\\nTotal written (filtered):        268,334 bp (90.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 926 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 30.5%\\n  C: 13.0%\\n  G: 53.3%\\n  T: 3.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t360\\t294.0\\t0\\t360\\n2\\t550\\t73.5\\t0\\t550\\n3\\t15\\t18.4\\t0\\t15\\n4\\t1\\t4.6\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar469-2019_S121_L001_R1_001.fastq.gz\\n=============================================\\n1176 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar469-2019_S121_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar469-2019_S121_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar469-2019_S121_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar469-2019_S121_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar469-2019_S121_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.42 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,176\\nReads with adapters:                        67 (5.7%)\\nReads written (passing filters):         1,176 (100.0%)\\n\\nTotal basepairs processed:       295,176 bp\\nQuality-trimmed:                  39,953 bp (13.5%)\\nTotal written (filtered):        255,145 bp (86.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 67 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.4%\\n  C: 16.4%\\n  G: 47.8%\\n  T: 16.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t58\\t294.0\\t0\\t58\\n2\\t7\\t73.5\\t0\\t7\\n3\\t2\\t18.4\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar469-2019_S121_L001_R2_001.fastq.gz\\n=============================================\\n1176 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar469-2019_S121_L001_R1_001_trimmed.fq.gz and sar469-2019_S121_L001_R2_001_trimmed.fq.gz\\nfile_1: sar469-2019_S121_L001_R1_001_trimmed.fq.gz, file_2: sar469-2019_S121_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar469-2019_S121_L001_R1_001_trimmed.fq.gz and sar469-2019_S121_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar469-2019_S121_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar469-2019_S121_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1176\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 36 (3.06%)\\n\\n\\n  >>> Now running FastQC on the validated data sar469-2019_S121_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar469-2019_S121_L001_R1_001_val_1.fq.gz\\nApprox 85% complete for sar469-2019_S121_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar469-2019_S121_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar469-2019_S121_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar469-2019_S121_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar469-2019_S121_L001_R1_001_trimmed.fq.gz and sar469-2019_S121_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar470-2019_S122_L001_R1_001.fastq.gz data/reads/sar470-2019_S122_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar470-2019_S122_L001_R1_001.fastq.gz data/reads/sar470-2019_S122_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar470-2019_S122_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar470-2019_S122_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar470-2019_S122_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar470-2019_S122_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar470-2019_S122_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,259\\nReads with adapters:                       970 (77.0%)\\nReads written (passing filters):         1,259 (100.0%)\\n\\nTotal basepairs processed:       316,009 bp\\nQuality-trimmed:                  29,823 bp (9.4%)\\nTotal written (filtered):        284,652 bp (90.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 970 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.2%\\n  C: 13.2%\\n  G: 53.2%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t424\\t314.8\\t0\\t424\\n2\\t528\\t78.7\\t0\\t528\\n3\\t18\\t19.7\\t0\\t18\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar470-2019_S122_L001_R1_001.fastq.gz\\n=============================================\\n1259 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar470-2019_S122_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar470-2019_S122_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar470-2019_S122_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar470-2019_S122_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar470-2019_S122_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.34 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,259\\nReads with adapters:                        79 (6.3%)\\nReads written (passing filters):         1,259 (100.0%)\\n\\nTotal basepairs processed:       316,009 bp\\nQuality-trimmed:                  46,152 bp (14.6%)\\nTotal written (filtered):        269,770 bp (85.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 79 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 32.9%\\n  C: 11.4%\\n  G: 31.6%\\n  T: 24.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t71\\t314.8\\t0\\t71\\n2\\t8\\t78.7\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar470-2019_S122_L001_R2_001.fastq.gz\\n=============================================\\n1259 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar470-2019_S122_L001_R1_001_trimmed.fq.gz and sar470-2019_S122_L001_R2_001_trimmed.fq.gz\\nfile_1: sar470-2019_S122_L001_R1_001_trimmed.fq.gz, file_2: sar470-2019_S122_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar470-2019_S122_L001_R1_001_trimmed.fq.gz and sar470-2019_S122_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar470-2019_S122_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar470-2019_S122_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1259\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 61 (4.85%)\\n\\n\\n  >>> Now running FastQC on the validated data sar470-2019_S122_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar470-2019_S122_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar470-2019_S122_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar470-2019_S122_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar470-2019_S122_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar470-2019_S122_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar470-2019_S122_L001_R1_001_trimmed.fq.gz and sar470-2019_S122_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar471-2019_S123_L001_R1_001.fastq.gz data/reads/sar471-2019_S123_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar471-2019_S123_L001_R1_001.fastq.gz data/reads/sar471-2019_S123_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar471-2019_S123_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar471-2019_S123_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar471-2019_S123_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar471-2019_S123_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar471-2019_S123_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (31 us/read; 1.91 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     694\\nReads with adapters:                       442 (63.7%)\\nReads written (passing filters):           694 (100.0%)\\n\\nTotal basepairs processed:       174,194 bp\\nQuality-trimmed:                  27,886 bp (16.0%)\\nTotal written (filtered):        145,635 bp (83.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 442 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.3%\\n  C: 17.2%\\n  G: 32.4%\\n  T: 8.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t219\\t173.5\\t0\\t219\\n2\\t215\\t43.4\\t0\\t215\\n3\\t8\\t10.8\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar471-2019_S123_L001_R1_001.fastq.gz\\n=============================================\\n694 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar471-2019_S123_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar471-2019_S123_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar471-2019_S123_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar471-2019_S123_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar471-2019_S123_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.29 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     694\\nReads with adapters:                        73 (10.5%)\\nReads written (passing filters):           694 (100.0%)\\n\\nTotal basepairs processed:       174,194 bp\\nQuality-trimmed:                  49,316 bp (28.3%)\\nTotal written (filtered):        124,785 bp (71.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 73 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.3%\\n  C: 26.0%\\n  G: 21.9%\\n  T: 28.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t61\\t173.5\\t0\\t61\\n2\\t7\\t43.4\\t0\\t7\\n3\\t3\\t10.8\\t0\\t3\\n4\\t1\\t2.7\\t0\\t1\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar471-2019_S123_L001_R2_001.fastq.gz\\n=============================================\\n694 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar471-2019_S123_L001_R1_001_trimmed.fq.gz and sar471-2019_S123_L001_R2_001_trimmed.fq.gz\\nfile_1: sar471-2019_S123_L001_R1_001_trimmed.fq.gz, file_2: sar471-2019_S123_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar471-2019_S123_L001_R1_001_trimmed.fq.gz and sar471-2019_S123_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar471-2019_S123_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar471-2019_S123_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 694\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 86 (12.39%)\\n\\n\\n  >>> Now running FastQC on the validated data sar471-2019_S123_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar471-2019_S123_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar471-2019_S123_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar471-2019_S123_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar471-2019_S123_L001_R1_001_trimmed.fq.gz and sar471-2019_S123_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar472-2019_S124_L001_R1_001.fastq.gz data/reads/sar472-2019_S124_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar472-2019_S124_L001_R1_001.fastq.gz data/reads/sar472-2019_S124_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar472-2019_S124_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar472-2019_S124_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar472-2019_S124_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar472-2019_S124_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar472-2019_S124_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.01 s (35 us/read; 1.71 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     383\\nReads with adapters:                       271 (70.8%)\\nReads written (passing filters):           383 (100.0%)\\n\\nTotal basepairs processed:        96,133 bp\\nQuality-trimmed:                  14,952 bp (15.6%)\\nTotal written (filtered):         80,758 bp (84.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 271 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 37.3%\\n  C: 16.2%\\n  G: 40.6%\\n  T: 5.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t126\\t95.8\\t0\\t126\\n2\\t138\\t23.9\\t0\\t138\\n3\\t7\\t6.0\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar472-2019_S124_L001_R1_001.fastq.gz\\n=============================================\\n383 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar472-2019_S124_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar472-2019_S124_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar472-2019_S124_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar472-2019_S124_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar472-2019_S124_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.01 s (30 us/read; 2.00 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     383\\nReads with adapters:                        31 (8.1%)\\nReads written (passing filters):           383 (100.0%)\\n\\nTotal basepairs processed:        96,133 bp\\nQuality-trimmed:                  17,653 bp (18.4%)\\nTotal written (filtered):         78,446 bp (81.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 31 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.4%\\n  C: 35.5%\\n  G: 29.0%\\n  T: 16.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t28\\t95.8\\t0\\t28\\n2\\t3\\t23.9\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar472-2019_S124_L001_R2_001.fastq.gz\\n=============================================\\n383 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar472-2019_S124_L001_R1_001_trimmed.fq.gz and sar472-2019_S124_L001_R2_001_trimmed.fq.gz\\nfile_1: sar472-2019_S124_L001_R1_001_trimmed.fq.gz, file_2: sar472-2019_S124_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar472-2019_S124_L001_R1_001_trimmed.fq.gz and sar472-2019_S124_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar472-2019_S124_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar472-2019_S124_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 383\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 13 (3.39%)\\n\\n\\n  >>> Now running FastQC on the validated data sar472-2019_S124_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar472-2019_S124_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar472-2019_S124_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar472-2019_S124_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar472-2019_S124_L001_R1_001_trimmed.fq.gz and sar472-2019_S124_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar473-2019_S125_L001_R1_001.fastq.gz data/reads/sar473-2019_S125_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar473-2019_S125_L001_R1_001.fastq.gz data/reads/sar473-2019_S125_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar473-2019_S125_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar473-2019_S125_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar473-2019_S125_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar473-2019_S125_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar473-2019_S125_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.92 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,204\\nReads with adapters:                       881 (73.2%)\\nReads written (passing filters):         1,204 (100.0%)\\n\\nTotal basepairs processed:       302,204 bp\\nQuality-trimmed:                  32,466 bp (10.7%)\\nTotal written (filtered):        268,343 bp (88.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 881 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.5%\\n  C: 12.1%\\n  G: 54.5%\\n  T: 4.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t385\\t301.0\\t0\\t385\\n2\\t478\\t75.2\\t0\\t478\\n3\\t18\\t18.8\\t0\\t18\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar473-2019_S125_L001_R1_001.fastq.gz\\n=============================================\\n1204 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar473-2019_S125_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar473-2019_S125_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar473-2019_S125_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar473-2019_S125_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar473-2019_S125_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (37 us/read; 1.64 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,204\\nReads with adapters:                       107 (8.9%)\\nReads written (passing filters):         1,204 (100.0%)\\n\\nTotal basepairs processed:       302,204 bp\\nQuality-trimmed:                  57,363 bp (19.0%)\\nTotal written (filtered):        244,710 bp (81.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 107 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.0%\\n  C: 14.0%\\n  G: 29.0%\\n  T: 28.0%\\n  none/other: 0.9%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t89\\t301.0\\t0\\t89\\n2\\t13\\t75.2\\t0\\t13\\n3\\t4\\t18.8\\t0\\t4\\n4\\t1\\t4.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar473-2019_S125_L001_R2_001.fastq.gz\\n=============================================\\n1204 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar473-2019_S125_L001_R1_001_trimmed.fq.gz and sar473-2019_S125_L001_R2_001_trimmed.fq.gz\\nfile_1: sar473-2019_S125_L001_R1_001_trimmed.fq.gz, file_2: sar473-2019_S125_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar473-2019_S125_L001_R1_001_trimmed.fq.gz and sar473-2019_S125_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar473-2019_S125_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar473-2019_S125_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1204\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 99 (8.22%)\\n\\n\\n  >>> Now running FastQC on the validated data sar473-2019_S125_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar473-2019_S125_L001_R1_001_val_1.fq.gz\\nApprox 90% complete for sar473-2019_S125_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar473-2019_S125_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar473-2019_S125_L001_R2_001_val_2.fq.gz\\nApprox 90% complete for sar473-2019_S125_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar473-2019_S125_L001_R1_001_trimmed.fq.gz and sar473-2019_S125_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar474-2019_S126_L001_R1_001.fastq.gz data/reads/sar474-2019_S126_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar474-2019_S126_L001_R1_001.fastq.gz data/reads/sar474-2019_S126_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar474-2019_S126_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar474-2019_S126_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar474-2019_S126_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar474-2019_S126_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar474-2019_S126_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.00 s (579 us/read; 0.10 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                       4\\nReads with adapters:                         4 (100.0%)\\nReads written (passing filters):             4 (100.0%)\\n\\nTotal basepairs processed:         1,004 bp\\nQuality-trimmed:                       2 bp (0.2%)\\nTotal written (filtered):            994 bp (99.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 4 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 75.0%\\n  C: 0.0%\\n  G: 0.0%\\n  T: 25.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t1\\t1.0\\t0\\t1\\n2\\t2\\t0.2\\t0\\t2\\n3\\t1\\t0.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar474-2019_S126_L001_R1_001.fastq.gz\\n=============================================\\n4 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar474-2019_S126_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar474-2019_S126_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar474-2019_S126_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar474-2019_S126_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar474-2019_S126_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (6993 us/read; 0.01 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                       4\\nReads with adapters:                         0 (0.0%)\\nReads written (passing filters):             4 (100.0%)\\n\\nTotal basepairs processed:         1,004 bp\\nQuality-trimmed:                       9 bp (0.9%)\\nTotal written (filtered):            995 bp (99.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 0 times.\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar474-2019_S126_L001_R2_001.fastq.gz\\n=============================================\\n4 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar474-2019_S126_L001_R1_001_trimmed.fq.gz and sar474-2019_S126_L001_R2_001_trimmed.fq.gz\\nfile_1: sar474-2019_S126_L001_R1_001_trimmed.fq.gz, file_2: sar474-2019_S126_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar474-2019_S126_L001_R1_001_trimmed.fq.gz and sar474-2019_S126_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar474-2019_S126_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar474-2019_S126_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 4\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 0 (0.00%)\\n\\n\\n  >>> Now running FastQC on the validated data sar474-2019_S126_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar474-2019_S126_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar474-2019_S126_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar474-2019_S126_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar474-2019_S126_L001_R1_001_trimmed.fq.gz and sar474-2019_S126_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar475-2019_S127_L001_R1_001.fastq.gz data/reads/sar475-2019_S127_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar475-2019_S127_L001_R1_001.fastq.gz data/reads/sar475-2019_S127_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar475-2019_S127_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar475-2019_S127_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar475-2019_S127_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar475-2019_S127_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar475-2019_S127_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (33 us/read; 1.83 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     547\\nReads with adapters:                       331 (60.5%)\\nReads written (passing filters):           547 (100.0%)\\n\\nTotal basepairs processed:       137,297 bp\\nQuality-trimmed:                  27,392 bp (20.0%)\\nTotal written (filtered):        109,410 bp (79.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 331 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.1%\\n  C: 19.3%\\n  G: 33.2%\\n  T: 9.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t172\\t136.8\\t0\\t172\\n2\\t154\\t34.2\\t0\\t154\\n3\\t5\\t8.5\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar475-2019_S127_L001_R1_001.fastq.gz\\n=============================================\\n547 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar475-2019_S127_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar475-2019_S127_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar475-2019_S127_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar475-2019_S127_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar475-2019_S127_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (53 us/read; 1.13 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     547\\nReads with adapters:                        69 (12.6%)\\nReads written (passing filters):           547 (100.0%)\\n\\nTotal basepairs processed:       137,297 bp\\nQuality-trimmed:                  48,002 bp (35.0%)\\nTotal written (filtered):         89,216 bp (65.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 69 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.9%\\n  C: 33.3%\\n  G: 21.7%\\n  T: 29.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t60\\t136.8\\t0\\t60\\n2\\t8\\t34.2\\t0\\t8\\n3\\t1\\t8.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar475-2019_S127_L001_R2_001.fastq.gz\\n=============================================\\n547 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar475-2019_S127_L001_R1_001_trimmed.fq.gz and sar475-2019_S127_L001_R2_001_trimmed.fq.gz\\nfile_1: sar475-2019_S127_L001_R1_001_trimmed.fq.gz, file_2: sar475-2019_S127_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar475-2019_S127_L001_R1_001_trimmed.fq.gz and sar475-2019_S127_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar475-2019_S127_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar475-2019_S127_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 547\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 74 (13.53%)\\n\\n\\n  >>> Now running FastQC on the validated data sar475-2019_S127_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar475-2019_S127_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar475-2019_S127_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar475-2019_S127_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar475-2019_S127_L001_R1_001_trimmed.fq.gz and sar475-2019_S127_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar476-2019_S128_L001_R1_001.fastq.gz data/reads/sar476-2019_S128_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar476-2019_S128_L001_R1_001.fastq.gz data/reads/sar476-2019_S128_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar476-2019_S128_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar476-2019_S128_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar476-2019_S128_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar476-2019_S128_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar476-2019_S128_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (36 us/read; 1.65 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     595\\nReads with adapters:                       452 (76.0%)\\nReads written (passing filters):           595 (100.0%)\\n\\nTotal basepairs processed:       149,345 bp\\nQuality-trimmed:                  19,297 bp (12.9%)\\nTotal written (filtered):        129,365 bp (86.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 452 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.3%\\n  C: 17.3%\\n  G: 39.8%\\n  T: 2.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t227\\t148.8\\t0\\t227\\n2\\t219\\t37.2\\t0\\t219\\n3\\t6\\t9.3\\t0\\t6\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar476-2019_S128_L001_R1_001.fastq.gz\\n=============================================\\n595 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar476-2019_S128_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar476-2019_S128_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar476-2019_S128_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar476-2019_S128_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar476-2019_S128_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (32 us/read; 1.85 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     595\\nReads with adapters:                        42 (7.1%)\\nReads written (passing filters):           595 (100.0%)\\n\\nTotal basepairs processed:       149,345 bp\\nQuality-trimmed:                  30,706 bp (20.6%)\\nTotal written (filtered):        118,589 bp (79.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 42 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.4%\\n  C: 26.2%\\n  G: 26.2%\\n  T: 26.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t34\\t148.8\\t0\\t34\\n2\\t8\\t37.2\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar476-2019_S128_L001_R2_001.fastq.gz\\n=============================================\\n595 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar476-2019_S128_L001_R1_001_trimmed.fq.gz and sar476-2019_S128_L001_R2_001_trimmed.fq.gz\\nfile_1: sar476-2019_S128_L001_R1_001_trimmed.fq.gz, file_2: sar476-2019_S128_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar476-2019_S128_L001_R1_001_trimmed.fq.gz and sar476-2019_S128_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar476-2019_S128_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar476-2019_S128_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 595\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 48 (8.07%)\\n\\n\\n  >>> Now running FastQC on the validated data sar476-2019_S128_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar476-2019_S128_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar476-2019_S128_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar476-2019_S128_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar476-2019_S128_L001_R1_001_trimmed.fq.gz and sar476-2019_S128_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar477-2019_S129_L001_R1_001.fastq.gz data/reads/sar477-2019_S129_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar477-2019_S129_L001_R1_001.fastq.gz data/reads/sar477-2019_S129_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar477-2019_S129_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar477-2019_S129_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar477-2019_S129_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar477-2019_S129_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar477-2019_S129_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (35 us/read; 1.71 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     434\\nReads with adapters:                       295 (68.0%)\\nReads written (passing filters):           434 (100.0%)\\n\\nTotal basepairs processed:       108,934 bp\\nQuality-trimmed:                  17,263 bp (15.8%)\\nTotal written (filtered):         91,219 bp (83.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 295 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.7%\\n  C: 15.3%\\n  G: 35.3%\\n  T: 9.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t143\\t108.5\\t0\\t143\\n2\\t147\\t27.1\\t0\\t147\\n3\\t5\\t6.8\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar477-2019_S129_L001_R1_001.fastq.gz\\n=============================================\\n434 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar477-2019_S129_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar477-2019_S129_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar477-2019_S129_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar477-2019_S129_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar477-2019_S129_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.01 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     434\\nReads with adapters:                        54 (12.4%)\\nReads written (passing filters):           434 (100.0%)\\n\\nTotal basepairs processed:       108,934 bp\\nQuality-trimmed:                  27,711 bp (25.4%)\\nTotal written (filtered):         81,162 bp (74.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 54 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.2%\\n  C: 20.4%\\n  G: 27.8%\\n  T: 25.9%\\n  none/other: 3.7%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t48\\t108.5\\t0\\t48\\n2\\t5\\t27.1\\t0\\t5\\n3\\t1\\t6.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar477-2019_S129_L001_R2_001.fastq.gz\\n=============================================\\n434 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar477-2019_S129_L001_R1_001_trimmed.fq.gz and sar477-2019_S129_L001_R2_001_trimmed.fq.gz\\nfile_1: sar477-2019_S129_L001_R1_001_trimmed.fq.gz, file_2: sar477-2019_S129_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar477-2019_S129_L001_R1_001_trimmed.fq.gz and sar477-2019_S129_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar477-2019_S129_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar477-2019_S129_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 434\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 41 (9.45%)\\n\\n\\n  >>> Now running FastQC on the validated data sar477-2019_S129_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar477-2019_S129_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar477-2019_S129_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar477-2019_S129_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar477-2019_S129_L001_R1_001_trimmed.fq.gz and sar477-2019_S129_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar478-2019_S130_L001_R1_001.fastq.gz data/reads/sar478-2019_S130_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar478-2019_S130_L001_R1_001.fastq.gz data/reads/sar478-2019_S130_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar478-2019_S130_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar478-2019_S130_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar478-2019_S130_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar478-2019_S130_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar478-2019_S130_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.91 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,017\\nReads with adapters:                       729 (71.7%)\\nReads written (passing filters):         1,017 (100.0%)\\n\\nTotal basepairs processed:       255,267 bp\\nQuality-trimmed:                  32,090 bp (12.6%)\\nTotal written (filtered):        222,084 bp (87.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 729 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.3%\\n  C: 16.9%\\n  G: 37.4%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t374\\t254.2\\t0\\t374\\n2\\t346\\t63.6\\t0\\t346\\n3\\t9\\t15.9\\t0\\t9\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar478-2019_S130_L001_R1_001.fastq.gz\\n=============================================\\n1017 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar478-2019_S130_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar478-2019_S130_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar478-2019_S130_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar478-2019_S130_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar478-2019_S130_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.33 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,017\\nReads with adapters:                        85 (8.4%)\\nReads written (passing filters):         1,017 (100.0%)\\n\\nTotal basepairs processed:       255,267 bp\\nQuality-trimmed:                  57,623 bp (22.6%)\\nTotal written (filtered):        197,534 bp (77.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 85 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.9%\\n  C: 14.1%\\n  G: 24.7%\\n  T: 34.1%\\n  none/other: 1.2%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t67\\t254.2\\t0\\t67\\n2\\t11\\t63.6\\t0\\t11\\n3\\t7\\t15.9\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar478-2019_S130_L001_R2_001.fastq.gz\\n=============================================\\n1017 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar478-2019_S130_L001_R1_001_trimmed.fq.gz and sar478-2019_S130_L001_R2_001_trimmed.fq.gz\\nfile_1: sar478-2019_S130_L001_R1_001_trimmed.fq.gz, file_2: sar478-2019_S130_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar478-2019_S130_L001_R1_001_trimmed.fq.gz and sar478-2019_S130_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar478-2019_S130_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar478-2019_S130_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1017\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 109 (10.72%)\\n\\n\\n  >>> Now running FastQC on the validated data sar478-2019_S130_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar478-2019_S130_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar478-2019_S130_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar478-2019_S130_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar478-2019_S130_L001_R1_001_trimmed.fq.gz and sar478-2019_S130_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar479-2019_S131_L001_R1_001.fastq.gz data/reads/sar479-2019_S131_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar479-2019_S131_L001_R1_001.fastq.gz data/reads/sar479-2019_S131_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar479-2019_S131_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar479-2019_S131_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar479-2019_S131_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar479-2019_S131_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar479-2019_S131_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (33 us/read; 1.83 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     709\\nReads with adapters:                       559 (78.8%)\\nReads written (passing filters):           709 (100.0%)\\n\\nTotal basepairs processed:       177,959 bp\\nQuality-trimmed:                  20,527 bp (11.5%)\\nTotal written (filtered):        156,587 bp (88.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 559 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.2%\\n  C: 17.0%\\n  G: 37.2%\\n  T: 3.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t286\\t177.2\\t0\\t286\\n2\\t263\\t44.3\\t0\\t263\\n3\\t8\\t11.1\\t0\\t8\\n4\\t1\\t2.8\\t0\\t1\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar479-2019_S131_L001_R1_001.fastq.gz\\n=============================================\\n709 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar479-2019_S131_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar479-2019_S131_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar479-2019_S131_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar479-2019_S131_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar479-2019_S131_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (27 us/read; 2.21 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     709\\nReads with adapters:                        47 (6.6%)\\nReads written (passing filters):           709 (100.0%)\\n\\nTotal basepairs processed:       177,959 bp\\nQuality-trimmed:                  28,871 bp (16.2%)\\nTotal written (filtered):        149,036 bp (83.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 47 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 10.6%\\n  C: 23.4%\\n  G: 17.0%\\n  T: 48.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t42\\t177.2\\t0\\t42\\n2\\t5\\t44.3\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar479-2019_S131_L001_R2_001.fastq.gz\\n=============================================\\n709 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar479-2019_S131_L001_R1_001_trimmed.fq.gz and sar479-2019_S131_L001_R2_001_trimmed.fq.gz\\nfile_1: sar479-2019_S131_L001_R1_001_trimmed.fq.gz, file_2: sar479-2019_S131_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar479-2019_S131_L001_R1_001_trimmed.fq.gz and sar479-2019_S131_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar479-2019_S131_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar479-2019_S131_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 709\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 31 (4.37%)\\n\\n\\n  >>> Now running FastQC on the validated data sar479-2019_S131_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar479-2019_S131_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar479-2019_S131_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar479-2019_S131_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar479-2019_S131_L001_R1_001_trimmed.fq.gz and sar479-2019_S131_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar480-2019_S132_L001_R1_001.fastq.gz data/reads/sar480-2019_S132_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar480-2019_S132_L001_R1_001.fastq.gz data/reads/sar480-2019_S132_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar480-2019_S132_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar480-2019_S132_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar480-2019_S132_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar480-2019_S132_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar480-2019_S132_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     977\\nReads with adapters:                       656 (67.1%)\\nReads written (passing filters):           977 (100.0%)\\n\\nTotal basepairs processed:       245,227 bp\\nQuality-trimmed:                  30,341 bp (12.4%)\\nTotal written (filtered):        213,871 bp (87.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 656 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 33.2%\\n  C: 15.5%\\n  G: 45.4%\\n  T: 5.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t307\\t244.2\\t0\\t307\\n2\\t339\\t61.1\\t0\\t339\\n3\\t10\\t15.3\\t0\\t10\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar480-2019_S132_L001_R1_001.fastq.gz\\n=============================================\\n977 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar480-2019_S132_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar480-2019_S132_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar480-2019_S132_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar480-2019_S132_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar480-2019_S132_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (27 us/read; 2.23 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     977\\nReads with adapters:                       102 (10.4%)\\nReads written (passing filters):           977 (100.0%)\\n\\nTotal basepairs processed:       245,227 bp\\nQuality-trimmed:                  51,323 bp (20.9%)\\nTotal written (filtered):        193,780 bp (79.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 102 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.7%\\n  C: 33.3%\\n  G: 27.5%\\n  T: 23.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t86\\t244.2\\t0\\t86\\n2\\t11\\t61.1\\t0\\t11\\n3\\t4\\t15.3\\t0\\t4\\n4\\t1\\t3.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar480-2019_S132_L001_R2_001.fastq.gz\\n=============================================\\n977 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar480-2019_S132_L001_R1_001_trimmed.fq.gz and sar480-2019_S132_L001_R2_001_trimmed.fq.gz\\nfile_1: sar480-2019_S132_L001_R1_001_trimmed.fq.gz, file_2: sar480-2019_S132_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar480-2019_S132_L001_R1_001_trimmed.fq.gz and sar480-2019_S132_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar480-2019_S132_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar480-2019_S132_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 977\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 80 (8.19%)\\n\\n\\n  >>> Now running FastQC on the validated data sar480-2019_S132_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar480-2019_S132_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar480-2019_S132_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar480-2019_S132_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar480-2019_S132_L001_R1_001_trimmed.fq.gz and sar480-2019_S132_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar481-2019_S133_L001_R1_001.fastq.gz data/reads/sar481-2019_S133_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar481-2019_S133_L001_R1_001.fastq.gz data/reads/sar481-2019_S133_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar481-2019_S133_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar481-2019_S133_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar481-2019_S133_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar481-2019_S133_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar481-2019_S133_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.92 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     848\\nReads with adapters:                       553 (65.2%)\\nReads written (passing filters):           848 (100.0%)\\n\\nTotal basepairs processed:       212,848 bp\\nQuality-trimmed:                  33,154 bp (15.6%)\\nTotal written (filtered):        178,845 bp (84.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 553 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.7%\\n  C: 15.9%\\n  G: 35.3%\\n  T: 8.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t278\\t212.0\\t0\\t278\\n2\\t257\\t53.0\\t0\\t257\\n3\\t17\\t13.2\\t0\\t17\\n6\\t1\\t0.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar481-2019_S133_L001_R1_001.fastq.gz\\n=============================================\\n848 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar481-2019_S133_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar481-2019_S133_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar481-2019_S133_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar481-2019_S133_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar481-2019_S133_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (28 us/read; 2.17 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     848\\nReads with adapters:                        98 (11.6%)\\nReads written (passing filters):           848 (100.0%)\\n\\nTotal basepairs processed:       212,848 bp\\nQuality-trimmed:                  57,771 bp (27.1%)\\nTotal written (filtered):        154,960 bp (72.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 98 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 18.4%\\n  C: 22.4%\\n  G: 26.5%\\n  T: 31.6%\\n  none/other: 1.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t79\\t212.0\\t0\\t79\\n2\\t19\\t53.0\\t0\\t19\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar481-2019_S133_L001_R2_001.fastq.gz\\n=============================================\\n848 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar481-2019_S133_L001_R1_001_trimmed.fq.gz and sar481-2019_S133_L001_R2_001_trimmed.fq.gz\\nfile_1: sar481-2019_S133_L001_R1_001_trimmed.fq.gz, file_2: sar481-2019_S133_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar481-2019_S133_L001_R1_001_trimmed.fq.gz and sar481-2019_S133_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar481-2019_S133_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar481-2019_S133_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 848\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 92 (10.85%)\\n\\n\\n  >>> Now running FastQC on the validated data sar481-2019_S133_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar481-2019_S133_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar481-2019_S133_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar481-2019_S133_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar481-2019_S133_L001_R1_001_trimmed.fq.gz and sar481-2019_S133_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar482-2019_S134_L001_R1_001.fastq.gz data/reads/sar482-2019_S134_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar482-2019_S134_L001_R1_001.fastq.gz data/reads/sar482-2019_S134_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar482-2019_S134_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar482-2019_S134_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar482-2019_S134_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar482-2019_S134_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar482-2019_S134_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.88 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,011\\nReads with adapters:                       748 (74.0%)\\nReads written (passing filters):         1,011 (100.0%)\\n\\nTotal basepairs processed:       253,761 bp\\nQuality-trimmed:                  23,992 bp (9.5%)\\nTotal written (filtered):        228,623 bp (90.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 748 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.2%\\n  C: 18.3%\\n  G: 42.9%\\n  T: 3.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t367\\t252.8\\t0\\t367\\n2\\t364\\t63.2\\t0\\t364\\n3\\t17\\t15.8\\t0\\t17\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar482-2019_S134_L001_R1_001.fastq.gz\\n=============================================\\n1011 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar482-2019_S134_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar482-2019_S134_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar482-2019_S134_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar482-2019_S134_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar482-2019_S134_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.28 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,011\\nReads with adapters:                        78 (7.7%)\\nReads written (passing filters):         1,011 (100.0%)\\n\\nTotal basepairs processed:       253,761 bp\\nQuality-trimmed:                  40,100 bp (15.8%)\\nTotal written (filtered):        213,561 bp (84.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 78 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.4%\\n  C: 37.2%\\n  G: 23.1%\\n  T: 15.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t59\\t252.8\\t0\\t59\\n2\\t16\\t63.2\\t0\\t16\\n3\\t3\\t15.8\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar482-2019_S134_L001_R2_001.fastq.gz\\n=============================================\\n1011 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar482-2019_S134_L001_R1_001_trimmed.fq.gz and sar482-2019_S134_L001_R2_001_trimmed.fq.gz\\nfile_1: sar482-2019_S134_L001_R1_001_trimmed.fq.gz, file_2: sar482-2019_S134_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar482-2019_S134_L001_R1_001_trimmed.fq.gz and sar482-2019_S134_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar482-2019_S134_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar482-2019_S134_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1011\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 46 (4.55%)\\n\\n\\n  >>> Now running FastQC on the validated data sar482-2019_S134_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar482-2019_S134_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar482-2019_S134_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar482-2019_S134_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar482-2019_S134_L001_R1_001_trimmed.fq.gz and sar482-2019_S134_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar483-2019_S135_L001_R1_001.fastq.gz data/reads/sar483-2019_S135_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar483-2019_S135_L001_R1_001.fastq.gz data/reads/sar483-2019_S135_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar483-2019_S135_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar483-2019_S135_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar483-2019_S135_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar483-2019_S135_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar483-2019_S135_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (39 us/read; 1.53 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     875\\nReads with adapters:                       627 (71.7%)\\nReads written (passing filters):           875 (100.0%)\\n\\nTotal basepairs processed:       219,625 bp\\nQuality-trimmed:                  23,331 bp (10.6%)\\nTotal written (filtered):        195,281 bp (88.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 627 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.4%\\n  C: 13.6%\\n  G: 54.2%\\n  T: 3.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t270\\t218.8\\t0\\t270\\n2\\t332\\t54.7\\t0\\t332\\n3\\t21\\t13.7\\t0\\t21\\n4\\t4\\t3.4\\t0\\t4\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar483-2019_S135_L001_R1_001.fastq.gz\\n=============================================\\n875 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar483-2019_S135_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar483-2019_S135_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar483-2019_S135_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar483-2019_S135_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar483-2019_S135_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (28 us/read; 2.17 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     875\\nReads with adapters:                        91 (10.4%)\\nReads written (passing filters):           875 (100.0%)\\n\\nTotal basepairs processed:       219,625 bp\\nQuality-trimmed:                  36,145 bp (16.5%)\\nTotal written (filtered):        183,376 bp (83.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 91 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.4%\\n  C: 19.8%\\n  G: 17.6%\\n  T: 46.2%\\n  none/other: 1.1%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t81\\t218.8\\t0\\t81\\n2\\t9\\t54.7\\t0\\t9\\n5\\t1\\t0.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar483-2019_S135_L001_R2_001.fastq.gz\\n=============================================\\n875 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar483-2019_S135_L001_R1_001_trimmed.fq.gz and sar483-2019_S135_L001_R2_001_trimmed.fq.gz\\nfile_1: sar483-2019_S135_L001_R1_001_trimmed.fq.gz, file_2: sar483-2019_S135_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar483-2019_S135_L001_R1_001_trimmed.fq.gz and sar483-2019_S135_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar483-2019_S135_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar483-2019_S135_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 875\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 39 (4.46%)\\n\\n\\n  >>> Now running FastQC on the validated data sar483-2019_S135_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar483-2019_S135_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar483-2019_S135_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar483-2019_S135_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar483-2019_S135_L001_R1_001_trimmed.fq.gz and sar483-2019_S135_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar484-2019_S136_L001_R1_001.fastq.gz data/reads/sar484-2019_S136_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar484-2019_S136_L001_R1_001.fastq.gz data/reads/sar484-2019_S136_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar484-2019_S136_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar484-2019_S136_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar484-2019_S136_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar484-2019_S136_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar484-2019_S136_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (44 us/read; 1.35 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     693\\nReads with adapters:                       447 (64.5%)\\nReads written (passing filters):           693 (100.0%)\\n\\nTotal basepairs processed:       173,943 bp\\nQuality-trimmed:                  24,415 bp (14.0%)\\nTotal written (filtered):        148,868 bp (85.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 447 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 31.8%\\n  C: 20.6%\\n  G: 40.5%\\n  T: 7.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t247\\t173.2\\t0\\t247\\n2\\t190\\t43.3\\t0\\t190\\n3\\t8\\t10.8\\t0\\t8\\n4\\t1\\t2.7\\t0\\t1\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar484-2019_S136_L001_R1_001.fastq.gz\\n=============================================\\n693 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar484-2019_S136_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar484-2019_S136_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar484-2019_S136_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar484-2019_S136_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar484-2019_S136_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (37 us/read; 1.63 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     693\\nReads with adapters:                       117 (16.9%)\\nReads written (passing filters):           693 (100.0%)\\n\\nTotal basepairs processed:       173,943 bp\\nQuality-trimmed:                  40,979 bp (23.6%)\\nTotal written (filtered):        132,824 bp (76.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 117 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.2%\\n  C: 39.3%\\n  G: 18.8%\\n  T: 19.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t97\\t173.2\\t0\\t97\\n2\\t17\\t43.3\\t0\\t17\\n3\\t3\\t10.8\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar484-2019_S136_L001_R2_001.fastq.gz\\n=============================================\\n693 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar484-2019_S136_L001_R1_001_trimmed.fq.gz and sar484-2019_S136_L001_R2_001_trimmed.fq.gz\\nfile_1: sar484-2019_S136_L001_R1_001_trimmed.fq.gz, file_2: sar484-2019_S136_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar484-2019_S136_L001_R1_001_trimmed.fq.gz and sar484-2019_S136_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar484-2019_S136_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar484-2019_S136_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 693\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 64 (9.24%)\\n\\n\\n  >>> Now running FastQC on the validated data sar484-2019_S136_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar484-2019_S136_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar484-2019_S136_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar484-2019_S136_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar484-2019_S136_L001_R1_001_trimmed.fq.gz and sar484-2019_S136_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar485-2019_S137_L001_R1_001.fastq.gz data/reads/sar485-2019_S137_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar485-2019_S137_L001_R1_001.fastq.gz data/reads/sar485-2019_S137_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar485-2019_S137_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar485-2019_S137_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar485-2019_S137_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar485-2019_S137_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar485-2019_S137_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     881\\nReads with adapters:                       640 (72.6%)\\nReads written (passing filters):           881 (100.0%)\\n\\nTotal basepairs processed:       221,131 bp\\nQuality-trimmed:                  26,106 bp (11.8%)\\nTotal written (filtered):        194,066 bp (87.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 640 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.2%\\n  C: 18.9%\\n  G: 35.9%\\n  T: 3.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t334\\t220.2\\t0\\t334\\n2\\t293\\t55.1\\t0\\t293\\n3\\t13\\t13.8\\t0\\t13\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar485-2019_S137_L001_R1_001.fastq.gz\\n=============================================\\n881 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar485-2019_S137_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar485-2019_S137_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar485-2019_S137_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar485-2019_S137_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar485-2019_S137_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.29 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     881\\nReads with adapters:                        84 (9.5%)\\nReads written (passing filters):           881 (100.0%)\\n\\nTotal basepairs processed:       221,131 bp\\nQuality-trimmed:                  43,236 bp (19.6%)\\nTotal written (filtered):        177,794 bp (80.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 84 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.0%\\n  C: 26.2%\\n  G: 27.4%\\n  T: 21.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t73\\t220.2\\t0\\t73\\n2\\t6\\t55.1\\t0\\t6\\n3\\t4\\t13.8\\t0\\t4\\n4\\t1\\t3.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar485-2019_S137_L001_R2_001.fastq.gz\\n=============================================\\n881 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar485-2019_S137_L001_R1_001_trimmed.fq.gz and sar485-2019_S137_L001_R2_001_trimmed.fq.gz\\nfile_1: sar485-2019_S137_L001_R1_001_trimmed.fq.gz, file_2: sar485-2019_S137_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar485-2019_S137_L001_R1_001_trimmed.fq.gz and sar485-2019_S137_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar485-2019_S137_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar485-2019_S137_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 881\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 64 (7.26%)\\n\\n\\n  >>> Now running FastQC on the validated data sar485-2019_S137_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar485-2019_S137_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar485-2019_S137_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar485-2019_S137_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar485-2019_S137_L001_R1_001_trimmed.fq.gz and sar485-2019_S137_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar486-2019_S138_L001_R1_001.fastq.gz data/reads/sar486-2019_S138_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar486-2019_S138_L001_R1_001.fastq.gz data/reads/sar486-2019_S138_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar486-2019_S138_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar486-2019_S138_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar486-2019_S138_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar486-2019_S138_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar486-2019_S138_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.93 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,098\\nReads with adapters:                       788 (71.8%)\\nReads written (passing filters):         1,098 (100.0%)\\n\\nTotal basepairs processed:       275,598 bp\\nQuality-trimmed:                  33,563 bp (12.2%)\\nTotal written (filtered):        240,862 bp (87.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 788 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.7%\\n  C: 18.9%\\n  G: 37.7%\\n  T: 4.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t419\\t274.5\\t0\\t419\\n2\\t353\\t68.6\\t0\\t353\\n3\\t16\\t17.2\\t0\\t16\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar486-2019_S138_L001_R1_001.fastq.gz\\n=============================================\\n1098 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar486-2019_S138_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar486-2019_S138_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar486-2019_S138_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar486-2019_S138_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar486-2019_S138_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (41 us/read; 1.46 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,098\\nReads with adapters:                       102 (9.3%)\\nReads written (passing filters):         1,098 (100.0%)\\n\\nTotal basepairs processed:       275,598 bp\\nQuality-trimmed:                  52,004 bp (18.9%)\\nTotal written (filtered):        223,472 bp (81.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 102 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.5%\\n  C: 25.5%\\n  G: 33.3%\\n  T: 15.7%\\n  none/other: 1.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t87\\t274.5\\t0\\t87\\n2\\t11\\t68.6\\t0\\t11\\n3\\t3\\t17.2\\t0\\t3\\n4\\t1\\t4.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar486-2019_S138_L001_R2_001.fastq.gz\\n=============================================\\n1098 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar486-2019_S138_L001_R1_001_trimmed.fq.gz and sar486-2019_S138_L001_R2_001_trimmed.fq.gz\\nfile_1: sar486-2019_S138_L001_R1_001_trimmed.fq.gz, file_2: sar486-2019_S138_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar486-2019_S138_L001_R1_001_trimmed.fq.gz and sar486-2019_S138_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar486-2019_S138_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar486-2019_S138_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1098\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 79 (7.19%)\\n\\n\\n  >>> Now running FastQC on the validated data sar486-2019_S138_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar486-2019_S138_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar486-2019_S138_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar486-2019_S138_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar486-2019_S138_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar486-2019_S138_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar486-2019_S138_L001_R1_001_trimmed.fq.gz and sar486-2019_S138_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar487-2019_S139_L001_R1_001.fastq.gz data/reads/sar487-2019_S139_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar487-2019_S139_L001_R1_001.fastq.gz data/reads/sar487-2019_S139_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar487-2019_S139_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar487-2019_S139_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar487-2019_S139_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar487-2019_S139_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar487-2019_S139_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,303\\nReads with adapters:                     1,033 (79.3%)\\nReads written (passing filters):         1,303 (100.0%)\\n\\nTotal basepairs processed:       327,053 bp\\nQuality-trimmed:                  23,014 bp (7.0%)\\nTotal written (filtered):        302,376 bp (92.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1033 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.5%\\n  C: 14.0%\\n  G: 55.6%\\n  T: 2.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t425\\t325.8\\t0\\t425\\n2\\t589\\t81.4\\t0\\t589\\n3\\t18\\t20.4\\t0\\t18\\n6\\t1\\t0.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar487-2019_S139_L001_R1_001.fastq.gz\\n=============================================\\n1303 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar487-2019_S139_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar487-2019_S139_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar487-2019_S139_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar487-2019_S139_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar487-2019_S139_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (32 us/read; 1.86 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,303\\nReads with adapters:                        68 (5.2%)\\nReads written (passing filters):         1,303 (100.0%)\\n\\nTotal basepairs processed:       327,053 bp\\nQuality-trimmed:                  38,030 bp (11.6%)\\nTotal written (filtered):        288,943 bp (88.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 68 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 17.6%\\n  C: 25.0%\\n  G: 35.3%\\n  T: 20.6%\\n  none/other: 1.5%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t59\\t325.8\\t0\\t59\\n2\\t6\\t81.4\\t0\\t6\\n3\\t3\\t20.4\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar487-2019_S139_L001_R2_001.fastq.gz\\n=============================================\\n1303 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar487-2019_S139_L001_R1_001_trimmed.fq.gz and sar487-2019_S139_L001_R2_001_trimmed.fq.gz\\nfile_1: sar487-2019_S139_L001_R1_001_trimmed.fq.gz, file_2: sar487-2019_S139_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar487-2019_S139_L001_R1_001_trimmed.fq.gz and sar487-2019_S139_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar487-2019_S139_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar487-2019_S139_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1303\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 58 (4.45%)\\n\\n\\n  >>> Now running FastQC on the validated data sar487-2019_S139_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar487-2019_S139_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar487-2019_S139_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar487-2019_S139_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar487-2019_S139_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar487-2019_S139_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar487-2019_S139_L001_R1_001_trimmed.fq.gz and sar487-2019_S139_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar488-2019_S140_L001_R1_001.fastq.gz data/reads/sar488-2019_S140_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar488-2019_S140_L001_R1_001.fastq.gz data/reads/sar488-2019_S140_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar488-2019_S140_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar488-2019_S140_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar488-2019_S140_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar488-2019_S140_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar488-2019_S140_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (33 us/read; 1.80 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     743\\nReads with adapters:                       580 (78.1%)\\nReads written (passing filters):           743 (100.0%)\\n\\nTotal basepairs processed:       186,493 bp\\nQuality-trimmed:                  16,955 bp (9.1%)\\nTotal written (filtered):        168,628 bp (90.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 580 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.6%\\n  C: 12.8%\\n  G: 56.4%\\n  T: 2.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t263\\t185.8\\t0\\t263\\n2\\t306\\t46.4\\t0\\t306\\n3\\t10\\t11.6\\t0\\t10\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar488-2019_S140_L001_R1_001.fastq.gz\\n=============================================\\n743 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar488-2019_S140_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar488-2019_S140_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar488-2019_S140_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar488-2019_S140_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar488-2019_S140_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (47 us/read; 1.29 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     743\\nReads with adapters:                        55 (7.4%)\\nReads written (passing filters):           743 (100.0%)\\n\\nTotal basepairs processed:       186,493 bp\\nQuality-trimmed:                  27,038 bp (14.5%)\\nTotal written (filtered):        159,394 bp (85.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 55 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.5%\\n  C: 20.0%\\n  G: 27.3%\\n  T: 27.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t49\\t185.8\\t0\\t49\\n2\\t6\\t46.4\\t0\\t6\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar488-2019_S140_L001_R2_001.fastq.gz\\n=============================================\\n743 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar488-2019_S140_L001_R1_001_trimmed.fq.gz and sar488-2019_S140_L001_R2_001_trimmed.fq.gz\\nfile_1: sar488-2019_S140_L001_R1_001_trimmed.fq.gz, file_2: sar488-2019_S140_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar488-2019_S140_L001_R1_001_trimmed.fq.gz and sar488-2019_S140_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar488-2019_S140_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar488-2019_S140_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 743\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 27 (3.63%)\\n\\n\\n  >>> Now running FastQC on the validated data sar488-2019_S140_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar488-2019_S140_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar488-2019_S140_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar488-2019_S140_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar488-2019_S140_L001_R1_001_trimmed.fq.gz and sar488-2019_S140_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar489-2019_S141_L001_R1_001.fastq.gz data/reads/sar489-2019_S141_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar489-2019_S141_L001_R1_001.fastq.gz data/reads/sar489-2019_S141_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar489-2019_S141_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar489-2019_S141_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar489-2019_S141_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar489-2019_S141_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar489-2019_S141_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (44 us/read; 1.38 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     833\\nReads with adapters:                       654 (78.5%)\\nReads written (passing filters):           833 (100.0%)\\n\\nTotal basepairs processed:       209,083 bp\\nQuality-trimmed:                  23,399 bp (11.2%)\\nTotal written (filtered):        184,710 bp (88.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 654 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.6%\\n  C: 19.0%\\n  G: 41.0%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t341\\t208.2\\t0\\t341\\n2\\t308\\t52.1\\t0\\t308\\n3\\t3\\t13.0\\t0\\t3\\n4\\t2\\t3.3\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar489-2019_S141_L001_R1_001.fastq.gz\\n=============================================\\n833 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar489-2019_S141_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar489-2019_S141_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar489-2019_S141_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar489-2019_S141_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar489-2019_S141_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.26 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     833\\nReads with adapters:                        56 (6.7%)\\nReads written (passing filters):           833 (100.0%)\\n\\nTotal basepairs processed:       209,083 bp\\nQuality-trimmed:                  41,401 bp (19.8%)\\nTotal written (filtered):        167,618 bp (80.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 56 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.4%\\n  C: 25.0%\\n  G: 25.0%\\n  T: 28.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t48\\t208.2\\t0\\t48\\n2\\t8\\t52.1\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar489-2019_S141_L001_R2_001.fastq.gz\\n=============================================\\n833 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar489-2019_S141_L001_R1_001_trimmed.fq.gz and sar489-2019_S141_L001_R2_001_trimmed.fq.gz\\nfile_1: sar489-2019_S141_L001_R1_001_trimmed.fq.gz, file_2: sar489-2019_S141_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar489-2019_S141_L001_R1_001_trimmed.fq.gz and sar489-2019_S141_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar489-2019_S141_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar489-2019_S141_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 833\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 58 (6.96%)\\n\\n\\n  >>> Now running FastQC on the validated data sar489-2019_S141_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar489-2019_S141_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar489-2019_S141_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar489-2019_S141_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar489-2019_S141_L001_R1_001_trimmed.fq.gz and sar489-2019_S141_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar490-2019_S142_L001_R1_001.fastq.gz data/reads/sar490-2019_S142_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar490-2019_S142_L001_R1_001.fastq.gz data/reads/sar490-2019_S142_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar490-2019_S142_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar490-2019_S142_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar490-2019_S142_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar490-2019_S142_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar490-2019_S142_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (33 us/read; 1.82 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     943\\nReads with adapters:                       741 (78.6%)\\nReads written (passing filters):           943 (100.0%)\\n\\nTotal basepairs processed:       236,693 bp\\nQuality-trimmed:                  18,639 bp (7.9%)\\nTotal written (filtered):        216,903 bp (91.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 741 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.5%\\n  C: 12.0%\\n  G: 56.4%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t338\\t235.8\\t0\\t338\\n2\\t396\\t58.9\\t0\\t396\\n3\\t7\\t14.7\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar490-2019_S142_L001_R1_001.fastq.gz\\n=============================================\\n943 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar490-2019_S142_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar490-2019_S142_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar490-2019_S142_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar490-2019_S142_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar490-2019_S142_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.35 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     943\\nReads with adapters:                        67 (7.1%)\\nReads written (passing filters):           943 (100.0%)\\n\\nTotal basepairs processed:       236,693 bp\\nQuality-trimmed:                  36,412 bp (15.4%)\\nTotal written (filtered):        200,206 bp (84.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 67 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 14.9%\\n  C: 16.4%\\n  G: 44.8%\\n  T: 22.4%\\n  none/other: 1.5%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t60\\t235.8\\t0\\t60\\n2\\t6\\t58.9\\t0\\t6\\n3\\t1\\t14.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar490-2019_S142_L001_R2_001.fastq.gz\\n=============================================\\n943 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar490-2019_S142_L001_R1_001_trimmed.fq.gz and sar490-2019_S142_L001_R2_001_trimmed.fq.gz\\nfile_1: sar490-2019_S142_L001_R1_001_trimmed.fq.gz, file_2: sar490-2019_S142_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar490-2019_S142_L001_R1_001_trimmed.fq.gz and sar490-2019_S142_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar490-2019_S142_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar490-2019_S142_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 943\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 51 (5.41%)\\n\\n\\n  >>> Now running FastQC on the validated data sar490-2019_S142_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar490-2019_S142_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar490-2019_S142_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar490-2019_S142_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar490-2019_S142_L001_R1_001_trimmed.fq.gz and sar490-2019_S142_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar491-2019_S143_L001_R1_001.fastq.gz data/reads/sar491-2019_S143_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar491-2019_S143_L001_R1_001.fastq.gz data/reads/sar491-2019_S143_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar491-2019_S143_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar491-2019_S143_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar491-2019_S143_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar491-2019_S143_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar491-2019_S143_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.91 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,051\\nReads with adapters:                       771 (73.4%)\\nReads written (passing filters):         1,051 (100.0%)\\n\\nTotal basepairs processed:       263,801 bp\\nQuality-trimmed:                  33,847 bp (12.8%)\\nTotal written (filtered):        228,774 bp (86.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 771 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 34.9%\\n  C: 19.3%\\n  G: 38.5%\\n  T: 7.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t393\\t262.8\\t0\\t393\\n2\\t357\\t65.7\\t0\\t357\\n3\\t18\\t16.4\\t0\\t18\\n4\\t1\\t4.1\\t0\\t1\\n5\\t1\\t1.0\\t0\\t1\\n10\\t1\\t0.0\\t1\\t0 1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar491-2019_S143_L001_R1_001.fastq.gz\\n=============================================\\n1051 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar491-2019_S143_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar491-2019_S143_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar491-2019_S143_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar491-2019_S143_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar491-2019_S143_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.31 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,051\\nReads with adapters:                        90 (8.6%)\\nReads written (passing filters):         1,051 (100.0%)\\n\\nTotal basepairs processed:       263,801 bp\\nQuality-trimmed:                  55,318 bp (21.0%)\\nTotal written (filtered):        208,380 bp (79.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 90 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.6%\\n  C: 27.8%\\n  G: 27.8%\\n  T: 27.8%\\n  none/other: 1.1%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t81\\t262.8\\t0\\t81\\n2\\t6\\t65.7\\t0\\t6\\n3\\t2\\t16.4\\t0\\t2\\n4\\t1\\t4.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar491-2019_S143_L001_R2_001.fastq.gz\\n=============================================\\n1051 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar491-2019_S143_L001_R1_001_trimmed.fq.gz and sar491-2019_S143_L001_R2_001_trimmed.fq.gz\\nfile_1: sar491-2019_S143_L001_R1_001_trimmed.fq.gz, file_2: sar491-2019_S143_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar491-2019_S143_L001_R1_001_trimmed.fq.gz and sar491-2019_S143_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar491-2019_S143_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar491-2019_S143_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1051\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 84 (7.99%)\\n\\n\\n  >>> Now running FastQC on the validated data sar491-2019_S143_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar491-2019_S143_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar491-2019_S143_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar491-2019_S143_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar491-2019_S143_L001_R1_001_trimmed.fq.gz and sar491-2019_S143_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar492-2019_S144_L001_R1_001.fastq.gz data/reads/sar492-2019_S144_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar492-2019_S144_L001_R1_001.fastq.gz data/reads/sar492-2019_S144_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar492-2019_S144_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar492-2019_S144_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar492-2019_S144_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar492-2019_S144_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar492-2019_S144_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.96 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,053\\nReads with adapters:                       800 (76.0%)\\nReads written (passing filters):         1,053 (100.0%)\\n\\nTotal basepairs processed:       264,303 bp\\nQuality-trimmed:                  29,802 bp (11.3%)\\nTotal written (filtered):        233,255 bp (88.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 800 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.4%\\n  C: 16.0%\\n  G: 50.2%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t372\\t263.2\\t0\\t372\\n2\\t412\\t65.8\\t0\\t412\\n3\\t14\\t16.5\\t0\\t14\\n4\\t2\\t4.1\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar492-2019_S144_L001_R1_001.fastq.gz\\n=============================================\\n1053 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar492-2019_S144_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar492-2019_S144_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar492-2019_S144_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar492-2019_S144_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar492-2019_S144_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.41 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,053\\nReads with adapters:                        71 (6.7%)\\nReads written (passing filters):         1,053 (100.0%)\\n\\nTotal basepairs processed:       264,303 bp\\nQuality-trimmed:                  44,959 bp (17.0%)\\nTotal written (filtered):        219,261 bp (83.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 71 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.8%\\n  C: 25.4%\\n  G: 22.5%\\n  T: 25.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t61\\t263.2\\t0\\t61\\n2\\t8\\t65.8\\t0\\t8\\n3\\t2\\t16.5\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar492-2019_S144_L001_R2_001.fastq.gz\\n=============================================\\n1053 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar492-2019_S144_L001_R1_001_trimmed.fq.gz and sar492-2019_S144_L001_R2_001_trimmed.fq.gz\\nfile_1: sar492-2019_S144_L001_R1_001_trimmed.fq.gz, file_2: sar492-2019_S144_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar492-2019_S144_L001_R1_001_trimmed.fq.gz and sar492-2019_S144_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar492-2019_S144_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar492-2019_S144_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1053\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 56 (5.32%)\\n\\n\\n  >>> Now running FastQC on the validated data sar492-2019_S144_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar492-2019_S144_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar492-2019_S144_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar492-2019_S144_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar492-2019_S144_L001_R1_001_trimmed.fq.gz and sar492-2019_S144_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar493-2019_S145_L001_R1_001.fastq.gz data/reads/sar493-2019_S145_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar493-2019_S145_L001_R1_001.fastq.gz data/reads/sar493-2019_S145_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar493-2019_S145_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar493-2019_S145_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar493-2019_S145_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar493-2019_S145_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar493-2019_S145_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.89 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     992\\nReads with adapters:                       825 (83.2%)\\nReads written (passing filters):           992 (100.0%)\\n\\nTotal basepairs processed:       248,992 bp\\nQuality-trimmed:                  20,057 bp (8.1%)\\nTotal written (filtered):        227,664 bp (91.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 825 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.7%\\n  C: 18.3%\\n  G: 39.6%\\n  T: 3.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t398\\t248.0\\t0\\t398\\n2\\t409\\t62.0\\t0\\t409\\n3\\t17\\t15.5\\t0\\t17\\n4\\t1\\t3.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar493-2019_S145_L001_R1_001.fastq.gz\\n=============================================\\n992 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar493-2019_S145_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar493-2019_S145_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar493-2019_S145_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar493-2019_S145_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar493-2019_S145_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.35 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     992\\nReads with adapters:                        62 (6.2%)\\nReads written (passing filters):           992 (100.0%)\\n\\nTotal basepairs processed:       248,992 bp\\nQuality-trimmed:                  32,355 bp (13.0%)\\nTotal written (filtered):        216,567 bp (87.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 62 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.4%\\n  C: 19.4%\\n  G: 27.4%\\n  T: 25.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t56\\t248.0\\t0\\t56\\n2\\t4\\t62.0\\t0\\t4\\n3\\t2\\t15.5\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar493-2019_S145_L001_R2_001.fastq.gz\\n=============================================\\n992 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar493-2019_S145_L001_R1_001_trimmed.fq.gz and sar493-2019_S145_L001_R2_001_trimmed.fq.gz\\nfile_1: sar493-2019_S145_L001_R1_001_trimmed.fq.gz, file_2: sar493-2019_S145_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar493-2019_S145_L001_R1_001_trimmed.fq.gz and sar493-2019_S145_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar493-2019_S145_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar493-2019_S145_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 992\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 49 (4.94%)\\n\\n\\n  >>> Now running FastQC on the validated data sar493-2019_S145_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar493-2019_S145_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar493-2019_S145_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar493-2019_S145_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar493-2019_S145_L001_R1_001_trimmed.fq.gz and sar493-2019_S145_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar494-2019_S146_L001_R1_001.fastq.gz data/reads/sar494-2019_S146_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar494-2019_S146_L001_R1_001.fastq.gz data/reads/sar494-2019_S146_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar494-2019_S146_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar494-2019_S146_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar494-2019_S146_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar494-2019_S146_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar494-2019_S146_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,331\\nReads with adapters:                     1,074 (80.7%)\\nReads written (passing filters):         1,331 (100.0%)\\n\\nTotal basepairs processed:       334,081 bp\\nQuality-trimmed:                  23,416 bp (7.0%)\\nTotal written (filtered):        308,971 bp (92.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1074 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.5%\\n  C: 14.2%\\n  G: 54.4%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t468\\t332.8\\t0\\t468\\n2\\t592\\t83.2\\t0\\t592\\n3\\t14\\t20.8\\t0\\t14\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar494-2019_S146_L001_R1_001.fastq.gz\\n=============================================\\n1331 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar494-2019_S146_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar494-2019_S146_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar494-2019_S146_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar494-2019_S146_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar494-2019_S146_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.32 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,331\\nReads with adapters:                        99 (7.4%)\\nReads written (passing filters):         1,331 (100.0%)\\n\\nTotal basepairs processed:       334,081 bp\\nQuality-trimmed:                  42,055 bp (12.6%)\\nTotal written (filtered):        291,911 bp (87.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 99 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 34.3%\\n  C: 13.1%\\n  G: 37.4%\\n  T: 15.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t87\\t332.8\\t0\\t87\\n2\\t9\\t83.2\\t0\\t9\\n3\\t2\\t20.8\\t0\\t2\\n4\\t1\\t5.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar494-2019_S146_L001_R2_001.fastq.gz\\n=============================================\\n1331 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar494-2019_S146_L001_R1_001_trimmed.fq.gz and sar494-2019_S146_L001_R2_001_trimmed.fq.gz\\nfile_1: sar494-2019_S146_L001_R1_001_trimmed.fq.gz, file_2: sar494-2019_S146_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar494-2019_S146_L001_R1_001_trimmed.fq.gz and sar494-2019_S146_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar494-2019_S146_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar494-2019_S146_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1331\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 53 (3.98%)\\n\\n\\n  >>> Now running FastQC on the validated data sar494-2019_S146_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar494-2019_S146_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar494-2019_S146_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar494-2019_S146_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar494-2019_S146_L001_R2_001_val_2.fq.gz\\nApprox 75% complete for sar494-2019_S146_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar494-2019_S146_L001_R1_001_trimmed.fq.gz and sar494-2019_S146_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar495-2019_S147_L001_R1_001.fastq.gz data/reads/sar495-2019_S147_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar495-2019_S147_L001_R1_001.fastq.gz data/reads/sar495-2019_S147_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nInput file 'data/reads/sar495-2019_S147_L001_R1_001.fastq.gz' seems to be completely empty. Consider respecifying!\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar496-2019_S148_L001_R1_001.fastq.gz data/reads/sar496-2019_S148_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar496-2019_S148_L001_R1_001.fastq.gz data/reads/sar496-2019_S148_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar496-2019_S148_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar496-2019_S148_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar496-2019_S148_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar496-2019_S148_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar496-2019_S148_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,324\\nReads with adapters:                     1,010 (76.3%)\\nReads written (passing filters):         1,324 (100.0%)\\n\\nTotal basepairs processed:       332,324 bp\\nQuality-trimmed:                  34,700 bp (10.4%)\\nTotal written (filtered):        296,047 bp (89.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1010 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.9%\\n  C: 14.9%\\n  G: 53.9%\\n  T: 4.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t467\\t331.0\\t0\\t467\\n2\\t520\\t82.8\\t0\\t520\\n3\\t22\\t20.7\\t0\\t22\\n4\\t1\\t5.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar496-2019_S148_L001_R1_001.fastq.gz\\n=============================================\\n1324 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar496-2019_S148_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar496-2019_S148_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar496-2019_S148_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar496-2019_S148_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar496-2019_S148_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.41 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,324\\nReads with adapters:                        99 (7.5%)\\nReads written (passing filters):         1,324 (100.0%)\\n\\nTotal basepairs processed:       332,324 bp\\nQuality-trimmed:                  55,272 bp (16.6%)\\nTotal written (filtered):        276,942 bp (83.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 99 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.2%\\n  C: 17.2%\\n  G: 32.3%\\n  T: 27.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t89\\t331.0\\t0\\t89\\n2\\t9\\t82.8\\t0\\t9\\n3\\t1\\t20.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar496-2019_S148_L001_R2_001.fastq.gz\\n=============================================\\n1324 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar496-2019_S148_L001_R1_001_trimmed.fq.gz and sar496-2019_S148_L001_R2_001_trimmed.fq.gz\\nfile_1: sar496-2019_S148_L001_R1_001_trimmed.fq.gz, file_2: sar496-2019_S148_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar496-2019_S148_L001_R1_001_trimmed.fq.gz and sar496-2019_S148_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar496-2019_S148_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar496-2019_S148_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1324\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 92 (6.95%)\\n\\n\\n  >>> Now running FastQC on the validated data sar496-2019_S148_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar496-2019_S148_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar496-2019_S148_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar496-2019_S148_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar496-2019_S148_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar496-2019_S148_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar496-2019_S148_L001_R1_001_trimmed.fq.gz and sar496-2019_S148_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar497-2019_S149_L001_R1_001.fastq.gz data/reads/sar497-2019_S149_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar497-2019_S149_L001_R1_001.fastq.gz data/reads/sar497-2019_S149_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar497-2019_S149_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar497-2019_S149_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar497-2019_S149_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar497-2019_S149_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar497-2019_S149_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (33 us/read; 1.84 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     854\\nReads with adapters:                       669 (78.3%)\\nReads written (passing filters):           854 (100.0%)\\n\\nTotal basepairs processed:       214,354 bp\\nQuality-trimmed:                  23,142 bp (10.8%)\\nTotal written (filtered):        190,176 bp (88.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 669 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 43.5%\\n  C: 17.0%\\n  G: 34.1%\\n  T: 5.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t310\\t213.5\\t0\\t310\\n2\\t351\\t53.4\\t0\\t351\\n3\\t8\\t13.3\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar497-2019_S149_L001_R1_001.fastq.gz\\n=============================================\\n854 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar497-2019_S149_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar497-2019_S149_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar497-2019_S149_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar497-2019_S149_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar497-2019_S149_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (27 us/read; 2.26 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     854\\nReads with adapters:                        81 (9.5%)\\nReads written (passing filters):           854 (100.0%)\\n\\nTotal basepairs processed:       214,354 bp\\nQuality-trimmed:                  39,517 bp (18.4%)\\nTotal written (filtered):        174,741 bp (81.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 81 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.8%\\n  C: 19.8%\\n  G: 37.0%\\n  T: 23.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t69\\t213.5\\t0\\t69\\n2\\t10\\t53.4\\t0\\t10\\n3\\t1\\t13.3\\t0\\t1\\n4\\t1\\t3.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar497-2019_S149_L001_R2_001.fastq.gz\\n=============================================\\n854 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar497-2019_S149_L001_R1_001_trimmed.fq.gz and sar497-2019_S149_L001_R2_001_trimmed.fq.gz\\nfile_1: sar497-2019_S149_L001_R1_001_trimmed.fq.gz, file_2: sar497-2019_S149_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar497-2019_S149_L001_R1_001_trimmed.fq.gz and sar497-2019_S149_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar497-2019_S149_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar497-2019_S149_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 854\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 66 (7.73%)\\n\\n\\n  >>> Now running FastQC on the validated data sar497-2019_S149_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar497-2019_S149_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar497-2019_S149_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar497-2019_S149_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar497-2019_S149_L001_R1_001_trimmed.fq.gz and sar497-2019_S149_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar498-2019_S150_L001_R1_001.fastq.gz data/reads/sar498-2019_S150_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar498-2019_S150_L001_R1_001.fastq.gz data/reads/sar498-2019_S150_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar498-2019_S150_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar498-2019_S150_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar498-2019_S150_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar498-2019_S150_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar498-2019_S150_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (30 us/read; 1.97 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,131\\nReads with adapters:                       820 (72.5%)\\nReads written (passing filters):         1,131 (100.0%)\\n\\nTotal basepairs processed:       283,881 bp\\nQuality-trimmed:                  37,327 bp (13.1%)\\nTotal written (filtered):        245,300 bp (86.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 820 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.2%\\n  C: 18.9%\\n  G: 32.3%\\n  T: 6.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t404\\t282.8\\t0\\t404\\n2\\t400\\t70.7\\t0\\t400\\n3\\t15\\t17.7\\t0\\t15\\n5\\t1\\t1.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar498-2019_S150_L001_R1_001.fastq.gz\\n=============================================\\n1131 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar498-2019_S150_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar498-2019_S150_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar498-2019_S150_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar498-2019_S150_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar498-2019_S150_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.27 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,131\\nReads with adapters:                       100 (8.8%)\\nReads written (passing filters):         1,131 (100.0%)\\n\\nTotal basepairs processed:       283,881 bp\\nQuality-trimmed:                  64,224 bp (22.6%)\\nTotal written (filtered):        219,541 bp (77.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 100 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.0%\\n  C: 21.0%\\n  G: 18.0%\\n  T: 39.0%\\n  none/other: 1.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t86\\t282.8\\t0\\t86\\n2\\t13\\t70.7\\t0\\t13\\n4\\t1\\t4.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar498-2019_S150_L001_R2_001.fastq.gz\\n=============================================\\n1131 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar498-2019_S150_L001_R1_001_trimmed.fq.gz and sar498-2019_S150_L001_R2_001_trimmed.fq.gz\\nfile_1: sar498-2019_S150_L001_R1_001_trimmed.fq.gz, file_2: sar498-2019_S150_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar498-2019_S150_L001_R1_001_trimmed.fq.gz and sar498-2019_S150_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar498-2019_S150_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar498-2019_S150_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1131\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 104 (9.20%)\\n\\n\\n  >>> Now running FastQC on the validated data sar498-2019_S150_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar498-2019_S150_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar498-2019_S150_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar498-2019_S150_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar498-2019_S150_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar498-2019_S150_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar498-2019_S150_L001_R1_001_trimmed.fq.gz and sar498-2019_S150_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar499-2019_S151_L001_R1_001.fastq.gz data/reads/sar499-2019_S151_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar499-2019_S151_L001_R1_001.fastq.gz data/reads/sar499-2019_S151_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar499-2019_S151_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar499-2019_S151_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar499-2019_S151_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar499-2019_S151_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar499-2019_S151_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.98 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,261\\nReads with adapters:                     1,033 (81.9%)\\nReads written (passing filters):         1,261 (100.0%)\\n\\nTotal basepairs processed:       316,511 bp\\nQuality-trimmed:                  22,239 bp (7.0%)\\nTotal written (filtered):        292,696 bp (92.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1033 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.1%\\n  C: 17.4%\\n  G: 39.7%\\n  T: 3.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t504\\t315.2\\t0\\t504\\n2\\t515\\t78.8\\t0\\t515\\n3\\t14\\t19.7\\t0\\t14\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar499-2019_S151_L001_R1_001.fastq.gz\\n=============================================\\n1261 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar499-2019_S151_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar499-2019_S151_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar499-2019_S151_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar499-2019_S151_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar499-2019_S151_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.40 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,261\\nReads with adapters:                        92 (7.3%)\\nReads written (passing filters):         1,261 (100.0%)\\n\\nTotal basepairs processed:       316,511 bp\\nQuality-trimmed:                  52,456 bp (16.6%)\\nTotal written (filtered):        263,950 bp (83.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 92 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.8%\\n  C: 17.4%\\n  G: 35.9%\\n  T: 23.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t81\\t315.2\\t0\\t81\\n2\\t9\\t78.8\\t0\\t9\\n3\\t2\\t19.7\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar499-2019_S151_L001_R2_001.fastq.gz\\n=============================================\\n1261 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar499-2019_S151_L001_R1_001_trimmed.fq.gz and sar499-2019_S151_L001_R2_001_trimmed.fq.gz\\nfile_1: sar499-2019_S151_L001_R1_001_trimmed.fq.gz, file_2: sar499-2019_S151_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar499-2019_S151_L001_R1_001_trimmed.fq.gz and sar499-2019_S151_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar499-2019_S151_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar499-2019_S151_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1261\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 56 (4.44%)\\n\\n\\n  >>> Now running FastQC on the validated data sar499-2019_S151_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar499-2019_S151_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar499-2019_S151_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar499-2019_S151_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar499-2019_S151_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar499-2019_S151_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar499-2019_S151_L001_R1_001_trimmed.fq.gz and sar499-2019_S151_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar500-2019_S152_L001_R1_001.fastq.gz data/reads/sar500-2019_S152_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar500-2019_S152_L001_R1_001.fastq.gz data/reads/sar500-2019_S152_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar500-2019_S152_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar500-2019_S152_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar500-2019_S152_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar500-2019_S152_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar500-2019_S152_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (29 us/read; 2.04 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,144\\nReads with adapters:                     1,700 (79.3%)\\nReads written (passing filters):         2,144 (100.0%)\\n\\nTotal basepairs processed:       538,144 bp\\nQuality-trimmed:                  47,858 bp (8.9%)\\nTotal written (filtered):        487,686 bp (90.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1700 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.8%\\n  C: 17.9%\\n  G: 38.1%\\n  T: 3.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t827\\t536.0\\t0\\t827\\n2\\t847\\t134.0\\t0\\t847\\n3\\t25\\t33.5\\t0\\t25\\n4\\t1\\t8.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar500-2019_S152_L001_R1_001.fastq.gz\\n=============================================\\n2144 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar500-2019_S152_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar500-2019_S152_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar500-2019_S152_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar500-2019_S152_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar500-2019_S152_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (24 us/read; 2.50 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,144\\nReads with adapters:                       167 (7.8%)\\nReads written (passing filters):         2,144 (100.0%)\\n\\nTotal basepairs processed:       538,144 bp\\nQuality-trimmed:                  78,308 bp (14.6%)\\nTotal written (filtered):        459,641 bp (85.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 167 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.3%\\n  C: 18.6%\\n  G: 28.1%\\n  T: 26.3%\\n  none/other: 0.6%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t143\\t536.0\\t0\\t143\\n2\\t20\\t134.0\\t0\\t20\\n3\\t4\\t33.5\\t0\\t4\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar500-2019_S152_L001_R2_001.fastq.gz\\n=============================================\\n2144 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar500-2019_S152_L001_R1_001_trimmed.fq.gz and sar500-2019_S152_L001_R2_001_trimmed.fq.gz\\nfile_1: sar500-2019_S152_L001_R1_001_trimmed.fq.gz, file_2: sar500-2019_S152_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar500-2019_S152_L001_R1_001_trimmed.fq.gz and sar500-2019_S152_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar500-2019_S152_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar500-2019_S152_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 2144\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 72 (3.36%)\\n\\n\\n  >>> Now running FastQC on the validated data sar500-2019_S152_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar500-2019_S152_L001_R1_001_val_1.fq.gz\\nApprox 45% complete for sar500-2019_S152_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar500-2019_S152_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar500-2019_S152_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar500-2019_S152_L001_R2_001_val_2.fq.gz\\nApprox 45% complete for sar500-2019_S152_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar500-2019_S152_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar500-2019_S152_L001_R1_001_trimmed.fq.gz and sar500-2019_S152_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar501-2019_S153_L001_R1_001.fastq.gz data/reads/sar501-2019_S153_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar501-2019_S153_L001_R1_001.fastq.gz data/reads/sar501-2019_S153_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar501-2019_S153_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar501-2019_S153_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar501-2019_S153_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar501-2019_S153_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar501-2019_S153_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (35 us/read; 1.72 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,502\\nReads with adapters:                     1,302 (86.7%)\\nReads written (passing filters):         1,502 (100.0%)\\n\\nTotal basepairs processed:       377,002 bp\\nQuality-trimmed:                  20,545 bp (5.4%)\\nTotal written (filtered):        354,453 bp (94.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1302 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 34.9%\\n  C: 15.7%\\n  G: 47.2%\\n  T: 2.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t618\\t375.5\\t0\\t618\\n2\\t666\\t93.9\\t0\\t666\\n3\\t18\\t23.5\\t0\\t18\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar501-2019_S153_L001_R1_001.fastq.gz\\n=============================================\\n1502 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar501-2019_S153_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar501-2019_S153_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar501-2019_S153_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar501-2019_S153_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar501-2019_S153_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (25 us/read; 2.44 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,502\\nReads with adapters:                        84 (5.6%)\\nReads written (passing filters):         1,502 (100.0%)\\n\\nTotal basepairs processed:       377,002 bp\\nQuality-trimmed:                  39,056 bp (10.4%)\\nTotal written (filtered):        337,856 bp (89.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 84 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.0%\\n  C: 9.5%\\n  G: 52.4%\\n  T: 19.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t79\\t375.5\\t0\\t79\\n2\\t4\\t93.9\\t0\\t4\\n3\\t1\\t23.5\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar501-2019_S153_L001_R2_001.fastq.gz\\n=============================================\\n1502 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar501-2019_S153_L001_R1_001_trimmed.fq.gz and sar501-2019_S153_L001_R2_001_trimmed.fq.gz\\nfile_1: sar501-2019_S153_L001_R1_001_trimmed.fq.gz, file_2: sar501-2019_S153_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar501-2019_S153_L001_R1_001_trimmed.fq.gz and sar501-2019_S153_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar501-2019_S153_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar501-2019_S153_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1502\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 52 (3.46%)\\n\\n\\n  >>> Now running FastQC on the validated data sar501-2019_S153_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar501-2019_S153_L001_R1_001_val_1.fq.gz\\nApprox 65% complete for sar501-2019_S153_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar501-2019_S153_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar501-2019_S153_L001_R2_001_val_2.fq.gz\\nApprox 65% complete for sar501-2019_S153_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar501-2019_S153_L001_R1_001_trimmed.fq.gz and sar501-2019_S153_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar502-2019_S154_L001_R1_001.fastq.gz data/reads/sar502-2019_S154_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar502-2019_S154_L001_R1_001.fastq.gz data/reads/sar502-2019_S154_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar502-2019_S154_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar502-2019_S154_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar502-2019_S154_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar502-2019_S154_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar502-2019_S154_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.91 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,032\\nReads with adapters:                       823 (79.7%)\\nReads written (passing filters):         1,032 (100.0%)\\n\\nTotal basepairs processed:       259,032 bp\\nQuality-trimmed:                  22,832 bp (8.8%)\\nTotal written (filtered):        234,925 bp (90.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 823 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 43.1%\\n  C: 16.2%\\n  G: 36.1%\\n  T: 4.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t393\\t258.0\\t0\\t393\\n2\\t411\\t64.5\\t0\\t411\\n3\\t17\\t16.1\\t0\\t17\\n4\\t1\\t4.0\\t0\\t1\\n5\\t1\\t1.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar502-2019_S154_L001_R1_001.fastq.gz\\n=============================================\\n1032 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar502-2019_S154_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar502-2019_S154_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar502-2019_S154_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar502-2019_S154_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar502-2019_S154_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.35 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,032\\nReads with adapters:                        68 (6.6%)\\nReads written (passing filters):         1,032 (100.0%)\\n\\nTotal basepairs processed:       259,032 bp\\nQuality-trimmed:                  37,945 bp (14.6%)\\nTotal written (filtered):        221,012 bp (85.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 68 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 13.2%\\n  C: 19.1%\\n  G: 30.9%\\n  T: 36.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t61\\t258.0\\t0\\t61\\n2\\t7\\t64.5\\t0\\t7\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar502-2019_S154_L001_R2_001.fastq.gz\\n=============================================\\n1032 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar502-2019_S154_L001_R1_001_trimmed.fq.gz and sar502-2019_S154_L001_R2_001_trimmed.fq.gz\\nfile_1: sar502-2019_S154_L001_R1_001_trimmed.fq.gz, file_2: sar502-2019_S154_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar502-2019_S154_L001_R1_001_trimmed.fq.gz and sar502-2019_S154_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar502-2019_S154_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar502-2019_S154_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1032\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 50 (4.84%)\\n\\n\\n  >>> Now running FastQC on the validated data sar502-2019_S154_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar502-2019_S154_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar502-2019_S154_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar502-2019_S154_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar502-2019_S154_L001_R1_001_trimmed.fq.gz and sar502-2019_S154_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar503-2019_S155_L001_R1_001.fastq.gz data/reads/sar503-2019_S155_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar503-2019_S155_L001_R1_001.fastq.gz data/reads/sar503-2019_S155_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar503-2019_S155_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar503-2019_S155_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar503-2019_S155_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar503-2019_S155_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar503-2019_S155_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.88 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,026\\nReads with adapters:                       759 (74.0%)\\nReads written (passing filters):         1,026 (100.0%)\\n\\nTotal basepairs processed:       257,526 bp\\nQuality-trimmed:                  25,333 bp (9.8%)\\nTotal written (filtered):        231,063 bp (89.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 759 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.2%\\n  C: 19.5%\\n  G: 40.7%\\n  T: 4.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t399\\t256.5\\t0\\t399\\n2\\t349\\t64.1\\t0\\t349\\n3\\t11\\t16.0\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar503-2019_S155_L001_R1_001.fastq.gz\\n=============================================\\n1026 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar503-2019_S155_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar503-2019_S155_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar503-2019_S155_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar503-2019_S155_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar503-2019_S155_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (28 us/read; 2.18 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,026\\nReads with adapters:                       106 (10.3%)\\nReads written (passing filters):         1,026 (100.0%)\\n\\nTotal basepairs processed:       257,526 bp\\nQuality-trimmed:                  43,781 bp (17.0%)\\nTotal written (filtered):        213,624 bp (83.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 106 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.8%\\n  C: 27.4%\\n  G: 30.2%\\n  T: 22.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t93\\t256.5\\t0\\t93\\n2\\t11\\t64.1\\t0\\t11\\n3\\t2\\t16.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar503-2019_S155_L001_R2_001.fastq.gz\\n=============================================\\n1026 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar503-2019_S155_L001_R1_001_trimmed.fq.gz and sar503-2019_S155_L001_R2_001_trimmed.fq.gz\\nfile_1: sar503-2019_S155_L001_R1_001_trimmed.fq.gz, file_2: sar503-2019_S155_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar503-2019_S155_L001_R1_001_trimmed.fq.gz and sar503-2019_S155_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar503-2019_S155_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar503-2019_S155_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1026\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 45 (4.39%)\\n\\n\\n  >>> Now running FastQC on the validated data sar503-2019_S155_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar503-2019_S155_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar503-2019_S155_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar503-2019_S155_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar503-2019_S155_L001_R1_001_trimmed.fq.gz and sar503-2019_S155_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar504-2019_S156_L001_R1_001.fastq.gz data/reads/sar504-2019_S156_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar504-2019_S156_L001_R1_001.fastq.gz data/reads/sar504-2019_S156_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar504-2019_S156_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar504-2019_S156_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar504-2019_S156_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar504-2019_S156_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar504-2019_S156_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.89 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     866\\nReads with adapters:                       609 (70.3%)\\nReads written (passing filters):           866 (100.0%)\\n\\nTotal basepairs processed:       217,366 bp\\nQuality-trimmed:                  27,535 bp (12.7%)\\nTotal written (filtered):        188,910 bp (86.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 609 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.6%\\n  C: 19.9%\\n  G: 34.8%\\n  T: 4.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t309\\t216.5\\t0\\t309\\n2\\t290\\t54.1\\t0\\t290\\n3\\t9\\t13.5\\t0\\t9\\n5\\t1\\t0.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar504-2019_S156_L001_R1_001.fastq.gz\\n=============================================\\n866 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar504-2019_S156_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar504-2019_S156_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar504-2019_S156_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar504-2019_S156_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar504-2019_S156_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.29 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     866\\nReads with adapters:                        85 (9.8%)\\nReads written (passing filters):           866 (100.0%)\\n\\nTotal basepairs processed:       217,366 bp\\nQuality-trimmed:                  45,636 bp (21.0%)\\nTotal written (filtered):        171,627 bp (79.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 85 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 12.9%\\n  C: 36.5%\\n  G: 22.4%\\n  T: 27.1%\\n  none/other: 1.2%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t72\\t216.5\\t0\\t72\\n2\\t9\\t54.1\\t0\\t9\\n3\\t3\\t13.5\\t0\\t3\\n4\\t1\\t3.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar504-2019_S156_L001_R2_001.fastq.gz\\n=============================================\\n866 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar504-2019_S156_L001_R1_001_trimmed.fq.gz and sar504-2019_S156_L001_R2_001_trimmed.fq.gz\\nfile_1: sar504-2019_S156_L001_R1_001_trimmed.fq.gz, file_2: sar504-2019_S156_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar504-2019_S156_L001_R1_001_trimmed.fq.gz and sar504-2019_S156_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar504-2019_S156_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar504-2019_S156_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 866\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 77 (8.89%)\\n\\n\\n  >>> Now running FastQC on the validated data sar504-2019_S156_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar504-2019_S156_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar504-2019_S156_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar504-2019_S156_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar504-2019_S156_L001_R1_001_trimmed.fq.gz and sar504-2019_S156_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar505-2019_S157_L001_R1_001.fastq.gz data/reads/sar505-2019_S157_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar505-2019_S157_L001_R1_001.fastq.gz data/reads/sar505-2019_S157_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar505-2019_S157_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar505-2019_S157_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar505-2019_S157_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar505-2019_S157_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar505-2019_S157_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,186\\nReads with adapters:                       879 (74.1%)\\nReads written (passing filters):         1,186 (100.0%)\\n\\nTotal basepairs processed:       297,686 bp\\nQuality-trimmed:                  30,493 bp (10.2%)\\nTotal written (filtered):        265,866 bp (89.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 879 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 40.5%\\n  C: 18.0%\\n  G: 36.5%\\n  T: 5.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t452\\t296.5\\t0\\t452\\n2\\t406\\t74.1\\t0\\t406\\n3\\t21\\t18.5\\t0\\t21\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar505-2019_S157_L001_R1_001.fastq.gz\\n=============================================\\n1186 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar505-2019_S157_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar505-2019_S157_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar505-2019_S157_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar505-2019_S157_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar505-2019_S157_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (32 us/read; 1.88 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,186\\nReads with adapters:                       110 (9.3%)\\nReads written (passing filters):         1,186 (100.0%)\\n\\nTotal basepairs processed:       297,686 bp\\nQuality-trimmed:                  52,058 bp (17.5%)\\nTotal written (filtered):        245,504 bp (82.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 110 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.8%\\n  C: 29.1%\\n  G: 24.5%\\n  T: 24.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t99\\t296.5\\t0\\t99\\n2\\t8\\t74.1\\t0\\t8\\n3\\t3\\t18.5\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar505-2019_S157_L001_R2_001.fastq.gz\\n=============================================\\n1186 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar505-2019_S157_L001_R1_001_trimmed.fq.gz and sar505-2019_S157_L001_R2_001_trimmed.fq.gz\\nfile_1: sar505-2019_S157_L001_R1_001_trimmed.fq.gz, file_2: sar505-2019_S157_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar505-2019_S157_L001_R1_001_trimmed.fq.gz and sar505-2019_S157_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar505-2019_S157_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar505-2019_S157_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1186\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 72 (6.07%)\\n\\n\\n  >>> Now running FastQC on the validated data sar505-2019_S157_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar505-2019_S157_L001_R1_001_val_1.fq.gz\\nApprox 90% complete for sar505-2019_S157_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar505-2019_S157_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar505-2019_S157_L001_R2_001_val_2.fq.gz\\nApprox 90% complete for sar505-2019_S157_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar505-2019_S157_L001_R1_001_trimmed.fq.gz and sar505-2019_S157_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar506-2019_S158_L001_R1_001.fastq.gz data/reads/sar506-2019_S158_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar506-2019_S158_L001_R1_001.fastq.gz data/reads/sar506-2019_S158_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar506-2019_S158_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar506-2019_S158_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar506-2019_S158_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar506-2019_S158_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar506-2019_S158_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (32 us/read; 1.85 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     763\\nReads with adapters:                       548 (71.8%)\\nReads written (passing filters):           763 (100.0%)\\n\\nTotal basepairs processed:       191,513 bp\\nQuality-trimmed:                  20,249 bp (10.6%)\\nTotal written (filtered):        170,385 bp (89.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 548 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.4%\\n  C: 14.2%\\n  G: 58.9%\\n  T: 3.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t226\\t190.8\\t0\\t226\\n2\\t315\\t47.7\\t0\\t315\\n3\\t6\\t11.9\\t0\\t6\\n5\\t1\\t0.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar506-2019_S158_L001_R1_001.fastq.gz\\n=============================================\\n763 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar506-2019_S158_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar506-2019_S158_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar506-2019_S158_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar506-2019_S158_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar506-2019_S158_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (27 us/read; 2.25 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     763\\nReads with adapters:                        60 (7.9%)\\nReads written (passing filters):           763 (100.0%)\\n\\nTotal basepairs processed:       191,513 bp\\nQuality-trimmed:                  35,299 bp (18.4%)\\nTotal written (filtered):        156,148 bp (81.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 60 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.0%\\n  C: 35.0%\\n  G: 28.3%\\n  T: 21.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t55\\t190.8\\t0\\t55\\n2\\t4\\t47.7\\t0\\t4\\n3\\t1\\t11.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar506-2019_S158_L001_R2_001.fastq.gz\\n=============================================\\n763 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar506-2019_S158_L001_R1_001_trimmed.fq.gz and sar506-2019_S158_L001_R2_001_trimmed.fq.gz\\nfile_1: sar506-2019_S158_L001_R1_001_trimmed.fq.gz, file_2: sar506-2019_S158_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar506-2019_S158_L001_R1_001_trimmed.fq.gz and sar506-2019_S158_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar506-2019_S158_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar506-2019_S158_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 763\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 47 (6.16%)\\n\\n\\n  >>> Now running FastQC on the validated data sar506-2019_S158_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar506-2019_S158_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar506-2019_S158_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar506-2019_S158_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar506-2019_S158_L001_R1_001_trimmed.fq.gz and sar506-2019_S158_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar507-2019_S159_L001_R1_001.fastq.gz data/reads/sar507-2019_S159_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar507-2019_S159_L001_R1_001.fastq.gz data/reads/sar507-2019_S159_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar507-2019_S159_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar507-2019_S159_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar507-2019_S159_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar507-2019_S159_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar507-2019_S159_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,038\\nReads with adapters:                       783 (75.4%)\\nReads written (passing filters):         1,038 (100.0%)\\n\\nTotal basepairs processed:       260,538 bp\\nQuality-trimmed:                  31,857 bp (12.2%)\\nTotal written (filtered):        227,471 bp (87.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 783 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.2%\\n  C: 17.2%\\n  G: 38.8%\\n  T: 4.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t379\\t259.5\\t0\\t379\\n2\\t382\\t64.9\\t0\\t382\\n3\\t21\\t16.2\\t0\\t21\\n4\\t1\\t4.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar507-2019_S159_L001_R1_001.fastq.gz\\n=============================================\\n1038 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar507-2019_S159_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar507-2019_S159_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar507-2019_S159_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar507-2019_S159_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar507-2019_S159_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.38 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,038\\nReads with adapters:                        87 (8.4%)\\nReads written (passing filters):         1,038 (100.0%)\\n\\nTotal basepairs processed:       260,538 bp\\nQuality-trimmed:                  51,381 bp (19.7%)\\nTotal written (filtered):        209,054 bp (80.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 87 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 18.4%\\n  C: 33.3%\\n  G: 28.7%\\n  T: 19.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t75\\t259.5\\t0\\t75\\n2\\t11\\t64.9\\t0\\t11\\n6\\t1\\t0.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar507-2019_S159_L001_R2_001.fastq.gz\\n=============================================\\n1038 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar507-2019_S159_L001_R1_001_trimmed.fq.gz and sar507-2019_S159_L001_R2_001_trimmed.fq.gz\\nfile_1: sar507-2019_S159_L001_R1_001_trimmed.fq.gz, file_2: sar507-2019_S159_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar507-2019_S159_L001_R1_001_trimmed.fq.gz and sar507-2019_S159_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar507-2019_S159_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar507-2019_S159_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1038\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 75 (7.23%)\\n\\n\\n  >>> Now running FastQC on the validated data sar507-2019_S159_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar507-2019_S159_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar507-2019_S159_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar507-2019_S159_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar507-2019_S159_L001_R1_001_trimmed.fq.gz and sar507-2019_S159_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar508-2019_S160_L001_R1_001.fastq.gz data/reads/sar508-2019_S160_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar508-2019_S160_L001_R1_001.fastq.gz data/reads/sar508-2019_S160_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar508-2019_S160_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar508-2019_S160_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar508-2019_S160_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar508-2019_S160_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar508-2019_S160_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.93 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     963\\nReads with adapters:                       689 (71.5%)\\nReads written (passing filters):           963 (100.0%)\\n\\nTotal basepairs processed:       241,713 bp\\nQuality-trimmed:                  27,472 bp (11.4%)\\nTotal written (filtered):        213,207 bp (88.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 689 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.9%\\n  C: 19.3%\\n  G: 35.3%\\n  T: 6.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t352\\t240.8\\t0\\t352\\n2\\t329\\t60.2\\t0\\t329\\n3\\t8\\t15.0\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar508-2019_S160_L001_R1_001.fastq.gz\\n=============================================\\n963 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar508-2019_S160_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar508-2019_S160_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar508-2019_S160_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar508-2019_S160_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar508-2019_S160_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.33 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     963\\nReads with adapters:                        93 (9.7%)\\nReads written (passing filters):           963 (100.0%)\\n\\nTotal basepairs processed:       241,713 bp\\nQuality-trimmed:                  44,689 bp (18.5%)\\nTotal written (filtered):        196,911 bp (81.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 93 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 25.8%\\n  C: 30.1%\\n  G: 15.1%\\n  T: 29.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t77\\t240.8\\t0\\t77\\n2\\t14\\t60.2\\t0\\t14\\n4\\t2\\t3.8\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar508-2019_S160_L001_R2_001.fastq.gz\\n=============================================\\n963 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar508-2019_S160_L001_R1_001_trimmed.fq.gz and sar508-2019_S160_L001_R2_001_trimmed.fq.gz\\nfile_1: sar508-2019_S160_L001_R1_001_trimmed.fq.gz, file_2: sar508-2019_S160_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar508-2019_S160_L001_R1_001_trimmed.fq.gz and sar508-2019_S160_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar508-2019_S160_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar508-2019_S160_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 963\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 45 (4.67%)\\n\\n\\n  >>> Now running FastQC on the validated data sar508-2019_S160_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar508-2019_S160_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar508-2019_S160_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar508-2019_S160_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar508-2019_S160_L001_R1_001_trimmed.fq.gz and sar508-2019_S160_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar509-2019_S161_L001_R1_001.fastq.gz data/reads/sar509-2019_S161_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar509-2019_S161_L001_R1_001.fastq.gz data/reads/sar509-2019_S161_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar509-2019_S161_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar509-2019_S161_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar509-2019_S161_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar509-2019_S161_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar509-2019_S161_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.03 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,721\\nReads with adapters:                     1,384 (80.4%)\\nReads written (passing filters):         1,721 (100.0%)\\n\\nTotal basepairs processed:       431,971 bp\\nQuality-trimmed:                  37,731 bp (8.7%)\\nTotal written (filtered):        392,126 bp (90.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1384 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 35.6%\\n  C: 17.1%\\n  G: 43.3%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t668\\t430.2\\t0\\t668\\n2\\t705\\t107.6\\t0\\t705\\n3\\t9\\t26.9\\t0\\t9\\n4\\t1\\t6.7\\t0\\t1\\n5\\t1\\t1.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar509-2019_S161_L001_R1_001.fastq.gz\\n=============================================\\n1721 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar509-2019_S161_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar509-2019_S161_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar509-2019_S161_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar509-2019_S161_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar509-2019_S161_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (45 us/read; 1.33 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,721\\nReads with adapters:                       127 (7.4%)\\nReads written (passing filters):         1,721 (100.0%)\\n\\nTotal basepairs processed:       431,971 bp\\nQuality-trimmed:                  70,185 bp (16.2%)\\nTotal written (filtered):        361,637 bp (83.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 127 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 13.4%\\n  C: 24.4%\\n  G: 33.9%\\n  T: 26.0%\\n  none/other: 2.4%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t108\\t430.2\\t0\\t108\\n2\\t17\\t107.6\\t0\\t17\\n3\\t1\\t26.9\\t0\\t1\\n4\\t1\\t6.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar509-2019_S161_L001_R2_001.fastq.gz\\n=============================================\\n1721 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar509-2019_S161_L001_R1_001_trimmed.fq.gz and sar509-2019_S161_L001_R2_001_trimmed.fq.gz\\nfile_1: sar509-2019_S161_L001_R1_001_trimmed.fq.gz, file_2: sar509-2019_S161_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar509-2019_S161_L001_R1_001_trimmed.fq.gz and sar509-2019_S161_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar509-2019_S161_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar509-2019_S161_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1721\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 125 (7.26%)\\n\\n\\n  >>> Now running FastQC on the validated data sar509-2019_S161_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar509-2019_S161_L001_R1_001_val_1.fq.gz\\nApprox 60% complete for sar509-2019_S161_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar509-2019_S161_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar509-2019_S161_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar509-2019_S161_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar509-2019_S161_L001_R1_001_trimmed.fq.gz and sar509-2019_S161_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar510-2019_S162_L001_R1_001.fastq.gz data/reads/sar510-2019_S162_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar510-2019_S162_L001_R1_001.fastq.gz data/reads/sar510-2019_S162_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar510-2019_S162_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar510-2019_S162_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar510-2019_S162_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar510-2019_S162_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar510-2019_S162_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (36 us/read; 1.66 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,162\\nReads with adapters:                       965 (83.0%)\\nReads written (passing filters):         1,162 (100.0%)\\n\\nTotal basepairs processed:       291,662 bp\\nQuality-trimmed:                  19,273 bp (6.6%)\\nTotal written (filtered):        270,882 bp (92.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 965 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 36.1%\\n  C: 17.7%\\n  G: 43.8%\\n  T: 2.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t446\\t290.5\\t0\\t446\\n2\\t496\\t72.6\\t0\\t496\\n3\\t23\\t18.2\\t0\\t23\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar510-2019_S162_L001_R1_001.fastq.gz\\n=============================================\\n1162 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar510-2019_S162_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar510-2019_S162_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar510-2019_S162_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar510-2019_S162_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar510-2019_S162_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.40 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,162\\nReads with adapters:                        67 (5.8%)\\nReads written (passing filters):         1,162 (100.0%)\\n\\nTotal basepairs processed:       291,662 bp\\nQuality-trimmed:                  33,210 bp (11.4%)\\nTotal written (filtered):        258,376 bp (88.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 67 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.9%\\n  C: 16.4%\\n  G: 29.9%\\n  T: 28.4%\\n  none/other: 1.5%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t58\\t290.5\\t0\\t58\\n2\\t9\\t72.6\\t0\\t9\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar510-2019_S162_L001_R2_001.fastq.gz\\n=============================================\\n1162 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar510-2019_S162_L001_R1_001_trimmed.fq.gz and sar510-2019_S162_L001_R2_001_trimmed.fq.gz\\nfile_1: sar510-2019_S162_L001_R1_001_trimmed.fq.gz, file_2: sar510-2019_S162_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar510-2019_S162_L001_R1_001_trimmed.fq.gz and sar510-2019_S162_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar510-2019_S162_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar510-2019_S162_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1162\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 36 (3.10%)\\n\\n\\n  >>> Now running FastQC on the validated data sar510-2019_S162_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar510-2019_S162_L001_R1_001_val_1.fq.gz\\nApprox 85% complete for sar510-2019_S162_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar510-2019_S162_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar510-2019_S162_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar510-2019_S162_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar510-2019_S162_L001_R1_001_trimmed.fq.gz and sar510-2019_S162_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar511-2019_S163_L001_R1_001.fastq.gz data/reads/sar511-2019_S163_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar511-2019_S163_L001_R1_001.fastq.gz data/reads/sar511-2019_S163_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar511-2019_S163_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar511-2019_S163_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar511-2019_S163_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar511-2019_S163_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar511-2019_S163_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (33 us/read; 1.81 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,029\\nReads with adapters:                       834 (81.0%)\\nReads written (passing filters):         1,029 (100.0%)\\n\\nTotal basepairs processed:       258,279 bp\\nQuality-trimmed:                  17,012 bp (6.6%)\\nTotal written (filtered):        239,930 bp (92.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 834 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.1%\\n  C: 12.9%\\n  G: 55.4%\\n  T: 2.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t345\\t257.2\\t0\\t345\\n2\\t475\\t64.3\\t0\\t475\\n3\\t14\\t16.1\\t0\\t14\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar511-2019_S163_L001_R1_001.fastq.gz\\n=============================================\\n1029 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar511-2019_S163_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar511-2019_S163_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar511-2019_S163_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar511-2019_S163_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar511-2019_S163_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (26 us/read; 2.33 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,029\\nReads with adapters:                        53 (5.2%)\\nReads written (passing filters):         1,029 (100.0%)\\n\\nTotal basepairs processed:       258,279 bp\\nQuality-trimmed:                  26,911 bp (10.4%)\\nTotal written (filtered):        231,305 bp (89.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 53 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 20.8%\\n  C: 18.9%\\n  G: 34.0%\\n  T: 26.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t50\\t257.2\\t0\\t50\\n2\\t2\\t64.3\\t0\\t2\\n9\\t1\\t0.0\\t0\\t0 1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar511-2019_S163_L001_R2_001.fastq.gz\\n=============================================\\n1029 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar511-2019_S163_L001_R1_001_trimmed.fq.gz and sar511-2019_S163_L001_R2_001_trimmed.fq.gz\\nfile_1: sar511-2019_S163_L001_R1_001_trimmed.fq.gz, file_2: sar511-2019_S163_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar511-2019_S163_L001_R1_001_trimmed.fq.gz and sar511-2019_S163_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar511-2019_S163_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar511-2019_S163_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1029\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 25 (2.43%)\\n\\n\\n  >>> Now running FastQC on the validated data sar511-2019_S163_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar511-2019_S163_L001_R1_001_val_1.fq.gz\\nApprox 100% complete for sar511-2019_S163_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar511-2019_S163_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar511-2019_S163_L001_R2_001_val_2.fq.gz\\nApprox 100% complete for sar511-2019_S163_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar511-2019_S163_L001_R1_001_trimmed.fq.gz and sar511-2019_S163_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar512-2019_S164_L001_R1_001.fastq.gz data/reads/sar512-2019_S164_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar512-2019_S164_L001_R1_001.fastq.gz data/reads/sar512-2019_S164_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar512-2019_S164_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar512-2019_S164_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar512-2019_S164_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar512-2019_S164_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar512-2019_S164_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (38 us/read; 1.59 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,023\\nReads with adapters:                       782 (76.4%)\\nReads written (passing filters):         1,023 (100.0%)\\n\\nTotal basepairs processed:       256,773 bp\\nQuality-trimmed:                  28,446 bp (11.1%)\\nTotal written (filtered):        227,142 bp (88.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 782 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 43.4%\\n  C: 19.1%\\n  G: 32.5%\\n  T: 5.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t393\\t255.8\\t0\\t393\\n2\\t375\\t63.9\\t0\\t375\\n3\\t14\\t16.0\\t0\\t14\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar512-2019_S164_L001_R1_001.fastq.gz\\n=============================================\\n1023 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar512-2019_S164_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar512-2019_S164_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar512-2019_S164_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar512-2019_S164_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar512-2019_S164_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (29 us/read; 2.10 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,023\\nReads with adapters:                        90 (8.8%)\\nReads written (passing filters):         1,023 (100.0%)\\n\\nTotal basepairs processed:       256,773 bp\\nQuality-trimmed:                  50,431 bp (19.6%)\\nTotal written (filtered):        206,240 bp (80.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 90 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.4%\\n  C: 24.4%\\n  G: 24.4%\\n  T: 26.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t83\\t255.8\\t0\\t83\\n2\\t4\\t63.9\\t0\\t4\\n3\\t2\\t16.0\\t0\\t2\\n5\\t1\\t1.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar512-2019_S164_L001_R2_001.fastq.gz\\n=============================================\\n1023 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar512-2019_S164_L001_R1_001_trimmed.fq.gz and sar512-2019_S164_L001_R2_001_trimmed.fq.gz\\nfile_1: sar512-2019_S164_L001_R1_001_trimmed.fq.gz, file_2: sar512-2019_S164_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar512-2019_S164_L001_R1_001_trimmed.fq.gz and sar512-2019_S164_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar512-2019_S164_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar512-2019_S164_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1023\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 65 (6.35%)\\n\\n\\n  >>> Now running FastQC on the validated data sar512-2019_S164_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar512-2019_S164_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar512-2019_S164_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar512-2019_S164_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar512-2019_S164_L001_R1_001_trimmed.fq.gz and sar512-2019_S164_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar513-2019_S165_L001_R1_001.fastq.gz data/reads/sar513-2019_S165_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar513-2019_S165_L001_R1_001.fastq.gz data/reads/sar513-2019_S165_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar513-2019_S165_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar513-2019_S165_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar513-2019_S165_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar513-2019_S165_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar513-2019_S165_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (31 us/read; 1.92 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,059\\nReads with adapters:                       780 (73.7%)\\nReads written (passing filters):         1,059 (100.0%)\\n\\nTotal basepairs processed:       265,809 bp\\nQuality-trimmed:                  29,709 bp (11.2%)\\nTotal written (filtered):        234,875 bp (88.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 780 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.1%\\n  C: 15.0%\\n  G: 51.0%\\n  T: 5.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t348\\t264.8\\t0\\t348\\n2\\t419\\t66.2\\t0\\t419\\n3\\t13\\t16.5\\t0\\t13\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar513-2019_S165_L001_R1_001.fastq.gz\\n=============================================\\n1059 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar513-2019_S165_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar513-2019_S165_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar513-2019_S165_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar513-2019_S165_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar513-2019_S165_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (25 us/read; 2.36 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,059\\nReads with adapters:                        85 (8.0%)\\nReads written (passing filters):         1,059 (100.0%)\\n\\nTotal basepairs processed:       265,809 bp\\nQuality-trimmed:                  48,153 bp (18.1%)\\nTotal written (filtered):        217,558 bp (81.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 85 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 16.5%\\n  C: 21.2%\\n  G: 30.6%\\n  T: 31.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t74\\t264.8\\t0\\t74\\n2\\t9\\t66.2\\t0\\t9\\n3\\t2\\t16.5\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar513-2019_S165_L001_R2_001.fastq.gz\\n=============================================\\n1059 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar513-2019_S165_L001_R1_001_trimmed.fq.gz and sar513-2019_S165_L001_R2_001_trimmed.fq.gz\\nfile_1: sar513-2019_S165_L001_R1_001_trimmed.fq.gz, file_2: sar513-2019_S165_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar513-2019_S165_L001_R1_001_trimmed.fq.gz and sar513-2019_S165_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar513-2019_S165_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar513-2019_S165_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1059\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 80 (7.55%)\\n\\n\\n  >>> Now running FastQC on the validated data sar513-2019_S165_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar513-2019_S165_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar513-2019_S165_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar513-2019_S165_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar513-2019_S165_L001_R1_001_trimmed.fq.gz and sar513-2019_S165_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar514-2019_S166_L001_R1_001.fastq.gz data/reads/sar514-2019_S166_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar514-2019_S166_L001_R1_001.fastq.gz data/reads/sar514-2019_S166_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar514-2019_S166_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar514-2019_S166_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar514-2019_S166_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar514-2019_S166_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar514-2019_S166_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.01 s (34 us/read; 1.75 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     375\\nReads with adapters:                       240 (64.0%)\\nReads written (passing filters):           375 (100.0%)\\n\\nTotal basepairs processed:        94,125 bp\\nQuality-trimmed:                  18,377 bp (19.5%)\\nTotal written (filtered):         75,384 bp (80.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 240 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.7%\\n  C: 12.5%\\n  G: 49.6%\\n  T: 9.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t121\\t93.8\\t0\\t121\\n2\\t114\\t23.4\\t0\\t114\\n3\\t5\\t5.9\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar514-2019_S166_L001_R1_001.fastq.gz\\n=============================================\\n375 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar514-2019_S166_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar514-2019_S166_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar514-2019_S166_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar514-2019_S166_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar514-2019_S166_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (71 us/read; 0.85 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     375\\nReads with adapters:                        41 (10.9%)\\nReads written (passing filters):           375 (100.0%)\\n\\nTotal basepairs processed:        94,125 bp\\nQuality-trimmed:                  26,680 bp (28.3%)\\nTotal written (filtered):         67,399 bp (71.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 41 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.8%\\n  C: 29.3%\\n  G: 22.0%\\n  T: 22.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t36\\t93.8\\t0\\t36\\n2\\t5\\t23.4\\t0\\t5\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar514-2019_S166_L001_R2_001.fastq.gz\\n=============================================\\n375 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar514-2019_S166_L001_R1_001_trimmed.fq.gz and sar514-2019_S166_L001_R2_001_trimmed.fq.gz\\nfile_1: sar514-2019_S166_L001_R1_001_trimmed.fq.gz, file_2: sar514-2019_S166_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar514-2019_S166_L001_R1_001_trimmed.fq.gz and sar514-2019_S166_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar514-2019_S166_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar514-2019_S166_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 375\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 39 (10.40%)\\n\\n\\n  >>> Now running FastQC on the validated data sar514-2019_S166_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar514-2019_S166_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar514-2019_S166_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar514-2019_S166_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar514-2019_S166_L001_R1_001_trimmed.fq.gz and sar514-2019_S166_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar515-2019_S167_L001_R1_001.fastq.gz data/reads/sar515-2019_S167_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar515-2019_S167_L001_R1_001.fastq.gz data/reads/sar515-2019_S167_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar515-2019_S167_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar515-2019_S167_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar515-2019_S167_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar515-2019_S167_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar515-2019_S167_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (39 us/read; 1.54 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,091\\nReads with adapters:                       773 (70.9%)\\nReads written (passing filters):         1,091 (100.0%)\\n\\nTotal basepairs processed:       273,841 bp\\nQuality-trimmed:                  32,682 bp (11.9%)\\nTotal written (filtered):        239,934 bp (87.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 773 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.9%\\n  C: 14.4%\\n  G: 53.3%\\n  T: 5.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t332\\t272.8\\t0\\t332\\n2\\t430\\t68.2\\t0\\t430\\n3\\t11\\t17.0\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar515-2019_S167_L001_R1_001.fastq.gz\\n=============================================\\n1091 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar515-2019_S167_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar515-2019_S167_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar515-2019_S167_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar515-2019_S167_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar515-2019_S167_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (39 us/read; 1.55 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,091\\nReads with adapters:                        93 (8.5%)\\nReads written (passing filters):         1,091 (100.0%)\\n\\nTotal basepairs processed:       273,841 bp\\nQuality-trimmed:                  56,918 bp (20.8%)\\nTotal written (filtered):        216,814 bp (79.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 93 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 23.7%\\n  C: 25.8%\\n  G: 24.7%\\n  T: 25.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t79\\t272.8\\t0\\t79\\n2\\t12\\t68.2\\t0\\t12\\n3\\t2\\t17.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar515-2019_S167_L001_R2_001.fastq.gz\\n=============================================\\n1091 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar515-2019_S167_L001_R1_001_trimmed.fq.gz and sar515-2019_S167_L001_R2_001_trimmed.fq.gz\\nfile_1: sar515-2019_S167_L001_R1_001_trimmed.fq.gz, file_2: sar515-2019_S167_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar515-2019_S167_L001_R1_001_trimmed.fq.gz and sar515-2019_S167_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar515-2019_S167_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar515-2019_S167_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1091\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 83 (7.61%)\\n\\n\\n  >>> Now running FastQC on the validated data sar515-2019_S167_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar515-2019_S167_L001_R1_001_val_1.fq.gz\\nApprox 100% complete for sar515-2019_S167_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar515-2019_S167_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar515-2019_S167_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar515-2019_S167_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar515-2019_S167_L001_R1_001_trimmed.fq.gz and sar515-2019_S167_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar516-2019_S168_L001_R1_001.fastq.gz data/reads/sar516-2019_S168_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar516-2019_S168_L001_R1_001.fastq.gz data/reads/sar516-2019_S168_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar516-2019_S168_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar516-2019_S168_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar516-2019_S168_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar516-2019_S168_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar516-2019_S168_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 1.98 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,237\\nReads with adapters:                       943 (76.2%)\\nReads written (passing filters):         1,237 (100.0%)\\n\\nTotal basepairs processed:       310,487 bp\\nQuality-trimmed:                  33,752 bp (10.9%)\\nTotal written (filtered):        275,268 bp (88.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 943 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 42.3%\\n  C: 16.9%\\n  G: 37.3%\\n  T: 3.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t449\\t309.2\\t0\\t449\\n2\\t464\\t77.3\\t0\\t464\\n3\\t30\\t19.3\\t0\\t30\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar516-2019_S168_L001_R1_001.fastq.gz\\n=============================================\\n1237 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar516-2019_S168_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar516-2019_S168_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar516-2019_S168_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar516-2019_S168_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar516-2019_S168_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (37 us/read; 1.62 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,237\\nReads with adapters:                        88 (7.1%)\\nReads written (passing filters):         1,237 (100.0%)\\n\\nTotal basepairs processed:       310,487 bp\\nQuality-trimmed:                  59,765 bp (19.2%)\\nTotal written (filtered):        250,620 bp (80.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 88 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.9%\\n  C: 27.3%\\n  G: 31.8%\\n  T: 25.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t76\\t309.2\\t0\\t76\\n2\\t10\\t77.3\\t0\\t10\\n3\\t2\\t19.3\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar516-2019_S168_L001_R2_001.fastq.gz\\n=============================================\\n1237 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar516-2019_S168_L001_R1_001_trimmed.fq.gz and sar516-2019_S168_L001_R2_001_trimmed.fq.gz\\nfile_1: sar516-2019_S168_L001_R1_001_trimmed.fq.gz, file_2: sar516-2019_S168_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar516-2019_S168_L001_R1_001_trimmed.fq.gz and sar516-2019_S168_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar516-2019_S168_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar516-2019_S168_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1237\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 103 (8.33%)\\n\\n\\n  >>> Now running FastQC on the validated data sar516-2019_S168_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar516-2019_S168_L001_R1_001_val_1.fq.gz\\nApprox 85% complete for sar516-2019_S168_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar516-2019_S168_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar516-2019_S168_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar516-2019_S168_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar516-2019_S168_L001_R1_001_trimmed.fq.gz and sar516-2019_S168_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar517-2019_S169_L001_R1_001.fastq.gz data/reads/sar517-2019_S169_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar517-2019_S169_L001_R1_001.fastq.gz data/reads/sar517-2019_S169_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar517-2019_S169_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar517-2019_S169_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar517-2019_S169_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar517-2019_S169_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar517-2019_S169_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (29 us/read; 2.06 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,181\\nReads with adapters:                     1,647 (75.5%)\\nReads written (passing filters):         2,181 (100.0%)\\n\\nTotal basepairs processed:       547,431 bp\\nQuality-trimmed:                  48,814 bp (8.9%)\\nTotal written (filtered):        495,996 bp (90.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1647 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.7%\\n  C: 13.6%\\n  G: 54.6%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t699\\t545.2\\t0\\t699\\n2\\t927\\t136.3\\t0\\t927\\n3\\t19\\t34.1\\t0\\t19\\n4\\t1\\t8.5\\t0\\t1\\n7\\t1\\t0.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar517-2019_S169_L001_R1_001.fastq.gz\\n=============================================\\n2181 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar517-2019_S169_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar517-2019_S169_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar517-2019_S169_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar517-2019_S169_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar517-2019_S169_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (38 us/read; 1.58 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,181\\nReads with adapters:                       176 (8.1%)\\nReads written (passing filters):         2,181 (100.0%)\\n\\nTotal basepairs processed:       547,431 bp\\nQuality-trimmed:                  93,165 bp (17.0%)\\nTotal written (filtered):        454,066 bp (82.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 176 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 20.5%\\n  C: 19.3%\\n  G: 30.1%\\n  T: 29.5%\\n  none/other: 0.6%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t156\\t545.2\\t0\\t156\\n2\\t16\\t136.3\\t0\\t16\\n3\\t4\\t34.1\\t0\\t4\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar517-2019_S169_L001_R2_001.fastq.gz\\n=============================================\\n2181 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar517-2019_S169_L001_R1_001_trimmed.fq.gz and sar517-2019_S169_L001_R2_001_trimmed.fq.gz\\nfile_1: sar517-2019_S169_L001_R1_001_trimmed.fq.gz, file_2: sar517-2019_S169_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar517-2019_S169_L001_R1_001_trimmed.fq.gz and sar517-2019_S169_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar517-2019_S169_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar517-2019_S169_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 2181\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 142 (6.51%)\\n\\n\\n  >>> Now running FastQC on the validated data sar517-2019_S169_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar517-2019_S169_L001_R1_001_val_1.fq.gz\\nApprox 45% complete for sar517-2019_S169_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar517-2019_S169_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar517-2019_S169_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar517-2019_S169_L001_R2_001_val_2.fq.gz\\nApprox 45% complete for sar517-2019_S169_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar517-2019_S169_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar517-2019_S169_L001_R1_001_trimmed.fq.gz and sar517-2019_S169_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar518-2019_S170_L001_R1_001.fastq.gz data/reads/sar518-2019_S170_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar518-2019_S170_L001_R1_001.fastq.gz data/reads/sar518-2019_S170_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar518-2019_S170_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar518-2019_S170_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar518-2019_S170_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar518-2019_S170_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar518-2019_S170_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (29 us/read; 2.06 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,718\\nReads with adapters:                     1,254 (73.0%)\\nReads written (passing filters):         1,718 (100.0%)\\n\\nTotal basepairs processed:       431,218 bp\\nQuality-trimmed:                  39,552 bp (9.2%)\\nTotal written (filtered):        389,647 bp (90.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1254 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.7%\\n  C: 13.6%\\n  G: 55.9%\\n  T: 2.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t514\\t429.5\\t0\\t514\\n2\\t715\\t107.4\\t0\\t715\\n3\\t25\\t26.8\\t0\\t25\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar518-2019_S170_L001_R1_001.fastq.gz\\n=============================================\\n1718 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar518-2019_S170_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar518-2019_S170_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar518-2019_S170_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar518-2019_S170_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar518-2019_S170_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (34 us/read; 1.79 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,718\\nReads with adapters:                        92 (5.4%)\\nReads written (passing filters):         1,718 (100.0%)\\n\\nTotal basepairs processed:       431,218 bp\\nQuality-trimmed:                  75,570 bp (17.5%)\\nTotal written (filtered):        355,538 bp (82.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 92 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 16.3%\\n  C: 15.2%\\n  G: 38.0%\\n  T: 29.3%\\n  none/other: 1.1%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t78\\t429.5\\t0\\t78\\n2\\t11\\t107.4\\t0\\t11\\n3\\t2\\t26.8\\t0\\t2\\n4\\t1\\t6.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar518-2019_S170_L001_R2_001.fastq.gz\\n=============================================\\n1718 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar518-2019_S170_L001_R1_001_trimmed.fq.gz and sar518-2019_S170_L001_R2_001_trimmed.fq.gz\\nfile_1: sar518-2019_S170_L001_R1_001_trimmed.fq.gz, file_2: sar518-2019_S170_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar518-2019_S170_L001_R1_001_trimmed.fq.gz and sar518-2019_S170_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar518-2019_S170_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar518-2019_S170_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1718\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 142 (8.27%)\\n\\n\\n  >>> Now running FastQC on the validated data sar518-2019_S170_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar518-2019_S170_L001_R1_001_val_1.fq.gz\\nApprox 60% complete for sar518-2019_S170_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar518-2019_S170_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar518-2019_S170_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar518-2019_S170_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar518-2019_S170_L001_R1_001_trimmed.fq.gz and sar518-2019_S170_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar519-2019_S171_L001_R1_001.fastq.gz data/reads/sar519-2019_S171_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar519-2019_S171_L001_R1_001.fastq.gz data/reads/sar519-2019_S171_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar519-2019_S171_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar519-2019_S171_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar519-2019_S171_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar519-2019_S171_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar519-2019_S171_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (55 us/read; 1.10 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     761\\nReads with adapters:                       540 (71.0%)\\nReads written (passing filters):           761 (100.0%)\\n\\nTotal basepairs processed:       191,011 bp\\nQuality-trimmed:                  23,892 bp (12.5%)\\nTotal written (filtered):        166,309 bp (87.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 540 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.7%\\n  C: 21.7%\\n  G: 30.9%\\n  T: 5.7%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t278\\t190.2\\t0\\t278\\n2\\t254\\t47.6\\t0\\t254\\n3\\t8\\t11.9\\t0\\t8\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar519-2019_S171_L001_R1_001.fastq.gz\\n=============================================\\n761 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar519-2019_S171_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar519-2019_S171_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar519-2019_S171_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar519-2019_S171_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar519-2019_S171_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (49 us/read; 1.23 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     761\\nReads with adapters:                        62 (8.1%)\\nReads written (passing filters):           761 (100.0%)\\n\\nTotal basepairs processed:       191,011 bp\\nQuality-trimmed:                  44,722 bp (23.4%)\\nTotal written (filtered):        146,202 bp (76.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 62 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 17.7%\\n  C: 24.2%\\n  G: 24.2%\\n  T: 33.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t48\\t190.2\\t0\\t48\\n2\\t6\\t47.6\\t0\\t6\\n3\\t5\\t11.9\\t0\\t5\\n4\\t3\\t3.0\\t0\\t3\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar519-2019_S171_L001_R2_001.fastq.gz\\n=============================================\\n761 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar519-2019_S171_L001_R1_001_trimmed.fq.gz and sar519-2019_S171_L001_R2_001_trimmed.fq.gz\\nfile_1: sar519-2019_S171_L001_R1_001_trimmed.fq.gz, file_2: sar519-2019_S171_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar519-2019_S171_L001_R1_001_trimmed.fq.gz and sar519-2019_S171_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar519-2019_S171_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar519-2019_S171_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 761\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 81 (10.64%)\\n\\n\\n  >>> Now running FastQC on the validated data sar519-2019_S171_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar519-2019_S171_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar519-2019_S171_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar519-2019_S171_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar519-2019_S171_L001_R1_001_trimmed.fq.gz and sar519-2019_S171_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar520-2019_S172_L001_R1_001.fastq.gz data/reads/sar520-2019_S172_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar520-2019_S172_L001_R1_001.fastq.gz data/reads/sar520-2019_S172_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar520-2019_S172_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar520-2019_S172_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar520-2019_S172_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar520-2019_S172_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar520-2019_S172_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,224\\nReads with adapters:                     1,034 (84.5%)\\nReads written (passing filters):         1,224 (100.0%)\\n\\nTotal basepairs processed:       307,224 bp\\nQuality-trimmed:                  19,812 bp (6.4%)\\nTotal written (filtered):        285,838 bp (93.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1034 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 36.3%\\n  C: 18.8%\\n  G: 41.6%\\n  T: 3.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t507\\t306.0\\t0\\t507\\n2\\t514\\t76.5\\t0\\t514\\n3\\t13\\t19.1\\t0\\t13\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar520-2019_S172_L001_R1_001.fastq.gz\\n=============================================\\n1224 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar520-2019_S172_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar520-2019_S172_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar520-2019_S172_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar520-2019_S172_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar520-2019_S172_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (32 us/read; 1.87 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,224\\nReads with adapters:                        67 (5.5%)\\nReads written (passing filters):         1,224 (100.0%)\\n\\nTotal basepairs processed:       307,224 bp\\nQuality-trimmed:                  33,908 bp (11.0%)\\nTotal written (filtered):        273,238 bp (88.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 67 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 22.4%\\n  C: 19.4%\\n  G: 28.4%\\n  T: 29.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t59\\t306.0\\t0\\t59\\n2\\t7\\t76.5\\t0\\t7\\n5\\t1\\t1.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar520-2019_S172_L001_R2_001.fastq.gz\\n=============================================\\n1224 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar520-2019_S172_L001_R1_001_trimmed.fq.gz and sar520-2019_S172_L001_R2_001_trimmed.fq.gz\\nfile_1: sar520-2019_S172_L001_R1_001_trimmed.fq.gz, file_2: sar520-2019_S172_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar520-2019_S172_L001_R1_001_trimmed.fq.gz and sar520-2019_S172_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar520-2019_S172_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar520-2019_S172_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1224\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 42 (3.43%)\\n\\n\\n  >>> Now running FastQC on the validated data sar520-2019_S172_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar520-2019_S172_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar520-2019_S172_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar520-2019_S172_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar520-2019_S172_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar520-2019_S172_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar520-2019_S172_L001_R1_001_trimmed.fq.gz and sar520-2019_S172_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar521-2019_S173_L001_R1_001.fastq.gz data/reads/sar521-2019_S173_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar521-2019_S173_L001_R1_001.fastq.gz data/reads/sar521-2019_S173_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar521-2019_S173_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar521-2019_S173_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar521-2019_S173_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar521-2019_S173_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar521-2019_S173_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (29 us/read; 2.04 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,575\\nReads with adapters:                     2,170 (84.3%)\\nReads written (passing filters):         2,575 (100.0%)\\n\\nTotal basepairs processed:       646,325 bp\\nQuality-trimmed:                  30,751 bp (4.8%)\\nTotal written (filtered):        612,073 bp (94.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 2170 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.2%\\n  C: 12.3%\\n  G: 57.7%\\n  T: 1.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t872\\t643.8\\t0\\t872\\n2\\t1265\\t160.9\\t0\\t1265\\n3\\t33\\t40.2\\t0\\t33\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar521-2019_S173_L001_R1_001.fastq.gz\\n=============================================\\n2575 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar521-2019_S173_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar521-2019_S173_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar521-2019_S173_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar521-2019_S173_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar521-2019_S173_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (33 us/read; 1.82 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,575\\nReads with adapters:                       120 (4.7%)\\nReads written (passing filters):         2,575 (100.0%)\\n\\nTotal basepairs processed:       646,325 bp\\nQuality-trimmed:                  55,217 bp (8.5%)\\nTotal written (filtered):        590,970 bp (91.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 120 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.7%\\n  C: 11.7%\\n  G: 42.5%\\n  T: 24.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t107\\t643.8\\t0\\t107\\n2\\t9\\t160.9\\t0\\t9\\n3\\t3\\t40.2\\t0\\t3\\n4\\t1\\t10.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar521-2019_S173_L001_R2_001.fastq.gz\\n=============================================\\n2575 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar521-2019_S173_L001_R1_001_trimmed.fq.gz and sar521-2019_S173_L001_R2_001_trimmed.fq.gz\\nfile_1: sar521-2019_S173_L001_R1_001_trimmed.fq.gz, file_2: sar521-2019_S173_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar521-2019_S173_L001_R1_001_trimmed.fq.gz and sar521-2019_S173_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar521-2019_S173_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar521-2019_S173_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 2575\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 59 (2.29%)\\n\\n\\n  >>> Now running FastQC on the validated data sar521-2019_S173_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar521-2019_S173_L001_R1_001_val_1.fq.gz\\nApprox 35% complete for sar521-2019_S173_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar521-2019_S173_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar521-2019_S173_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar521-2019_S173_L001_R2_001_val_2.fq.gz\\nApprox 35% complete for sar521-2019_S173_L001_R2_001_val_2.fq.gz\\nApprox 75% complete for sar521-2019_S173_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar521-2019_S173_L001_R1_001_trimmed.fq.gz and sar521-2019_S173_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar522-2019_S174_L001_R1_001.fastq.gz data/reads/sar522-2019_S174_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar522-2019_S174_L001_R1_001.fastq.gz data/reads/sar522-2019_S174_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar522-2019_S174_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar522-2019_S174_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar522-2019_S174_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar522-2019_S174_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar522-2019_S174_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (29 us/read; 2.05 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,835\\nReads with adapters:                     1,460 (79.6%)\\nReads written (passing filters):         1,835 (100.0%)\\n\\nTotal basepairs processed:       460,585 bp\\nQuality-trimmed:                  30,455 bp (6.6%)\\nTotal written (filtered):        427,797 bp (92.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1460 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.5%\\n  C: 13.4%\\n  G: 55.3%\\n  T: 3.9%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t629\\t458.8\\t0\\t629\\n2\\t798\\t114.7\\t0\\t798\\n3\\t28\\t28.7\\t0\\t28\\n4\\t1\\t7.2\\t0\\t1\\n5\\t4\\t1.8\\t0\\t4\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar522-2019_S174_L001_R1_001.fastq.gz\\n=============================================\\n1835 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar522-2019_S174_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar522-2019_S174_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar522-2019_S174_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar522-2019_S174_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar522-2019_S174_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (24 us/read; 2.46 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,835\\nReads with adapters:                       128 (7.0%)\\nReads written (passing filters):         1,835 (100.0%)\\n\\nTotal basepairs processed:       460,585 bp\\nQuality-trimmed:                  55,182 bp (12.0%)\\nTotal written (filtered):        405,260 bp (88.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 128 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 18.8%\\n  C: 19.5%\\n  G: 34.4%\\n  T: 25.8%\\n  none/other: 1.6%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t116\\t458.8\\t0\\t116\\n2\\t10\\t114.7\\t0\\t10\\n3\\t1\\t28.7\\t0\\t1\\n4\\t1\\t7.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar522-2019_S174_L001_R2_001.fastq.gz\\n=============================================\\n1835 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar522-2019_S174_L001_R1_001_trimmed.fq.gz and sar522-2019_S174_L001_R2_001_trimmed.fq.gz\\nfile_1: sar522-2019_S174_L001_R1_001_trimmed.fq.gz, file_2: sar522-2019_S174_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar522-2019_S174_L001_R1_001_trimmed.fq.gz and sar522-2019_S174_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar522-2019_S174_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar522-2019_S174_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1835\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 50 (2.72%)\\n\\n\\n  >>> Now running FastQC on the validated data sar522-2019_S174_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar522-2019_S174_L001_R1_001_val_1.fq.gz\\nApprox 50% complete for sar522-2019_S174_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar522-2019_S174_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar522-2019_S174_L001_R2_001_val_2.fq.gz\\nApprox 55% complete for sar522-2019_S174_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar522-2019_S174_L001_R1_001_trimmed.fq.gz and sar522-2019_S174_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar523-2019_S175_L001_R1_001.fastq.gz data/reads/sar523-2019_S175_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar523-2019_S175_L001_R1_001.fastq.gz data/reads/sar523-2019_S175_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar523-2019_S175_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar523-2019_S175_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar523-2019_S175_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar523-2019_S175_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar523-2019_S175_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 1.98 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,716\\nReads with adapters:                     1,485 (86.5%)\\nReads written (passing filters):         1,716 (100.0%)\\n\\nTotal basepairs processed:       430,716 bp\\nQuality-trimmed:                  25,252 bp (5.9%)\\nTotal written (filtered):        403,211 bp (93.6%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1485 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.3%\\n  C: 17.4%\\n  G: 40.0%\\n  T: 3.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t741\\t429.0\\t0\\t741\\n2\\t720\\t107.2\\t0\\t720\\n3\\t24\\t26.8\\t0\\t24\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar523-2019_S175_L001_R1_001.fastq.gz\\n=============================================\\n1716 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar523-2019_S175_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar523-2019_S175_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar523-2019_S175_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar523-2019_S175_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar523-2019_S175_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (24 us/read; 2.48 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,716\\nReads with adapters:                        87 (5.1%)\\nReads written (passing filters):         1,716 (100.0%)\\n\\nTotal basepairs processed:       430,716 bp\\nQuality-trimmed:                  47,650 bp (11.1%)\\nTotal written (filtered):        382,968 bp (88.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 87 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.5%\\n  C: 21.8%\\n  G: 41.4%\\n  T: 17.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t77\\t429.0\\t0\\t77\\n2\\t9\\t107.2\\t0\\t9\\n3\\t1\\t26.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar523-2019_S175_L001_R2_001.fastq.gz\\n=============================================\\n1716 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar523-2019_S175_L001_R1_001_trimmed.fq.gz and sar523-2019_S175_L001_R2_001_trimmed.fq.gz\\nfile_1: sar523-2019_S175_L001_R1_001_trimmed.fq.gz, file_2: sar523-2019_S175_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar523-2019_S175_L001_R1_001_trimmed.fq.gz and sar523-2019_S175_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar523-2019_S175_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar523-2019_S175_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1716\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 65 (3.79%)\\n\\n\\n  >>> Now running FastQC on the validated data sar523-2019_S175_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar523-2019_S175_L001_R1_001_val_1.fq.gz\\nApprox 55% complete for sar523-2019_S175_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar523-2019_S175_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar523-2019_S175_L001_R2_001_val_2.fq.gz\\nApprox 60% complete for sar523-2019_S175_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar523-2019_S175_L001_R1_001_trimmed.fq.gz and sar523-2019_S175_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar524-2019_S176_L001_R1_001.fastq.gz data/reads/sar524-2019_S176_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar524-2019_S176_L001_R1_001.fastq.gz data/reads/sar524-2019_S176_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar524-2019_S176_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar524-2019_S176_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar524-2019_S176_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar524-2019_S176_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar524-2019_S176_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (32 us/read; 1.90 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     874\\nReads with adapters:                       746 (85.4%)\\nReads written (passing filters):           874 (100.0%)\\n\\nTotal basepairs processed:       219,374 bp\\nQuality-trimmed:                  13,475 bp (6.1%)\\nTotal written (filtered):        204,726 bp (93.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 746 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 37.5%\\n  C: 16.0%\\n  G: 42.0%\\n  T: 4.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t339\\t218.5\\t0\\t339\\n2\\t393\\t54.6\\t0\\t393\\n3\\t13\\t13.7\\t0\\t13\\n9\\t1\\t0.0\\t0\\t0 1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar524-2019_S176_L001_R1_001.fastq.gz\\n=============================================\\n874 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar524-2019_S176_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar524-2019_S176_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar524-2019_S176_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar524-2019_S176_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar524-2019_S176_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.02 s (26 us/read; 2.32 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                     874\\nReads with adapters:                        57 (6.5%)\\nReads written (passing filters):           874 (100.0%)\\n\\nTotal basepairs processed:       219,374 bp\\nQuality-trimmed:                  30,999 bp (14.1%)\\nTotal written (filtered):        188,312 bp (85.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 57 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.3%\\n  C: 17.5%\\n  G: 45.6%\\n  T: 17.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t51\\t218.5\\t0\\t51\\n2\\t6\\t54.6\\t0\\t6\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar524-2019_S176_L001_R2_001.fastq.gz\\n=============================================\\n874 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar524-2019_S176_L001_R1_001_trimmed.fq.gz and sar524-2019_S176_L001_R2_001_trimmed.fq.gz\\nfile_1: sar524-2019_S176_L001_R1_001_trimmed.fq.gz, file_2: sar524-2019_S176_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar524-2019_S176_L001_R1_001_trimmed.fq.gz and sar524-2019_S176_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar524-2019_S176_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar524-2019_S176_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 874\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 23 (2.63%)\\n\\n\\n  >>> Now running FastQC on the validated data sar524-2019_S176_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar524-2019_S176_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar524-2019_S176_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar524-2019_S176_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar524-2019_S176_L001_R1_001_trimmed.fq.gz and sar524-2019_S176_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar525-2019_S177_L001_R1_001.fastq.gz data/reads/sar525-2019_S177_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar525-2019_S177_L001_R1_001.fastq.gz data/reads/sar525-2019_S177_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar525-2019_S177_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar525-2019_S177_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar525-2019_S177_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar525-2019_S177_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar525-2019_S177_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,848\\nReads with adapters:                     1,456 (78.8%)\\nReads written (passing filters):         1,848 (100.0%)\\n\\nTotal basepairs processed:       463,848 bp\\nQuality-trimmed:                  31,073 bp (6.7%)\\nTotal written (filtered):        430,460 bp (92.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1456 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 29.4%\\n  C: 13.7%\\n  G: 53.7%\\n  T: 3.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t620\\t462.0\\t0\\t620\\n2\\t815\\t115.5\\t0\\t815\\n3\\t20\\t28.9\\t0\\t20\\n5\\t1\\t1.8\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar525-2019_S177_L001_R1_001.fastq.gz\\n=============================================\\n1848 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar525-2019_S177_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar525-2019_S177_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar525-2019_S177_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar525-2019_S177_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar525-2019_S177_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (24 us/read; 2.49 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,848\\nReads with adapters:                       115 (6.2%)\\nReads written (passing filters):         1,848 (100.0%)\\n\\nTotal basepairs processed:       463,848 bp\\nQuality-trimmed:                  62,470 bp (13.5%)\\nTotal written (filtered):        401,250 bp (86.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 115 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.7%\\n  C: 20.9%\\n  G: 33.0%\\n  T: 23.5%\\n  none/other: 0.9%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t104\\t462.0\\t0\\t104\\n2\\t10\\t115.5\\t0\\t10\\n4\\t1\\t7.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar525-2019_S177_L001_R2_001.fastq.gz\\n=============================================\\n1848 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar525-2019_S177_L001_R1_001_trimmed.fq.gz and sar525-2019_S177_L001_R2_001_trimmed.fq.gz\\nfile_1: sar525-2019_S177_L001_R1_001_trimmed.fq.gz, file_2: sar525-2019_S177_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar525-2019_S177_L001_R1_001_trimmed.fq.gz and sar525-2019_S177_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar525-2019_S177_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar525-2019_S177_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1848\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 123 (6.66%)\\n\\n\\n  >>> Now running FastQC on the validated data sar525-2019_S177_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar525-2019_S177_L001_R1_001_val_1.fq.gz\\nApprox 55% complete for sar525-2019_S177_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar525-2019_S177_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar525-2019_S177_L001_R2_001_val_2.fq.gz\\nApprox 55% complete for sar525-2019_S177_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar525-2019_S177_L001_R1_001_trimmed.fq.gz and sar525-2019_S177_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar526-2019_S178_L001_R1_001.fastq.gz data/reads/sar526-2019_S178_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar526-2019_S178_L001_R1_001.fastq.gz data/reads/sar526-2019_S178_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar526-2019_S178_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar526-2019_S178_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar526-2019_S178_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar526-2019_S178_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar526-2019_S178_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 1.99 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,569\\nReads with adapters:                     1,232 (78.5%)\\nReads written (passing filters):         1,569 (100.0%)\\n\\nTotal basepairs processed:       393,819 bp\\nQuality-trimmed:                  32,975 bp (8.4%)\\nTotal written (filtered):        358,953 bp (91.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1232 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 36.4%\\n  C: 17.4%\\n  G: 42.2%\\n  T: 4.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t594\\t392.2\\t0\\t594\\n2\\t617\\t98.1\\t0\\t617\\n3\\t21\\t24.5\\t0\\t21\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar526-2019_S178_L001_R1_001.fastq.gz\\n=============================================\\n1569 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar526-2019_S178_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar526-2019_S178_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar526-2019_S178_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar526-2019_S178_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar526-2019_S178_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (38 us/read; 1.60 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,569\\nReads with adapters:                       102 (6.5%)\\nReads written (passing filters):         1,569 (100.0%)\\n\\nTotal basepairs processed:       393,819 bp\\nQuality-trimmed:                  63,206 bp (16.0%)\\nTotal written (filtered):        330,491 bp (83.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 102 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 17.6%\\n  C: 22.5%\\n  G: 22.5%\\n  T: 37.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t88\\t392.2\\t0\\t88\\n2\\t9\\t98.1\\t0\\t9\\n3\\t4\\t24.5\\t0\\t4\\n4\\t1\\t6.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar526-2019_S178_L001_R2_001.fastq.gz\\n=============================================\\n1569 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar526-2019_S178_L001_R1_001_trimmed.fq.gz and sar526-2019_S178_L001_R2_001_trimmed.fq.gz\\nfile_1: sar526-2019_S178_L001_R1_001_trimmed.fq.gz, file_2: sar526-2019_S178_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar526-2019_S178_L001_R1_001_trimmed.fq.gz and sar526-2019_S178_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar526-2019_S178_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar526-2019_S178_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1569\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 105 (6.69%)\\n\\n\\n  >>> Now running FastQC on the validated data sar526-2019_S178_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar526-2019_S178_L001_R1_001_val_1.fq.gz\\nApprox 65% complete for sar526-2019_S178_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar526-2019_S178_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar526-2019_S178_L001_R2_001_val_2.fq.gz\\nApprox 65% complete for sar526-2019_S178_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar526-2019_S178_L001_R1_001_trimmed.fq.gz and sar526-2019_S178_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar527-2019_S179_L001_R1_001.fastq.gz data/reads/sar527-2019_S179_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar527-2019_S179_L001_R1_001.fastq.gz data/reads/sar527-2019_S179_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar527-2019_S179_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar527-2019_S179_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar527-2019_S179_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar527-2019_S179_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar527-2019_S179_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.08 s (37 us/read; 1.62 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,155\\nReads with adapters:                     1,885 (87.5%)\\nReads written (passing filters):         2,155 (100.0%)\\n\\nTotal basepairs processed:       540,905 bp\\nQuality-trimmed:                  22,157 bp (4.1%)\\nTotal written (filtered):        515,705 bp (95.3%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1885 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 27.2%\\n  C: 11.7%\\n  G: 59.0%\\n  T: 2.1%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t764\\t538.8\\t0\\t764\\n2\\t1086\\t134.7\\t0\\t1086\\n3\\t34\\t33.7\\t0\\t34\\n5\\t1\\t2.1\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar527-2019_S179_L001_R1_001.fastq.gz\\n=============================================\\n2155 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar527-2019_S179_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar527-2019_S179_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar527-2019_S179_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar527-2019_S179_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar527-2019_S179_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (24 us/read; 2.50 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   2,155\\nReads with adapters:                       104 (4.8%)\\nReads written (passing filters):         2,155 (100.0%)\\n\\nTotal basepairs processed:       540,905 bp\\nQuality-trimmed:                  39,493 bp (7.3%)\\nTotal written (filtered):        501,297 bp (92.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 104 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 16.3%\\n  C: 34.6%\\n  G: 32.7%\\n  T: 16.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t93\\t538.8\\t0\\t93\\n2\\t11\\t134.7\\t0\\t11\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar527-2019_S179_L001_R2_001.fastq.gz\\n=============================================\\n2155 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar527-2019_S179_L001_R1_001_trimmed.fq.gz and sar527-2019_S179_L001_R2_001_trimmed.fq.gz\\nfile_1: sar527-2019_S179_L001_R1_001_trimmed.fq.gz, file_2: sar527-2019_S179_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar527-2019_S179_L001_R1_001_trimmed.fq.gz and sar527-2019_S179_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar527-2019_S179_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar527-2019_S179_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 2155\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 52 (2.41%)\\n\\n\\n  >>> Now running FastQC on the validated data sar527-2019_S179_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar527-2019_S179_L001_R1_001_val_1.fq.gz\\nApprox 45% complete for sar527-2019_S179_L001_R1_001_val_1.fq.gz\\nApprox 95% complete for sar527-2019_S179_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar527-2019_S179_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar527-2019_S179_L001_R2_001_val_2.fq.gz\\nApprox 45% complete for sar527-2019_S179_L001_R2_001_val_2.fq.gz\\nApprox 95% complete for sar527-2019_S179_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar527-2019_S179_L001_R1_001_trimmed.fq.gz and sar527-2019_S179_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar528-2019_S180_L001_R1_001.fastq.gz data/reads/sar528-2019_S180_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar528-2019_S180_L001_R1_001.fastq.gz data/reads/sar528-2019_S180_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar528-2019_S180_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar528-2019_S180_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar528-2019_S180_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar528-2019_S180_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar528-2019_S180_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.95 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,283\\nReads with adapters:                     1,101 (85.8%)\\nReads written (passing filters):         1,283 (100.0%)\\n\\nTotal basepairs processed:       322,033 bp\\nQuality-trimmed:                  18,170 bp (5.6%)\\nTotal written (filtered):        302,209 bp (93.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1101 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.2%\\n  C: 18.1%\\n  G: 38.3%\\n  T: 2.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t565\\t320.8\\t0\\t565\\n2\\t519\\t80.2\\t0\\t519\\n3\\t17\\t20.0\\t0\\t17\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar528-2019_S180_L001_R1_001.fastq.gz\\n=============================================\\n1283 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar528-2019_S180_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar528-2019_S180_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar528-2019_S180_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar528-2019_S180_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar528-2019_S180_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (24 us/read; 2.47 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,283\\nReads with adapters:                        61 (4.8%)\\nReads written (passing filters):         1,283 (100.0%)\\n\\nTotal basepairs processed:       322,033 bp\\nQuality-trimmed:                  37,286 bp (11.6%)\\nTotal written (filtered):        284,675 bp (88.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 61 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 26.2%\\n  C: 21.3%\\n  G: 37.7%\\n  T: 14.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t52\\t320.8\\t0\\t52\\n2\\t7\\t80.2\\t0\\t7\\n3\\t2\\t20.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar528-2019_S180_L001_R2_001.fastq.gz\\n=============================================\\n1283 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar528-2019_S180_L001_R1_001_trimmed.fq.gz and sar528-2019_S180_L001_R2_001_trimmed.fq.gz\\nfile_1: sar528-2019_S180_L001_R1_001_trimmed.fq.gz, file_2: sar528-2019_S180_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar528-2019_S180_L001_R1_001_trimmed.fq.gz and sar528-2019_S180_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar528-2019_S180_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar528-2019_S180_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1283\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 58 (4.52%)\\n\\n\\n  >>> Now running FastQC on the validated data sar528-2019_S180_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar528-2019_S180_L001_R1_001_val_1.fq.gz\\nApprox 80% complete for sar528-2019_S180_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar528-2019_S180_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar528-2019_S180_L001_R2_001_val_2.fq.gz\\nApprox 80% complete for sar528-2019_S180_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar528-2019_S180_L001_R1_001_trimmed.fq.gz and sar528-2019_S180_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar529-2019_S181_L001_R1_001.fastq.gz data/reads/sar529-2019_S181_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar529-2019_S181_L001_R1_001.fastq.gz data/reads/sar529-2019_S181_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar529-2019_S181_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar529-2019_S181_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar529-2019_S181_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar529-2019_S181_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar529-2019_S181_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (30 us/read; 1.98 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,837\\nReads with adapters:                     1,609 (87.6%)\\nReads written (passing filters):         1,837 (100.0%)\\n\\nTotal basepairs processed:       461,087 bp\\nQuality-trimmed:                  21,500 bp (4.7%)\\nTotal written (filtered):        437,120 bp (94.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1609 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 38.4%\\n  C: 16.5%\\n  G: 42.4%\\n  T: 2.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t770\\t459.2\\t0\\t770\\n2\\t820\\t114.8\\t0\\t820\\n3\\t19\\t28.7\\t0\\t19\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar529-2019_S181_L001_R1_001.fastq.gz\\n=============================================\\n1837 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar529-2019_S181_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar529-2019_S181_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar529-2019_S181_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar529-2019_S181_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar529-2019_S181_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (33 us/read; 1.84 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,837\\nReads with adapters:                        76 (4.1%)\\nReads written (passing filters):         1,837 (100.0%)\\n\\nTotal basepairs processed:       461,087 bp\\nQuality-trimmed:                  35,975 bp (7.8%)\\nTotal written (filtered):        425,027 bp (92.2%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 76 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 15.8%\\n  C: 17.1%\\n  G: 42.1%\\n  T: 23.7%\\n  none/other: 1.3%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t68\\t459.2\\t0\\t68\\n2\\t7\\t114.8\\t0\\t7\\n3\\t1\\t28.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar529-2019_S181_L001_R2_001.fastq.gz\\n=============================================\\n1837 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar529-2019_S181_L001_R1_001_trimmed.fq.gz and sar529-2019_S181_L001_R2_001_trimmed.fq.gz\\nfile_1: sar529-2019_S181_L001_R1_001_trimmed.fq.gz, file_2: sar529-2019_S181_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar529-2019_S181_L001_R1_001_trimmed.fq.gz and sar529-2019_S181_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar529-2019_S181_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar529-2019_S181_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1837\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 37 (2.01%)\\n\\n\\n  >>> Now running FastQC on the validated data sar529-2019_S181_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar529-2019_S181_L001_R1_001_val_1.fq.gz\\nApprox 50% complete for sar529-2019_S181_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar529-2019_S181_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar529-2019_S181_L001_R2_001_val_2.fq.gz\\nApprox 55% complete for sar529-2019_S181_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar529-2019_S181_L001_R1_001_trimmed.fq.gz and sar529-2019_S181_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar530-2019_S182_L001_R1_001.fastq.gz data/reads/sar530-2019_S182_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar530-2019_S182_L001_R1_001.fastq.gz data/reads/sar530-2019_S182_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar530-2019_S182_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar530-2019_S182_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar530-2019_S182_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar530-2019_S182_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar530-2019_S182_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (30 us/read; 2.03 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,973\\nReads with adapters:                     1,644 (83.3%)\\nReads written (passing filters):         1,973 (100.0%)\\n\\nTotal basepairs processed:       495,223 bp\\nQuality-trimmed:                  22,869 bp (4.6%)\\nTotal written (filtered):        469,710 bp (94.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1644 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 28.3%\\n  C: 12.6%\\n  G: 56.7%\\n  T: 2.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t674\\t493.2\\t0\\t674\\n2\\t942\\t123.3\\t0\\t942\\n3\\t26\\t30.8\\t0\\t26\\n4\\t2\\t7.7\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar530-2019_S182_L001_R1_001.fastq.gz\\n=============================================\\n1973 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar530-2019_S182_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar530-2019_S182_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar530-2019_S182_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar530-2019_S182_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar530-2019_S182_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (24 us/read; 2.52 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,973\\nReads with adapters:                       110 (5.6%)\\nReads written (passing filters):         1,973 (100.0%)\\n\\nTotal basepairs processed:       495,223 bp\\nQuality-trimmed:                  39,452 bp (8.0%)\\nTotal written (filtered):        455,640 bp (92.0%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 110 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 17.3%\\n  C: 27.3%\\n  G: 39.1%\\n  T: 16.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t97\\t493.2\\t0\\t97\\n2\\t11\\t123.3\\t0\\t11\\n4\\t1\\t7.7\\t0\\t1\\n8\\t1\\t0.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar530-2019_S182_L001_R2_001.fastq.gz\\n=============================================\\n1973 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar530-2019_S182_L001_R1_001_trimmed.fq.gz and sar530-2019_S182_L001_R2_001_trimmed.fq.gz\\nfile_1: sar530-2019_S182_L001_R1_001_trimmed.fq.gz, file_2: sar530-2019_S182_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar530-2019_S182_L001_R1_001_trimmed.fq.gz and sar530-2019_S182_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar530-2019_S182_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar530-2019_S182_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1973\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 56 (2.84%)\\n\\n\\n  >>> Now running FastQC on the validated data sar530-2019_S182_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar530-2019_S182_L001_R1_001_val_1.fq.gz\\nApprox 50% complete for sar530-2019_S182_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar530-2019_S182_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar530-2019_S182_L001_R2_001_val_2.fq.gz\\nApprox 50% complete for sar530-2019_S182_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar530-2019_S182_L001_R1_001_trimmed.fq.gz and sar530-2019_S182_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar531-2019_S183_L001_R1_001.fastq.gz data/reads/sar531-2019_S183_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar531-2019_S183_L001_R1_001.fastq.gz data/reads/sar531-2019_S183_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar531-2019_S183_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar531-2019_S183_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar531-2019_S183_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar531-2019_S183_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar531-2019_S183_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (30 us/read; 2.00 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,498\\nReads with adapters:                     1,178 (78.6%)\\nReads written (passing filters):         1,498 (100.0%)\\n\\nTotal basepairs processed:       375,998 bp\\nQuality-trimmed:                  32,896 bp (8.7%)\\nTotal written (filtered):        341,274 bp (90.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1178 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.0%\\n  C: 16.4%\\n  G: 41.0%\\n  T: 3.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t549\\t374.5\\t0\\t549\\n2\\t609\\t93.6\\t0\\t609\\n3\\t19\\t23.4\\t0\\t19\\n4\\t1\\t5.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar531-2019_S183_L001_R1_001.fastq.gz\\n=============================================\\n1498 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar531-2019_S183_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar531-2019_S183_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar531-2019_S183_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar531-2019_S183_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar531-2019_S183_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (25 us/read; 2.42 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,498\\nReads with adapters:                        92 (6.1%)\\nReads written (passing filters):         1,498 (100.0%)\\n\\nTotal basepairs processed:       375,998 bp\\nQuality-trimmed:                  64,232 bp (17.1%)\\nTotal written (filtered):        311,661 bp (82.9%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 92 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 20.7%\\n  C: 20.7%\\n  G: 29.3%\\n  T: 29.3%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t81\\t374.5\\t0\\t81\\n2\\t9\\t93.6\\t0\\t9\\n3\\t2\\t23.4\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar531-2019_S183_L001_R2_001.fastq.gz\\n=============================================\\n1498 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar531-2019_S183_L001_R1_001_trimmed.fq.gz and sar531-2019_S183_L001_R2_001_trimmed.fq.gz\\nfile_1: sar531-2019_S183_L001_R1_001_trimmed.fq.gz, file_2: sar531-2019_S183_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar531-2019_S183_L001_R1_001_trimmed.fq.gz and sar531-2019_S183_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar531-2019_S183_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar531-2019_S183_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1498\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 120 (8.01%)\\n\\n\\n  >>> Now running FastQC on the validated data sar531-2019_S183_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar531-2019_S183_L001_R1_001_val_1.fq.gz\\nApprox 70% complete for sar531-2019_S183_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar531-2019_S183_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar531-2019_S183_L001_R2_001_val_2.fq.gz\\nApprox 70% complete for sar531-2019_S183_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar531-2019_S183_L001_R1_001_trimmed.fq.gz and sar531-2019_S183_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar532-2019_S184_L001_R1_001.fastq.gz data/reads/sar532-2019_S184_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar532-2019_S184_L001_R1_001.fastq.gz data/reads/sar532-2019_S184_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar532-2019_S184_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar532-2019_S184_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar532-2019_S184_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar532-2019_S184_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar532-2019_S184_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,551\\nReads with adapters:                     1,264 (81.5%)\\nReads written (passing filters):         1,551 (100.0%)\\n\\nTotal basepairs processed:       389,301 bp\\nQuality-trimmed:                  26,418 bp (6.8%)\\nTotal written (filtered):        360,961 bp (92.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1264 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 41.8%\\n  C: 16.9%\\n  G: 38.1%\\n  T: 3.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t627\\t387.8\\t0\\t627\\n2\\t621\\t96.9\\t0\\t621\\n3\\t14\\t24.2\\t0\\t14\\n5\\t1\\t1.5\\t0\\t1\\n6\\t1\\t0.4\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar532-2019_S184_L001_R1_001.fastq.gz\\n=============================================\\n1551 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar532-2019_S184_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar532-2019_S184_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar532-2019_S184_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar532-2019_S184_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar532-2019_S184_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.06 s (36 us/read; 1.68 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,551\\nReads with adapters:                       102 (6.6%)\\nReads written (passing filters):         1,551 (100.0%)\\n\\nTotal basepairs processed:       389,301 bp\\nQuality-trimmed:                  52,428 bp (13.5%)\\nTotal written (filtered):        336,756 bp (86.5%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 102 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 11.8%\\n  C: 21.6%\\n  G: 41.2%\\n  T: 25.5%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t88\\t387.8\\t0\\t88\\n2\\t13\\t96.9\\t0\\t13\\n3\\t1\\t24.2\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar532-2019_S184_L001_R2_001.fastq.gz\\n=============================================\\n1551 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar532-2019_S184_L001_R1_001_trimmed.fq.gz and sar532-2019_S184_L001_R2_001_trimmed.fq.gz\\nfile_1: sar532-2019_S184_L001_R1_001_trimmed.fq.gz, file_2: sar532-2019_S184_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar532-2019_S184_L001_R1_001_trimmed.fq.gz and sar532-2019_S184_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar532-2019_S184_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar532-2019_S184_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1551\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 87 (5.61%)\\n\\n\\n  >>> Now running FastQC on the validated data sar532-2019_S184_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar532-2019_S184_L001_R1_001_val_1.fq.gz\\nApprox 65% complete for sar532-2019_S184_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar532-2019_S184_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar532-2019_S184_L001_R2_001_val_2.fq.gz\\nApprox 65% complete for sar532-2019_S184_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar532-2019_S184_L001_R1_001_trimmed.fq.gz and sar532-2019_S184_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar533-2019_S185_L001_R1_001.fastq.gz data/reads/sar533-2019_S185_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar533-2019_S185_L001_R1_001.fastq.gz data/reads/sar533-2019_S185_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar533-2019_S185_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar533-2019_S185_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar533-2019_S185_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar533-2019_S185_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar533-2019_S185_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.94 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,208\\nReads with adapters:                     1,013 (83.9%)\\nReads written (passing filters):         1,208 (100.0%)\\n\\nTotal basepairs processed:       303,208 bp\\nQuality-trimmed:                  18,486 bp (6.1%)\\nTotal written (filtered):        283,090 bp (93.4%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1013 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 30.3%\\n  C: 11.9%\\n  G: 54.8%\\n  T: 3.0%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t417\\t302.0\\t0\\t417\\n2\\t574\\t75.5\\t0\\t574\\n3\\t21\\t18.9\\t0\\t21\\n4\\t1\\t4.7\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar533-2019_S185_L001_R1_001.fastq.gz\\n=============================================\\n1208 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar533-2019_S185_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar533-2019_S185_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar533-2019_S185_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar533-2019_S185_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar533-2019_S185_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.03 s (24 us/read; 2.45 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,208\\nReads with adapters:                        66 (5.5%)\\nReads written (passing filters):         1,208 (100.0%)\\n\\nTotal basepairs processed:       303,208 bp\\nQuality-trimmed:                  30,804 bp (10.2%)\\nTotal written (filtered):        272,331 bp (89.8%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 66 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 30.3%\\n  C: 27.3%\\n  G: 28.8%\\n  T: 13.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t60\\t302.0\\t0\\t60\\n2\\t5\\t75.5\\t0\\t5\\n3\\t1\\t18.9\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar533-2019_S185_L001_R2_001.fastq.gz\\n=============================================\\n1208 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar533-2019_S185_L001_R1_001_trimmed.fq.gz and sar533-2019_S185_L001_R2_001_trimmed.fq.gz\\nfile_1: sar533-2019_S185_L001_R1_001_trimmed.fq.gz, file_2: sar533-2019_S185_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar533-2019_S185_L001_R1_001_trimmed.fq.gz and sar533-2019_S185_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar533-2019_S185_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar533-2019_S185_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1208\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 49 (4.06%)\\n\\n\\n  >>> Now running FastQC on the validated data sar533-2019_S185_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar533-2019_S185_L001_R1_001_val_1.fq.gz\\nApprox 85% complete for sar533-2019_S185_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar533-2019_S185_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar533-2019_S185_L001_R2_001_val_2.fq.gz\\nApprox 85% complete for sar533-2019_S185_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar533-2019_S185_L001_R1_001_trimmed.fq.gz and sar533-2019_S185_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar534-2019_S186_L001_R1_001.fastq.gz data/reads/sar534-2019_S186_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar534-2019_S186_L001_R1_001.fastq.gz data/reads/sar534-2019_S186_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar534-2019_S186_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar534-2019_S186_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar534-2019_S186_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar534-2019_S186_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar534-2019_S186_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (30 us/read; 2.02 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,538\\nReads with adapters:                     1,262 (82.1%)\\nReads written (passing filters):         1,538 (100.0%)\\n\\nTotal basepairs processed:       386,038 bp\\nQuality-trimmed:                  24,484 bp (6.3%)\\nTotal written (filtered):        359,554 bp (93.1%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1262 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 39.9%\\n  C: 15.4%\\n  G: 41.2%\\n  T: 3.6%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t545\\t384.5\\t0\\t545\\n2\\t698\\t96.1\\t0\\t698\\n3\\t17\\t24.0\\t0\\t17\\n4\\t2\\t6.0\\t0\\t2\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar534-2019_S186_L001_R1_001.fastq.gz\\n=============================================\\n1538 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar534-2019_S186_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar534-2019_S186_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar534-2019_S186_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar534-2019_S186_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar534-2019_S186_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (36 us/read; 1.69 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,538\\nReads with adapters:                        97 (6.3%)\\nReads written (passing filters):         1,538 (100.0%)\\n\\nTotal basepairs processed:       386,038 bp\\nQuality-trimmed:                  47,254 bp (12.2%)\\nTotal written (filtered):        338,675 bp (87.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 97 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 21.6%\\n  C: 23.7%\\n  G: 28.9%\\n  T: 25.8%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t86\\t384.5\\t0\\t86\\n2\\t10\\t96.1\\t0\\t10\\n3\\t1\\t24.0\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar534-2019_S186_L001_R2_001.fastq.gz\\n=============================================\\n1538 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar534-2019_S186_L001_R1_001_trimmed.fq.gz and sar534-2019_S186_L001_R2_001_trimmed.fq.gz\\nfile_1: sar534-2019_S186_L001_R1_001_trimmed.fq.gz, file_2: sar534-2019_S186_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar534-2019_S186_L001_R1_001_trimmed.fq.gz and sar534-2019_S186_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar534-2019_S186_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar534-2019_S186_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1538\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 73 (4.75%)\\n\\n\\n  >>> Now running FastQC on the validated data sar534-2019_S186_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar534-2019_S186_L001_R1_001_val_1.fq.gz\\nApprox 65% complete for sar534-2019_S186_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar534-2019_S186_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar534-2019_S186_L001_R2_001_val_2.fq.gz\\nApprox 65% complete for sar534-2019_S186_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar534-2019_S186_L001_R1_001_trimmed.fq.gz and sar534-2019_S186_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n",
      "sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar535-2019_S187_L001_R1_001.fastq.gz data/reads/sar535-2019_S187_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "----------\n",
      "Executing: sw/TrimGalore-0.6.5/trim_galore --fastqc --paired data/reads/sar535-2019_S187_L001_R1_001.fastq.gz data/reads/sar535-2019_S187_L001_R2_001.fastq.gz --output_dir data/clean/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
      "b\"Multicore support not enabled. Proceeding with single-core trimming.\\nPath to Cutadapt set as: 'cutadapt' (default)\\nCutadapt seems to be working fine (tested command 'cutadapt --version')\\nCutadapt version: 1.15\\nsingle-core operation.\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/juan/Desktop/juan/bio/gliadinas/data/clean/\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar535-2019_S187_L001_R1_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar535-2019_S187_L001_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j 1\\nWriting final adapter and quality trimmed output to sar535-2019_S187_L001_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar535-2019_S187_L001_R1_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar535-2019_S187_L001_R1_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.04 s (31 us/read; 1.96 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,347\\nReads with adapters:                     1,089 (80.8%)\\nReads written (passing filters):         1,347 (100.0%)\\n\\nTotal basepairs processed:       338,097 bp\\nQuality-trimmed:                  23,042 bp (6.8%)\\nTotal written (filtered):        313,287 bp (92.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 1089 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 24.8%\\n  C: 13.0%\\n  G: 59.8%\\n  T: 2.4%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t428\\t336.8\\t0\\t428\\n2\\t645\\t84.2\\t0\\t645\\n3\\t15\\t21.0\\t0\\t15\\n5\\t1\\t1.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar535-2019_S187_L001_R1_001.fastq.gz\\n=============================================\\n1347 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nWriting report to '/home/juan/Desktop/juan/bio/gliadinas/data/clean/sar535-2019_S187_L001_R2_001.fastq.gz_trimming_report.txt'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: data/reads/sar535-2019_S187_L001_R2_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.4_dev\\nCutadapt version: 1.15\\nNumber of cores used for trimming: 1\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' (user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 1 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nRunning FastQC on the data once trimming has completed\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be reasonably up-to-date. Setting -j -j 1\\nWriting final adapter and quality trimmed output to sar535-2019_S187_L001_R2_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff '-q 20') and adapter trimming in a single pass for the adapter sequence: 'CAAGCAGAAGACGGCATACGAGAT' from file data/reads/sar535-2019_S187_L001_R2_001.fastq.gz <<< \\nThis is cutadapt 1.15 with Python 3.6.8\\nCommand line parameters: -j 1 -e 0.1 -q 20 -O 1 -a CAAGCAGAAGACGGCATACGAGAT data/reads/sar535-2019_S187_L001_R2_001.fastq.gz\\nRunning on 1 core\\nTrimming 1 adapter with at most 10.0% errors in single-end mode ...\\nFinished in 0.05 s (37 us/read; 1.61 M reads/minute).\\n\\n=== Summary ===\\n\\nTotal reads processed:                   1,347\\nReads with adapters:                        66 (4.9%)\\nReads written (passing filters):         1,347 (100.0%)\\n\\nTotal basepairs processed:       338,097 bp\\nQuality-trimmed:                  41,580 bp (12.3%)\\nTotal written (filtered):        296,437 bp (87.7%)\\n\\n=== Adapter 1 ===\\n\\nSequence: CAAGCAGAAGACGGCATACGAGAT; Type: regular 3'; Length: 24; Trimmed: 66 times.\\n\\nNo. of allowed errors:\\n0-9 bp: 0; 10-19 bp: 1; 20-24 bp: 2\\n\\nBases preceding removed adapters:\\n  A: 19.7%\\n  C: 21.2%\\n  G: 34.8%\\n  T: 24.2%\\n  none/other: 0.0%\\n\\nOverview of removed sequences\\nlength\\tcount\\texpect\\tmax.err\\terror counts\\n1\\t59\\t336.8\\t0\\t59\\n2\\t3\\t84.2\\t0\\t3\\n3\\t2\\t21.0\\t0\\t2\\n4\\t1\\t5.3\\t0\\t1\\n5\\t1\\t1.3\\t0\\t1\\n\\n\\nRUN STATISTICS FOR INPUT FILE: data/reads/sar535-2019_S187_L001_R2_001.fastq.gz\\n=============================================\\n1347 sequences processed in total\\nThe length threshold of paired-end sequences gets evaluated later on (in the validation step)\\n\\nValidate paired-end files sar535-2019_S187_L001_R1_001_trimmed.fq.gz and sar535-2019_S187_L001_R2_001_trimmed.fq.gz\\nfile_1: sar535-2019_S187_L001_R1_001_trimmed.fq.gz, file_2: sar535-2019_S187_L001_R2_001_trimmed.fq.gz\\n\\n\\n>>>>> Now validing the length of the 2 paired-end infiles: sar535-2019_S187_L001_R1_001_trimmed.fq.gz and sar535-2019_S187_L001_R2_001_trimmed.fq.gz <<<<<\\nWriting validated paired-end Read 1 reads to sar535-2019_S187_L001_R1_001_val_1.fq.gz\\nWriting validated paired-end Read 2 reads to sar535-2019_S187_L001_R2_001_val_2.fq.gz\\n\\nTotal number of sequences analysed: 1347\\n\\nNumber of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 68 (5.05%)\\n\\n\\n  >>> Now running FastQC on the validated data sar535-2019_S187_L001_R1_001_val_1.fq.gz<<<\\n\\nStarted analysis of sar535-2019_S187_L001_R1_001_val_1.fq.gz\\nApprox 75% complete for sar535-2019_S187_L001_R1_001_val_1.fq.gz\\n\\n  >>> Now running FastQC on the validated data sar535-2019_S187_L001_R2_001_val_2.fq.gz<<<\\n\\nStarted analysis of sar535-2019_S187_L001_R2_001_val_2.fq.gz\\nApprox 75% complete for sar535-2019_S187_L001_R2_001_val_2.fq.gz\\nDeleting both intermediate output files sar535-2019_S187_L001_R1_001_trimmed.fq.gz and sar535-2019_S187_L001_R2_001_trimmed.fq.gz\\n\\n====================================================================================================\\n\\n\"\n"
     ]
    }
   ],
   "source": [
    "s = 86\n",
    "for i in range(434, 536):\n",
    "    sample_1 = \"data/reads/sar{}-2019_S{}_L001_R1_001.fastq.gz\".format(i,s)\n",
    "    sample_2 = \"data/reads/sar{}-2019_S{}_L001_R2_001.fastq.gz\".format(i,s)\n",
    "    cmd = \"sw/TrimGalore-0.6.5/trim_galore --fastqc --paired {} {} --output_dir {} -a {} -a {}\"\n",
    "    cmd = cmd.format(sample_1, sample_2, path_output_dir, adapter_f, adapter_r)\n",
    "    print(cmd)\n",
    "    out, err = sys(cmd)\n",
    "    s += 1\n",
    "    #if err:\n",
    "    #    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar434-2019_S86_L001_R1_001_val_1.fq.gz -2 data/clean/sar434-2019_S86_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-434-2019_S86_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar435-2019_S87_L001_R1_001_val_1.fq.gz -2 data/clean/sar435-2019_S87_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-435-2019_S87_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar436-2019_S88_L001_R1_001_val_1.fq.gz -2 data/clean/sar436-2019_S88_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-436-2019_S88_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar437-2019_S89_L001_R1_001_val_1.fq.gz -2 data/clean/sar437-2019_S89_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-437-2019_S89_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar438-2019_S90_L001_R1_001_val_1.fq.gz -2 data/clean/sar438-2019_S90_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-438-2019_S90_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar439-2019_S91_L001_R1_001_val_1.fq.gz -2 data/clean/sar439-2019_S91_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-439-2019_S91_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar440-2019_S92_L001_R1_001_val_1.fq.gz -2 data/clean/sar440-2019_S92_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-440-2019_S92_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar441-2019_S93_L001_R1_001_val_1.fq.gz -2 data/clean/sar441-2019_S93_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-441-2019_S93_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar442-2019_S94_L001_R1_001_val_1.fq.gz -2 data/clean/sar442-2019_S94_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-442-2019_S94_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar443-2019_S95_L001_R1_001_val_1.fq.gz -2 data/clean/sar443-2019_S95_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-443-2019_S95_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar444-2019_S96_L001_R1_001_val_1.fq.gz -2 data/clean/sar444-2019_S96_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-444-2019_S96_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar445-2019_S97_L001_R1_001_val_1.fq.gz -2 data/clean/sar445-2019_S97_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-445-2019_S97_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar446-2019_S98_L001_R1_001_val_1.fq.gz -2 data/clean/sar446-2019_S98_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-446-2019_S98_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar447-2019_S99_L001_R1_001_val_1.fq.gz -2 data/clean/sar447-2019_S99_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-447-2019_S99_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar448-2019_S100_L001_R1_001_val_1.fq.gz -2 data/clean/sar448-2019_S100_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-448-2019_S100_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar449-2019_S101_L001_R1_001_val_1.fq.gz -2 data/clean/sar449-2019_S101_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-449-2019_S101_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar450-2019_S102_L001_R1_001_val_1.fq.gz -2 data/clean/sar450-2019_S102_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-450-2019_S102_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar451-2019_S103_L001_R1_001_val_1.fq.gz -2 data/clean/sar451-2019_S103_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-451-2019_S103_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar452-2019_S104_L001_R1_001_val_1.fq.gz -2 data/clean/sar452-2019_S104_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-452-2019_S104_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar453-2019_S105_L001_R1_001_val_1.fq.gz -2 data/clean/sar453-2019_S105_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-453-2019_S105_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar454-2019_S106_L001_R1_001_val_1.fq.gz -2 data/clean/sar454-2019_S106_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-454-2019_S106_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar455-2019_S107_L001_R1_001_val_1.fq.gz -2 data/clean/sar455-2019_S107_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-455-2019_S107_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar456-2019_S108_L001_R1_001_val_1.fq.gz -2 data/clean/sar456-2019_S108_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-456-2019_S108_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar457-2019_S109_L001_R1_001_val_1.fq.gz -2 data/clean/sar457-2019_S109_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-457-2019_S109_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar458-2019_S110_L001_R1_001_val_1.fq.gz -2 data/clean/sar458-2019_S110_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-458-2019_S110_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar459-2019_S111_L001_R1_001_val_1.fq.gz -2 data/clean/sar459-2019_S111_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-459-2019_S111_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar460-2019_S112_L001_R1_001_val_1.fq.gz -2 data/clean/sar460-2019_S112_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-460-2019_S112_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar461-2019_S113_L001_R1_001_val_1.fq.gz -2 data/clean/sar461-2019_S113_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-461-2019_S113_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar462-2019_S114_L001_R1_001_val_1.fq.gz -2 data/clean/sar462-2019_S114_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-462-2019_S114_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar463-2019_S115_L001_R1_001_val_1.fq.gz -2 data/clean/sar463-2019_S115_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-463-2019_S115_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar464-2019_S116_L001_R1_001_val_1.fq.gz -2 data/clean/sar464-2019_S116_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-464-2019_S116_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar465-2019_S117_L001_R1_001_val_1.fq.gz -2 data/clean/sar465-2019_S117_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-465-2019_S117_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar466-2019_S118_L001_R1_001_val_1.fq.gz -2 data/clean/sar466-2019_S118_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-466-2019_S118_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar467-2019_S119_L001_R1_001_val_1.fq.gz -2 data/clean/sar467-2019_S119_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-467-2019_S119_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar468-2019_S120_L001_R1_001_val_1.fq.gz -2 data/clean/sar468-2019_S120_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-468-2019_S120_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar469-2019_S121_L001_R1_001_val_1.fq.gz -2 data/clean/sar469-2019_S121_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-469-2019_S121_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar470-2019_S122_L001_R1_001_val_1.fq.gz -2 data/clean/sar470-2019_S122_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-470-2019_S122_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar471-2019_S123_L001_R1_001_val_1.fq.gz -2 data/clean/sar471-2019_S123_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-471-2019_S123_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar472-2019_S124_L001_R1_001_val_1.fq.gz -2 data/clean/sar472-2019_S124_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-472-2019_S124_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar473-2019_S125_L001_R1_001_val_1.fq.gz -2 data/clean/sar473-2019_S125_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-473-2019_S125_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar474-2019_S126_L001_R1_001_val_1.fq.gz -2 data/clean/sar474-2019_S126_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-474-2019_S126_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar475-2019_S127_L001_R1_001_val_1.fq.gz -2 data/clean/sar475-2019_S127_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-475-2019_S127_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar476-2019_S128_L001_R1_001_val_1.fq.gz -2 data/clean/sar476-2019_S128_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-476-2019_S128_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar477-2019_S129_L001_R1_001_val_1.fq.gz -2 data/clean/sar477-2019_S129_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-477-2019_S129_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar478-2019_S130_L001_R1_001_val_1.fq.gz -2 data/clean/sar478-2019_S130_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-478-2019_S130_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar479-2019_S131_L001_R1_001_val_1.fq.gz -2 data/clean/sar479-2019_S131_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-479-2019_S131_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar480-2019_S132_L001_R1_001_val_1.fq.gz -2 data/clean/sar480-2019_S132_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-480-2019_S132_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar481-2019_S133_L001_R1_001_val_1.fq.gz -2 data/clean/sar481-2019_S133_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-481-2019_S133_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar482-2019_S134_L001_R1_001_val_1.fq.gz -2 data/clean/sar482-2019_S134_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-482-2019_S134_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar483-2019_S135_L001_R1_001_val_1.fq.gz -2 data/clean/sar483-2019_S135_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-483-2019_S135_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar484-2019_S136_L001_R1_001_val_1.fq.gz -2 data/clean/sar484-2019_S136_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-484-2019_S136_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar485-2019_S137_L001_R1_001_val_1.fq.gz -2 data/clean/sar485-2019_S137_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-485-2019_S137_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar486-2019_S138_L001_R1_001_val_1.fq.gz -2 data/clean/sar486-2019_S138_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-486-2019_S138_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar487-2019_S139_L001_R1_001_val_1.fq.gz -2 data/clean/sar487-2019_S139_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-487-2019_S139_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar488-2019_S140_L001_R1_001_val_1.fq.gz -2 data/clean/sar488-2019_S140_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-488-2019_S140_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar489-2019_S141_L001_R1_001_val_1.fq.gz -2 data/clean/sar489-2019_S141_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-489-2019_S141_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar490-2019_S142_L001_R1_001_val_1.fq.gz -2 data/clean/sar490-2019_S142_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-490-2019_S142_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar491-2019_S143_L001_R1_001_val_1.fq.gz -2 data/clean/sar491-2019_S143_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-491-2019_S143_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar492-2019_S144_L001_R1_001_val_1.fq.gz -2 data/clean/sar492-2019_S144_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-492-2019_S144_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar493-2019_S145_L001_R1_001_val_1.fq.gz -2 data/clean/sar493-2019_S145_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-493-2019_S145_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar494-2019_S146_L001_R1_001_val_1.fq.gz -2 data/clean/sar494-2019_S146_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-494-2019_S146_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar495-2019_S147_L001_R1_001_val_1.fq.gz -2 data/clean/sar495-2019_S147_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-495-2019_S147_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar496-2019_S148_L001_R1_001_val_1.fq.gz -2 data/clean/sar496-2019_S148_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-496-2019_S148_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar497-2019_S149_L001_R1_001_val_1.fq.gz -2 data/clean/sar497-2019_S149_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-497-2019_S149_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar498-2019_S150_L001_R1_001_val_1.fq.gz -2 data/clean/sar498-2019_S150_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-498-2019_S150_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar499-2019_S151_L001_R1_001_val_1.fq.gz -2 data/clean/sar499-2019_S151_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-499-2019_S151_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar500-2019_S152_L001_R1_001_val_1.fq.gz -2 data/clean/sar500-2019_S152_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-500-2019_S152_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar501-2019_S153_L001_R1_001_val_1.fq.gz -2 data/clean/sar501-2019_S153_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-501-2019_S153_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar502-2019_S154_L001_R1_001_val_1.fq.gz -2 data/clean/sar502-2019_S154_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-502-2019_S154_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar503-2019_S155_L001_R1_001_val_1.fq.gz -2 data/clean/sar503-2019_S155_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-503-2019_S155_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar504-2019_S156_L001_R1_001_val_1.fq.gz -2 data/clean/sar504-2019_S156_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-504-2019_S156_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar505-2019_S157_L001_R1_001_val_1.fq.gz -2 data/clean/sar505-2019_S157_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-505-2019_S157_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar506-2019_S158_L001_R1_001_val_1.fq.gz -2 data/clean/sar506-2019_S158_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-506-2019_S158_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar507-2019_S159_L001_R1_001_val_1.fq.gz -2 data/clean/sar507-2019_S159_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-507-2019_S159_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar508-2019_S160_L001_R1_001_val_1.fq.gz -2 data/clean/sar508-2019_S160_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-508-2019_S160_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar509-2019_S161_L001_R1_001_val_1.fq.gz -2 data/clean/sar509-2019_S161_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-509-2019_S161_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar510-2019_S162_L001_R1_001_val_1.fq.gz -2 data/clean/sar510-2019_S162_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-510-2019_S162_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar511-2019_S163_L001_R1_001_val_1.fq.gz -2 data/clean/sar511-2019_S163_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-511-2019_S163_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar512-2019_S164_L001_R1_001_val_1.fq.gz -2 data/clean/sar512-2019_S164_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-512-2019_S164_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar513-2019_S165_L001_R1_001_val_1.fq.gz -2 data/clean/sar513-2019_S165_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-513-2019_S165_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar514-2019_S166_L001_R1_001_val_1.fq.gz -2 data/clean/sar514-2019_S166_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-514-2019_S166_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar515-2019_S167_L001_R1_001_val_1.fq.gz -2 data/clean/sar515-2019_S167_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-515-2019_S167_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar516-2019_S168_L001_R1_001_val_1.fq.gz -2 data/clean/sar516-2019_S168_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-516-2019_S168_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar517-2019_S169_L001_R1_001_val_1.fq.gz -2 data/clean/sar517-2019_S169_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-517-2019_S169_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar518-2019_S170_L001_R1_001_val_1.fq.gz -2 data/clean/sar518-2019_S170_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-518-2019_S170_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar519-2019_S171_L001_R1_001_val_1.fq.gz -2 data/clean/sar519-2019_S171_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-519-2019_S171_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar520-2019_S172_L001_R1_001_val_1.fq.gz -2 data/clean/sar520-2019_S172_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-520-2019_S172_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar521-2019_S173_L001_R1_001_val_1.fq.gz -2 data/clean/sar521-2019_S173_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-521-2019_S173_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar522-2019_S174_L001_R1_001_val_1.fq.gz -2 data/clean/sar522-2019_S174_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-522-2019_S174_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar523-2019_S175_L001_R1_001_val_1.fq.gz -2 data/clean/sar523-2019_S175_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-523-2019_S175_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar524-2019_S176_L001_R1_001_val_1.fq.gz -2 data/clean/sar524-2019_S176_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-524-2019_S176_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar525-2019_S177_L001_R1_001_val_1.fq.gz -2 data/clean/sar525-2019_S177_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-525-2019_S177_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar526-2019_S178_L001_R1_001_val_1.fq.gz -2 data/clean/sar526-2019_S178_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-526-2019_S178_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar527-2019_S179_L001_R1_001_val_1.fq.gz -2 data/clean/sar527-2019_S179_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-527-2019_S179_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar528-2019_S180_L001_R1_001_val_1.fq.gz -2 data/clean/sar528-2019_S180_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-528-2019_S180_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar529-2019_S181_L001_R1_001_val_1.fq.gz -2 data/clean/sar529-2019_S181_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-529-2019_S181_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar530-2019_S182_L001_R1_001_val_1.fq.gz -2 data/clean/sar530-2019_S182_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-530-2019_S182_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar531-2019_S183_L001_R1_001_val_1.fq.gz -2 data/clean/sar531-2019_S183_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-531-2019_S183_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar532-2019_S184_L001_R1_001_val_1.fq.gz -2 data/clean/sar532-2019_S184_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-532-2019_S184_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar533-2019_S185_L001_R1_001_val_1.fq.gz -2 data/clean/sar533-2019_S185_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-533-2019_S185_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar534-2019_S186_L001_R1_001_val_1.fq.gz -2 data/clean/sar534-2019_S186_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-534-2019_S186_L001_R2_001.fasta\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 data/clean/sar535-2019_S187_L001_R1_001_val_1.fq.gz -2 data/clean/sar535-2019_S187_L001_R2_001_val_2.fq.gz -o data/assembly/assembly-535-2019_S187_L001_R2_001.fasta\n"
     ]
    }
   ],
   "source": [
    "s = 86\n",
    "for i in range(434, 536):\n",
    "    sample_1 = \"data/clean/sar{}-2019_S{}_L001_R1_001_val_1.fq.gz\".format(i,s)\n",
    "    sample_2 = \"data/clean/sar{}-2019_S{}_L001_R2_001_val_2.fq.gz\".format(i,s)\n",
    "    output = \"data/assembly/assembly-{}-2019_S{}_L001_R2_001.fasta\".format(i,s)\n",
    "    cmd = \"/home/juan/Desktop/juan/bio/gliadinas/sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 {} -2 {} -o {}\"\n",
    "    cmd = cmd.format(sample_1, sample_2, output)\n",
    "    out, err = sys(cmd)\n",
    "    s += 1\n",
    "    #if err:\n",
    "    #    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-434-2019_S86_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-434-2019_S86.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-435-2019_S87_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-435-2019_S87.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-436-2019_S88_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-436-2019_S88.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-437-2019_S89_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-437-2019_S89.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-438-2019_S90_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-438-2019_S90.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-439-2019_S91_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-439-2019_S91.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-440-2019_S92_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-440-2019_S92.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-441-2019_S93_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-441-2019_S93.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-442-2019_S94_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-442-2019_S94.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-443-2019_S95_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-443-2019_S95.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-444-2019_S96_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-444-2019_S96.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-445-2019_S97_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-445-2019_S97.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-446-2019_S98_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-446-2019_S98.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-447-2019_S99_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-447-2019_S99.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-448-2019_S100_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-448-2019_S100.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-449-2019_S101_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-449-2019_S101.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-450-2019_S102_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-450-2019_S102.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-451-2019_S103_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-451-2019_S103.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-452-2019_S104_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-452-2019_S104.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-453-2019_S105_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-453-2019_S105.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-454-2019_S106_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-454-2019_S106.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-455-2019_S107_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-455-2019_S107.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-456-2019_S108_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-456-2019_S108.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-457-2019_S109_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-457-2019_S109.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-458-2019_S110_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-458-2019_S110.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-459-2019_S111_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-459-2019_S111.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-460-2019_S112_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-460-2019_S112.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-461-2019_S113_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-461-2019_S113.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-462-2019_S114_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-462-2019_S114.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-463-2019_S115_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-463-2019_S115.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-464-2019_S116_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-464-2019_S116.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-465-2019_S117_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-465-2019_S117.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-466-2019_S118_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-466-2019_S118.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-467-2019_S119_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-467-2019_S119.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-468-2019_S120_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-468-2019_S120.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-469-2019_S121_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-469-2019_S121.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-470-2019_S122_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-470-2019_S122.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-471-2019_S123_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-471-2019_S123.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-472-2019_S124_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-472-2019_S124.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-473-2019_S125_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-473-2019_S125.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-474-2019_S126_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-474-2019_S126.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-475-2019_S127_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-475-2019_S127.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-476-2019_S128_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-476-2019_S128.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-477-2019_S129_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-477-2019_S129.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-478-2019_S130_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-478-2019_S130.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-479-2019_S131_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-479-2019_S131.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-480-2019_S132_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-480-2019_S132.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-481-2019_S133_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-481-2019_S133.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-482-2019_S134_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-482-2019_S134.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-483-2019_S135_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-483-2019_S135.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-484-2019_S136_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-484-2019_S136.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-485-2019_S137_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-485-2019_S137.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-486-2019_S138_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-486-2019_S138.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-487-2019_S139_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-487-2019_S139.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-488-2019_S140_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-488-2019_S140.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-489-2019_S141_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-489-2019_S141.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-490-2019_S142_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-490-2019_S142.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-491-2019_S143_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-491-2019_S143.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-492-2019_S144_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-492-2019_S144.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-493-2019_S145_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-493-2019_S145.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-494-2019_S146_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-494-2019_S146.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-495-2019_S147_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-495-2019_S147.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-496-2019_S148_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-496-2019_S148.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-497-2019_S149_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-497-2019_S149.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-498-2019_S150_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-498-2019_S150.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-499-2019_S151_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-499-2019_S151.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-500-2019_S152_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-500-2019_S152.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-501-2019_S153_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-501-2019_S153.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-502-2019_S154_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-502-2019_S154.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-503-2019_S155_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-503-2019_S155.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-504-2019_S156_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-504-2019_S156.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-505-2019_S157_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-505-2019_S157.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-506-2019_S158_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-506-2019_S158.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-507-2019_S159_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-507-2019_S159.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-508-2019_S160_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-508-2019_S160.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-509-2019_S161_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-509-2019_S161.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-510-2019_S162_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-510-2019_S162.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-511-2019_S163_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-511-2019_S163.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-512-2019_S164_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-512-2019_S164.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-513-2019_S165_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-513-2019_S165.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-514-2019_S166_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-514-2019_S166.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-515-2019_S167_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-515-2019_S167.fasta -c 0.98\n",
      "b'\\nFatal Error:\\nFailed to open the database file\\nProgram halted !!\\n\\n'\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-516-2019_S168_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-516-2019_S168.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-517-2019_S169_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-517-2019_S169.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-518-2019_S170_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-518-2019_S170.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-519-2019_S171_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-519-2019_S171.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-520-2019_S172_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-520-2019_S172.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-521-2019_S173_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-521-2019_S173.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-522-2019_S174_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-522-2019_S174.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-523-2019_S175_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-523-2019_S175.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-524-2019_S176_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-524-2019_S176.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-525-2019_S177_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-525-2019_S177.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-526-2019_S178_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-526-2019_S178.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-527-2019_S179_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-527-2019_S179.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-528-2019_S180_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-528-2019_S180.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-529-2019_S181_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-529-2019_S181.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-530-2019_S182_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-530-2019_S182.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-531-2019_S183_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-531-2019_S183.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-532-2019_S184_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-532-2019_S184.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-533-2019_S185_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-533-2019_S185.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-534-2019_S186_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-534-2019_S186.fasta -c 0.98\n",
      "----------\n",
      "Executing: /home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i data/assembly/assembly-535-2019_S187_L001_R2_001.fasta/contigs.fasta -o data/clusters/cluster-535-2019_S187.fasta -c 0.98\n"
     ]
    }
   ],
   "source": [
    "s = 86\n",
    "for i in range(434, 536):\n",
    "    assembly = \"data/assembly/assembly-{}-2019_S{}_L001_R2_001.fasta/contigs.fasta\".format(i,s)\n",
    "    output = \"data/clusters/cluster-{}-2019_S{}.fasta\".format(i,s)\n",
    "    cmd = \"/home/juan/Desktop/juan/bio/gliadinas/sw/cd-hit-v4.8.1-2019-0228/cd-hit-est -i {} -o {} -c 0.98\"\n",
    "    cmd = cmd.format(assembly, output)\n",
    "    out, err = sys(cmd)\n",
    "    s += 1\n",
    "    #if err:\n",
    "    #    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/usr/bin/env bds\n",
    "\n",
    "string adapters = \"data/adapters_new_rc.fa\"\n",
    "\n",
    "\n",
    "sys export PATH=$PATH:$PWD/sw/SPAdes-3.13.0-Linux/bin\n",
    "sys export PATH=$PATH:$PWD/sw/.bds\n",
    "sys export PATH=$PATH:$PWD/sw/cutadapt-2.7\n",
    "sys export PATH=$PATH:$PWD/sw/FastQC\n",
    "\n",
    "sys rm -r data/clean_1/*\n",
    "sys rm -r data/clean_2/*\n",
    "sys rm -r data/clean_3/*\n",
    "sys rm -r data/assembly/*\n",
    "\n",
    "int s = 86\n",
    "for( int i=434 ; i <= 535 ; i++ ) {\n",
    "\n",
    "    string sample_1_1 = \"data/reads/sar\"+i+\"-2019_S\"+s+\"_L001_R1_001.fastq.gz\"\n",
    "    string sample_2_1 = \"data/reads/sar\"+i+\"-2019_S\"+s+\"_L001_R2_001.fastq.gz\"\n",
    "    try{\n",
    "        sys sw/TrimGalore-0.6.5/trim_galore --paired $sample_1_1 $sample_2_1 --output_dir data/clean_1/\n",
    "    }catch (Exception e){\n",
    "        print \"error in $sample_1_1 $sample_2_1\\n\"\n",
    "    }\n",
    "    \n",
    "\n",
    "    string sample_1_2 = \"data/clean_1/sar\"+i+\"-2019_S\"+s+\"_L001_R1_001_val_1.fq.gz\"\n",
    "    string sample_2_2 = \"data/clean_1/sar\"+i+\"-2019_S\"+s+\"_L001_R2_001_val_2.fq.gz\"\n",
    "    try{\n",
    "        sys sw/TrimGalore-0.6.5/trim_galore --fastqc --paired $sample_1_2 $sample_2_2 --output_dir data/clean_2/ -a AATGATACGGCGACCACCGAGATCTACAC -a CAAGCAGAAGACGGCATACGAGAT\n",
    "    }catch (Exception e){\n",
    "        print \"error in $sample_1_2 $sample_2_2\\n\"\n",
    "    }\n",
    "\n",
    "    string sample_1_3 = \"data/clean_2/sar\"+i+\"-2019_S\"+s+\"_L001_R1_001_val_1_val_1.fq.gz\"\n",
    "    string sample_2_3 = \"data/clean_2/sar\"+i+\"-2019_S\"+s+\"_L001_R2_001_val_2_val_2.fq.gz\"\n",
    "    try{\n",
    "        sys sw/TrimGalore-0.6.5/trim_galore --fastqc --paired $sample_1_3 $sample_2_3 --output_dir data/clean_3/ -a GTGTAGATCTCGGTGGTCGCCGTATCATT -a ATCTCGTATGCCGTCTTCTGCTTG\n",
    "    }catch (Exception e){\n",
    "        print \"error in $sample_1_3 $sample_2_3\\n\"\n",
    "    }\n",
    "\n",
    "    string sample_1 = \"data/clean_3/sar\"+i+\"-2019_S\"+s+\"_L001_R1_001_val_1_val_1_val_1.fq.gz\"\n",
    "    string sample_2 = \"data/clean_3/sar\"+i+\"-2019_S\"+s+\"_L001_R2_001_val_2_val_2_val_2.fq.gz\"\n",
    "    string sample_out = \"data/assembly/sar\" + i + \"-2019_S\" + s + \"/\"\n",
    "    \n",
    "    sys sw/SPAdes-3.13.0-Linux/bin/spades.py --careful -1 $sample_1_3 -2 $sample_2_3 -o $sample_out\n",
    "\n",
    "    s = s + 1\n",
    "}\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
